<!DOCTYPE html>
<html lang="en">
<head prefix="og: http://ogp.me/ns#">
    <meta charset="utf-8">
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta name="referrer" content="origin">
    <meta name="Rg-Request-Token" id="Rg-Request-Token" content="aad-R4iy2IO7Z+D12/rwoVgVqEXiUTGDNTCvoB02xD0FTSDOzj0opT9UIkzUM07GsVLrgGpYcqxtc9VYLY7tdrp1A+6r7+XqW5RGCnefJCU36dvGVLnQFcE+pojvGVqIF5UdC1aAJhxUX1kmnfv7osRTsdrIi+1v2RRpY7gaPQ16u8iCAqcx4njfsf2WRW+SF42M4jwWtxU9TlDvQDUOwa7dcO1rM5Cj+mKejoMVpmttTwldm8zWjlArqX3rU6O2IpHAfpN0xERpya/7yPUkCoU=">
    <meta http-equiv="expires" content="0">
    <link rel="preload" as="script" href="https://c5.rgstatic.net/m/4183121042180509/javascript/lite/lite.js" crossOrigin="anonymous">        <link rel="icon" type="image/png" href="https://c5.rgstatic.net/m/41542880220916/images/favicon/favicon-32x32.png" sizes="32x32">
    <link rel="icon" type="image/png" href="https://www.researchgate.net/favicon-96x96.png" sizes="96x96">
    <link rel="icon" type="image/png" href="https://c5.rgstatic.net/m/42134677640462/images/favicon/favicon-16x16.png" sizes="16x16">
    <link rel="shortcut icon" type="image/x-icon" href="https://c5.rgstatic.net/m/426351313275430/images/favicon/favicon.ico">
    <link rel="mask-icon" href="https://c5.rgstatic.net/m/441272785461785/images/favicon/favicon.svg" color="#00ccbb">    <link rel="manifest" href="https://www.researchgate.net/manifest.json">
    <meta name="application-name" content="ResearchGate">
    <meta name="msapplication-TileColor" content="#00ccbb">    <meta name="msapplication-TileImage" content="https://www.researchgate.net/mstile-144x144.png">
    <meta name="theme-color" content="#444444">     <link rel="search" type="application/opensearchdescription+xml" title="ResearchGate search" href="https://www.researchgate.net/application.DownloadOpenSearchPlugin.html">
    <link rel="meta" type="application/rdf+xml" title="ICRA labels" href="https://www.researchgate.net/application.DownloadLabels.html">
    <link rel="http://oexchange.org/spec/0.8/rel/related-target" type="application/xrd+xml" href="https://www.researchgate.net/application.DownloadOExchange.html">
    <base href="https://www.researchgate.net/"/>
    <script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">var pageConfig={"module":"lite","action":"lite.PublicationDetails","product":"application","stylesHome":"https:\/\/c5.rgstatic.net\/m\/","staticHost":"https:\/\/c5.rgstatic.net","yuiDisabled":true,"longRunningRequestIdentifier":"LongRunningRequest.lite.PublicationDetails","longRunningRequestFp":"72b825cfd06b891b98628b96f2fb7028d123ac29","platformFeatures":{"nova":[],"rigel":{"useRigelForgeryOverlay":false,"useNativeEmbeds":false},"react":{"propTypesStrictMode":false}},"isLitePage":true};var requestConfig={"correlationId":"rgreq-e45334f3c3c8ed80b7578f91463ce383","isLoggedIn":false};var requestCachedConfig={"isCachedPage":true,"correlationId":"rgreq-e9b06d24d157003c3d0ca93a3887c852","backendTime":3};window.rootUrl="https://www.researchgate.net/";</script>
            <style>
            .qc-cmp-ui .qc-cmp-arrow-down {
                background: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='none' stroke='%230080ff' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3E%3Cpath d='M2 5l6 6 6-6'/%3E%3C/svg%3E") 50% no-repeat !important;
            }
        </style>
        <script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">var __wd=["https://www.researchgate.net/", "https://c5.rgstatic.net/", "https://quantcast.mgr.consensu.org", "https://static.quantcast.mgr.consensu.org"];window.quantcastCountryDecision=true;window.useQuantcast=true;
            (function(d,k){function l(b){return(new RegExp("^(.*;)?\\s*"+b+"\\s*=")).test(k.cookie)}function m(b){return(b=(new RegExp("^(.*;)?\\s*"+b+"\\s*=(\\s*[^;])")).exec(k.cookie))&&b[2]||""}function n(){var b=!1;if(l("disable_quantcast")&&"1"===m("disable_quantcast"))return!1;if(d.quantcastCountryDecision)return!0;if(d.trackingConfig){if(d.trackingConfig.isCrawler)return!1;b=d.trackingConfig.isEU}l("cur_country_is_eu")&&(b="1"===m("cur_country_is_eu"));return b}function f(b,d){return d.some(function(c){return 0===
                b.indexOf(c)})}var p=!1,g=[],q=[],h=["appendChild","insertBefore","replaceChild"];if(n())for(var e=0;e<h.length;e++)Element.prototype[h[e]]=function(b){var e=Element.prototype[b];return function(){var c;if(c=0<arguments.length){c=arguments;var a;if(a=0<c.length&&c[0].tagName&&-1!==["script","iframe","link","style"].indexOf(c[0].tagName.toLowerCase()))a=c[0],a.src||a.href?(a=a.src||a.href,f(a,d.__wd)&&-1===a.indexOf("/delivery/manager.js")?a=!0:(a=f(a,q),a=p&&!a)):a=!0,a=!a;a?(g.push({s:this,m:b,a:c}),
                c=!0):c=!1}if(!c)return e.apply(this,arguments)}}(h[e]);d.loadDeferredObjects=function(b){b=b||[];p=!0;q=b;var d=[];g.forEach(function(c){var a=c.a[0];f(a.src||a.href,b)?d.push(c):Element.prototype[c.m].apply(c.s,c.a)});g=d};d.canAskForCookieConsent=n})(window,document);</script>
        <script async=true nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">
            if(window.canAskForCookieConsent()&&(function(){var a=new XMLHttpRequest,e=window.location.hostname,b=document.createElement("script"),d=document.getElementsByTagName("script")[0],f=(new Date).getTime(),c="https://quantcast.mgr.consensu.org".concat("/choice/","AaqDBHcavs6a0","/",e,"/choice.js").concat("?timestamp=",f);a.onreadystatechange=function(){if(4===this.readyState){b.async=!0;b.type="text/javascript";if(200===this.status)b.src=c;else{var a="https://quantcast.mgr.consensu.org".concat("/choice.js");
                c=b.src=a}d.parentNode.insertBefore(b,d)}};a.open("GET",c,!0);a.send()}(),"undefined"===typeof window.__cmp)){var count=0;window.__cmp=function(){var a=arguments;if("object"!=typeof window.__cmp.a)10>count?(setTimeout(function(){window.__cmp.apply(window.__cmp,a)},400),count++):console.warn("CMP not loaded after 4 seconds");else return window.__cmp.apply(window.__cmp,a)}};
            (function(s,o,g){var a,m;a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(document,'script', 'https://www.researchgate.net/javascript/cookie-consent/cookie-consent.js?rnd=' + (new Date).getTime());</script>
        <noscript></noscript><meta property="twitter:site" content="@ResearchGate">
<meta property="twitter:creator" content="@ResearchGate">
<meta property="twitter:card" content="summary">
<meta property="twitter:url" content="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera">
<meta property="og:site" content="ResearchGate">
<meta property="og:site_name" content="ResearchGate">
<meta property="og:type" content="website">
<meta property="og:title" content="Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera | Request PDF">
<meta property="og:description" content="Request PDF | Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera | Bristly millipedes (subclass Penicillata, order Polyxenida) are minute diplopods characterised by uncalcified cuticle and a body covered with... | Find, read and cite all the research you need on ResearchGate">
<meta property="og:url" content="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera">
<meta property="og:image" content="https://www.researchgate.net/images/template/default_publication_preview_large.png">
<meta property="gs_meta_revision" content="1.1">
<meta property="citation_title" content="Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera">
<meta property="citation_abstract_html_url" content="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera">
<meta property="citation_fulltext_html_url" content="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera">
<meta property="citation_author" content="Megan Short">
<meta property="citation_author" content="Varpu Vahtera">
<meta property="citation_publication_date" content="2017/10/10">
<meta property="citation_journal_title" content="Journal of Natural History">
<meta property="citation_issn" content="0022-2933">
<meta property="citation_volume" content="51">
<meta property="citation_issue" content="41-42">
<meta property="citation_firstpage" content="2443">
<meta property="citation_lastpage" content="2461">
<meta property="citation_doi" content="10.1080/00222933.2017.1380241">
<meta property="DC.identifier" content="http://dx.doi.org/10.1080/00222933.2017.1380241">
<meta property="rg:id" content="PB:320321240">
<link rel="canonical" href="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera">
<link rel="dns-prefetch" href="https://c5.rgstatic.net">
<link rel="dns-prefetch" href="https://i1.rgstatic.net">
<link rel="preconnect" href="https://www.googletagservices.com">
<link rel="preconnect" href="https://securepubads.g.doubleclick.net">
<link rel="preconnect" href="https://www.google-analytics.com">
<link rel="preconnect" href="https://tags.crwdcntrl.net">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="robots" content="noarchive">
<script type="application/ld+json">{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2017-10-10","headline":"Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera","image":"https:\/\/www.researchgate.net\/images\/template\/default_publication_preview_large.png","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Megan Short","url":"https:\/\/www.researchgate.net\/profile\/Megan_Short3","image":"https:\/\/c5.rgstatic.net\/m\/4671872220764\/images\/template\/default\/profile\/profile_default_m.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Deakin University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Varpu Vahtera","url":"https:\/\/www.researchgate.net\/profile\/Varpu_Vahtera","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/272488118616066-1441977638089_Q64\/Varpu_Vahtera.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"University of Turku"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Journal of Natural History","url":"http:\/\/www.informaworld.com\/openurl?genre=journal&issn=00222933"},"citation":[{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2016-07-01","headline":"MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/329566653_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Sudhir Kumar","url":"https:\/\/www.researchgate.net\/profile\/Sudhir_Kumar35","image":"https:\/\/c5.rgstatic.net\/m\/4671872220764\/images\/template\/default\/profile\/profile_default_m.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"amara hospital tirupati"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Glen Stecher","url":"https:\/\/www.researchgate.net\/profile\/Glen_Stecher","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273740978192404-1442276343506_Q64\/Glen_Stecher.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Temple University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Koichiro Tamura","url":"https:\/\/www.researchgate.net\/profile\/Koichiro_Tamura","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273705918005258-1442267984412_Q64\/Koichiro_Tamura.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Tokyo Metropolitan University"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Molecular Biology and Evolution","url":"http:\/\/mbe.oxfordjournals.org\/"}},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2016-07-01","headline":"MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/318502387_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Sudhir Kumar","url":"https:\/\/www.researchgate.net\/profile\/Sudhir_Kumar35","image":"https:\/\/c5.rgstatic.net\/m\/4671872220764\/images\/template\/default\/profile\/profile_default_m.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"amara hospital tirupati"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Glen Stecher","url":"https:\/\/www.researchgate.net\/profile\/Glen_Stecher","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273740978192404-1442276343506_Q64\/Glen_Stecher.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Temple University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Koichiro Tamura","url":"https:\/\/www.researchgate.net\/profile\/Koichiro_Tamura","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273705918005258-1442267984412_Q64\/Koichiro_Tamura.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Tokyo Metropolitan University"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Molecular Biology and Evolution","url":"http:\/\/mbe.oxfordjournals.org\/"}},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2015-01-01","headline":"MEGA7: Molecular evolutionary genetics analysis version 7.0 for bigger datasets","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/313391996_MEGA7_Molecular_evolutionary_genetics_analysis_version_70_for_bigger_datasets","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Sumitxl Kumar","url":"https:\/\/www.researchgate.net\/profile\/Sumitxl_Kumar","image":"https:\/\/c5.rgstatic.net\/m\/4671872220764\/images\/template\/default\/profile\/profile_default_m.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Arizona State University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Glen Stecher","url":"https:\/\/www.researchgate.net\/profile\/Glen_Stecher","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273740978192404-1442276343506_Q64\/Glen_Stecher.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Temple University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Koichiro Tamura","url":"https:\/\/www.researchgate.net\/profile\/Koichiro_Tamura","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273705918005258-1442267984412_Q64\/Koichiro_Tamura.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Tokyo Metropolitan University"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Molecular Biology and Evolution","url":"http:\/\/mbe.oxfordjournals.org\/"}},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2016-04-25","headline":"TNT version 1.5, including a full implementation of phylogenetic morphometrics","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/301627335_TNT_version_15_including_a_full_implementation_of_phylogenetic_morphometrics","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Pablo A. Goloboff","url":"https:\/\/www.researchgate.net\/profile\/Pablo_Goloboff","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/277069004066826-1443069806230_Q64\/Pablo_Goloboff.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"National Scientific and Technical Research Council"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Santiago A. Catalano","url":"https:\/\/www.researchgate.net\/profile\/Santiago_Catalano","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273674053877789-1442260387933_Q64\/Santiago_Catalano.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"National Scientific and Technical Research Council"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Cladistics","url":"http:\/\/www.interscience.wiley.com\/jpages\/0748-3007\/"}},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2016-03-22","headline":"MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/299371757_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Sudhir Kumar","url":"https:\/\/www.researchgate.net\/profile\/Sudhir_Kumar35","image":"https:\/\/c5.rgstatic.net\/m\/4671872220764\/images\/template\/default\/profile\/profile_default_m.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"amara hospital tirupati"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Glen Stecher","url":"https:\/\/www.researchgate.net\/profile\/Glen_Stecher","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273740978192404-1442276343506_Q64\/Glen_Stecher.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Temple University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Koichiro Tamura","url":"https:\/\/www.researchgate.net\/profile\/Koichiro_Tamura","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273705918005258-1442267984412_Q64\/Koichiro_Tamura.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Tokyo Metropolitan University"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Molecular Biology and Evolution","url":"http:\/\/mbe.oxfordjournals.org\/"}},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2020-02-10","headline":"Histoire naturelle des animaux articul\u00e9s. Crustac\u00e9s, Arachnides, Myriapodes et Hexapodes","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/291022395_Histoire_naturelle_des_animaux_articules_Crustaces_Arachnides_Myriapodes_et_Hexapodes","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"H. Lucas","url":"https:\/\/www.researchgate.net\/scientific-contributions\/2094834961_H_Lucas","image":"https:\/\/c5.rgstatic.net\/m\/435982309481010\/images\/template\/default\/author\/author_default_m.jpg"}]},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2008-01-01","headline":"Present classification of Diplopoda Penicillata (Myriapoda), with new definitions of taxa","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/290185348_Present_classification_of_Diplopoda_Penicillata_Myriapoda_with_new_definitions_of_taxa","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"B. Cond\u00e9","url":"https:\/\/www.researchgate.net\/scientific-contributions\/2093527069_B_Conde","image":"https:\/\/c5.rgstatic.net\/m\/435982309481010\/images\/template\/default\/author\/author_default_m.jpg"},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"M. Nguyen Duy-Jacquemin","url":"https:\/\/www.researchgate.net\/scientific-contributions\/2094409504_M_Nguyen_Duy-Jacquemin","image":"https:\/\/c5.rgstatic.net\/m\/435982309481010\/images\/template\/default\/author\/author_default_m.jpg"}]},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2020-02-10","headline":"Results of Dr. E. Mj\u00f6berg's scientific expeditions to Australia 1910-1913. 34. Myriapoda: Diplopoda","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/287356655_Results_of_Dr_E_Mjoberg's_scientific_expeditions_to_Australia_1910-1913_34_Myriapoda_Diplopoda","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"K.W. Verhoeff","url":"https:\/\/www.researchgate.net\/scientific-contributions\/2089661383_KW_Verhoeff","image":"https:\/\/c5.rgstatic.net\/m\/435982309481010\/images\/template\/default\/author\/author_default_m.jpg"}]},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2003-01-01","headline":"Fossil giants and surviving dwarfs. Arthropleurida and Pselaphognatha (Atelocerata, Diplopoda):...","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/286669749_Fossil_giants_and_surviving_dwarfs_Arthropleurida_and_Pselaphognatha_Atelocerata_Diplopoda_Characters_phylogenetic_relationships_and_construction","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"O. Kraus","url":"https:\/\/www.researchgate.net\/scientific-contributions\/2088463375_O_Kraus","image":"https:\/\/c5.rgstatic.net\/m\/435982309481010\/images\/template\/default\/author\/author_default_m.jpg"},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Carsten Brauckmann","url":"https:\/\/www.researchgate.net\/profile\/Carsten_Brauckmann","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/272913618173959-1442079085304_Q64\/Carsten_Brauckmann.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Technische Universit\u00e4t Clausthal"}}]},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"1758-01-01","headline":"Systema Naturae per Regna Tria Naturae, Secundum Classes, Ordines, Genera, Species, cum Characteribus,...","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/285818651_Systema_Naturae_per_Regna_Tria_Naturae_Secundum_Classes_Ordines_Genera_Species_cum_Characteribus_Differentiis_Synonymis_Locis","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"C. Linnaeus","url":"https:\/\/www.researchgate.net\/scientific-contributions\/2087348717_C_Linnaeus","image":"https:\/\/c5.rgstatic.net\/m\/435982309481010\/images\/template\/default\/author\/author_default_m.jpg"}]},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2018-08-01","headline":"Phylogenetic Study of Dioecious and Parthenogenetic Populations of Canthocamptus staphylinus (Crustacea,...","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/327352347_Phylogenetic_Study_of_Dioecious_and_Parthenogenetic_Populations_of_Canthocamptus_staphylinus_Crustacea_Copepoda_Harpacticoida","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Elena Kochanova","url":"https:\/\/www.researchgate.net\/profile\/Elena_Kochanova","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/477857798463489-1490941585181_Q64\/Elena_Kochanova.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"University of Helsinki"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Jouko Sarvala","url":"https:\/\/www.researchgate.net\/profile\/Jouko_Sarvala","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273630126931972-1442249914236_Q64\/Jouko_Sarvala.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"University of Turku"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"E. B. Fefilova","url":"https:\/\/www.researchgate.net\/profile\/E_Fefilova","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/638419080343553-1529222378747_Q64\/E_Fefilova.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Komi Scientific Center"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Bulletin Southern California Academy of Sciences"}},{"@context":"https:\/\/schema.org\/","@type":"ScholarlyArticle","datePublished":"2018-01-01","headline":"Two new species of Phryssonotus (Diplopoda:Synxenidae) from southern and Western Australia","mainEntityOfPage":"https:\/\/www.researchgate.net\/publication\/323352482_Two_new_species_of_Phryssonotus_DiplopodaSynxenidae_from_southern_and_Western_Australia","author":[{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Cuong Huynh","url":"https:\/\/www.researchgate.net\/profile\/Cuong_Huynh","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/273615451062279-1442246415718_Q64\/Cuong_Huynh.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Deakin University"}},{"@context":"https:\/\/schema.org\/","@type":"Person","name":"Anneke Veenstra","url":"https:\/\/www.researchgate.net\/profile\/Anneke_Veenstra","image":"https:\/\/i1.rgstatic.net\/ii\/profile.image\/324488602816512-1454375519265_Q64\/Anneke_Veenstra.jpg","memberOf":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Deakin University"}}],"publisher":{"@context":"https:\/\/schema.org\/","@type":"Organization","name":"Australian Journal of Zoology","url":"http:\/\/www.publish.csiro.au\/journals\/ajz\/"}}]}</script>
    <!-- TradeDoubler site verification 2966599 -->
<style>.nova-e-icon{width:1em;height:1em;vertical-align:-.1875rem;display:inline-block;overflow:visible}.nova-e-icon{position:relative;fill:currentColor}.nova-e-icon--size-s{width:1rem;height:1rem}.nova-e-icon--size-m{width:1.5rem;height:1.5rem}.nova-e-icon--color-white.nova-e-icon--theme-bare.nova-e-icon--luminosity-low,.nova-e-icon--color-white.nova-e-icon--theme-bare.nova-e-icon--luminosity-medium{color:#fff}.nova-e-text{margin-top:0;color:inherit;font-weight:400;font-family:Roboto,Arial,sans-serif;font-size:.875rem;line-height:1.3;margin-bottom:1.42857em}@media screen and (min-width:0px){.nova-e-text--size-s{font-size:.75rem}.nova-e-text--size-l,.nova-e-text--size-m{font-size:.875rem}.nova-e-text--size-xl{font-size:1.1875rem}.nova-e-text--size-xxl{font-size:1.375rem}.nova-e-text--size-xxxl{font-size:1.5rem}}@media screen and (min-width:992px){.nova-e-text--size-s{font-size:.75rem}.nova-e-text--size-m{font-size:.875rem}.nova-e-text--size-l{font-size:1rem}.nova-e-text--size-xl{font-size:1.375rem}.nova-e-text--size-xxl{font-size:1.75rem}.nova-e-text--size-xxxl{font-size:2rem}}.nova-e-text--clamp{overflow:hidden;white-space:nowrap;text-overflow:ellipsis}.nova-e-text[class*=nova-e-text--clamp-]{overflow:hidden;display:block;display:-webkit-box;text-overflow:ellipsis;text-overflow:-o-ellipsis-lastline;-webkit-box-orient:vertical;text-align:left}@-moz-document url-prefix(""){.nova-e-text[class*=nova-e-text--clamp-]{padding-right:1em;position:relative}.nova-e-text[class*=nova-e-text--clamp-]:before{position:absolute;right:0;bottom:0}.nova-e-text[class*=nova-e-text--clamp-]:after{position:absolute;right:0;width:1em;background:#fff}}.nova-e-text--size-s{margin-bottom:1.25em}.nova-e-text--size-s.nova-e-text--clamp-2{max-height:2.6em;-webkit-line-clamp:2}@-moz-document url-prefix(""){.nova-e-text--size-s.nova-e-text--clamp-2:before{content:"\2026"}.nova-e-text--size-s.nova-e-text--clamp-2:after{content:"";height:1.3em}}.nova-e-text--size-s.nova-e-text--clamp-3{max-height:3.9em;-webkit-line-clamp:3}@-moz-document url-prefix(""){.nova-e-text--size-s.nova-e-text--clamp-3:before{content:"\2026"}.nova-e-text--size-s.nova-e-text--clamp-3:after{content:"";height:1.3em}}.nova-e-text--size-m.nova-e-text--clamp-2{max-height:2.6em;-webkit-line-clamp:2}@-moz-document url-prefix(""){.nova-e-text--size-m.nova-e-text--clamp-2:before{content:"\2026"}.nova-e-text--size-m.nova-e-text--clamp-2:after{content:"";height:1.3em}}.nova-e-text--size-m.nova-e-text--clamp-3{max-height:3.9em;-webkit-line-clamp:3}@-moz-document url-prefix(""){.nova-e-text--size-m.nova-e-text--clamp-3:before{content:"\2026"}.nova-e-text--size-m.nova-e-text--clamp-3:after{content:"";height:1.3em}}.nova-e-text--size-l{margin-bottom:1.25em}.nova-e-text--size-l.nova-e-text--clamp-2{max-height:2.6em;-webkit-line-clamp:2}@-moz-document url-prefix(""){.nova-e-text--size-l.nova-e-text--clamp-2:before{content:"\2026"}.nova-e-text--size-l.nova-e-text--clamp-2:after{content:"";height:1.3em}}.nova-e-text--size-l.nova-e-text--clamp-3{max-height:3.9em;-webkit-line-clamp:3}@-moz-document url-prefix(""){.nova-e-text--size-l.nova-e-text--clamp-3:before{content:"\2026"}.nova-e-text--size-l.nova-e-text--clamp-3:after{content:"";height:1.3em}}.nova-e-text--size-xl{line-height:1.2;margin-bottom:.90909em}.nova-e-text--size-xl.nova-e-text--clamp-2{max-height:2.4em;-webkit-line-clamp:2}@-moz-document url-prefix(""){.nova-e-text--size-xl.nova-e-text--clamp-2:before{content:"\2026"}.nova-e-text--size-xl.nova-e-text--clamp-2:after{content:"";height:1.2em}}.nova-e-text--size-xl.nova-e-text--clamp-3{max-height:3.6em;-webkit-line-clamp:3}@-moz-document url-prefix(""){.nova-e-text--size-xl.nova-e-text--clamp-3:before{content:"\2026"}.nova-e-text--size-xl.nova-e-text--clamp-3:after{content:"";height:1.2em}}.nova-e-text--size-xxl{line-height:1.2;margin-bottom:.89286em}.nova-e-text--size-xxl.nova-e-text--clamp-2{max-height:2.4em;-webkit-line-clamp:2}@-moz-document url-prefix(""){.nova-e-text--size-xxl.nova-e-text--clamp-2:before{content:"\2026"}.nova-e-text--size-xxl.nova-e-text--clamp-2:after{content:"";height:1.2em}}.nova-e-text--size-xxl.nova-e-text--clamp-3{max-height:3.6em;-webkit-line-clamp:3}@-moz-document url-prefix(""){.nova-e-text--size-xxl.nova-e-text--clamp-3:before{content:"\2026"}.nova-e-text--size-xxl.nova-e-text--clamp-3:after{content:"";height:1.2em}}.nova-e-text--size-xxxl{line-height:1.2;margin-bottom:.78125em;font-weight:300}.nova-e-text--size-xxxl.nova-e-text--clamp-2{max-height:2.4em;-webkit-line-clamp:2}@-moz-document url-prefix(""){.nova-e-text--size-xxxl.nova-e-text--clamp-2:before{content:"\2026"}.nova-e-text--size-xxxl.nova-e-text--clamp-2:after{content:"";height:1.2em}}.nova-e-text--size-xxxl.nova-e-text--clamp-3{max-height:3.6em;-webkit-line-clamp:3}@-moz-document url-prefix(""){.nova-e-text--size-xxxl.nova-e-text--clamp-3:before{content:"\2026"}.nova-e-text--size-xxxl.nova-e-text--clamp-3:after{content:"";height:1.2em}}.nova-e-text--spacing-none{margin-bottom:0}.nova-e-text--spacing-xxs{margin-bottom:5px}.nova-e-text--spacing-xs{margin-bottom:10px}.nova-e-text--spacing-s{margin-bottom:15px}.nova-e-text--spacing-m{margin-bottom:20px}.nova-e-text--spacing-l{margin-bottom:25px}.nova-e-text--color-white,.nova-e-text--color-white:hover{color:#fff}.nova-e-text--color-grey-100,.nova-e-text--color-grey-100:hover{color:#f4f4f4}.nova-e-text--color-grey-500,.nova-e-text--color-grey-500:hover{color:#888}.nova-e-text--color-grey-700,.nova-e-text--color-grey-700:hover{color:#555}.nova-e-text--color-grey-900,.nova-e-text--color-grey-900:hover{color:#111}.nova-e-text--color-inherit{color:inherit}.nova-c-button-group{display:-webkit-inline-box;display:-ms-inline-flexbox;display:inline-flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center;border-collapse:separate}.nova-c-button-group--gutter-m{margin-left:-20px;margin-bottom:-20px}.nova-c-button-group--gutter-m>.nova-c-button-group__item{margin-left:20px;margin-bottom:20px}.nova-c-button-group__item{-ms-flex-item-align:center;align-self:center;position:relative}.nova-c-button-group__item:empty{display:none!important}.nova-c-button-group__item:hover{z-index:1}@-webkit-keyframes fadeFocus{to{-webkit-box-shadow:0 0 0 1px transparent;box-shadow:0 0 0 1px transparent}}@keyframes fadeFocus{to{-webkit-box-shadow:0 0 0 1px transparent;box-shadow:0 0 0 1px transparent}}@-webkit-keyframes fadeFocusBare{to{-webkit-box-shadow:0 2px 0 0 transparent;box-shadow:0 2px 0 0 transparent;background:transparent}}@keyframes fadeFocusBare{to{-webkit-box-shadow:0 2px 0 0 transparent;box-shadow:0 2px 0 0 transparent;background:transparent}}button::-moz-focus-inner,input[type=button]::-moz-focus-inner{border:0;padding:0;margin-top:-.4px;margin-bottom:-.4px}.nova-c-button{-webkit-box-sizing:border-box;box-sizing:border-box;font-family:Roboto,Arial,sans-serif;border:1px solid transparent;outline:none;background-color:transparent;cursor:pointer;text-decoration:none;position:relative;text-align:center;line-height:1;overflow:hidden;margin:0;vertical-align:top;display:inline-block;white-space:nowrap;-webkit-transition:.1s linear;transition:.1s linear}.nova-c-button *,.nova-c-button :after,.nova-c-button :before{-webkit-box-sizing:inherit;box-sizing:inherit}.nova-c-button__icon{font-style:normal;vertical-align:middle;pointer-events:none;display:inline-block;text-align:center;fill:currentColor;-webkit-transition:opacity .05s linear,-webkit-transform .05s linear;transition:opacity .05s linear,-webkit-transform .05s linear;transition:transform .05s linear,opacity .05s linear;transition:transform .05s linear,opacity .05s linear,-webkit-transform .05s linear;min-height:1.3em;position:relative}.nova-c-button__icon+.nova-c-button__label{margin-left:.6em}.nova-c-button__label{-webkit-transition:opacity .05s linear,-webkit-transform .05s linear;transition:opacity .05s linear,-webkit-transform .05s linear;transition:transform .05s linear,opacity .05s linear;transition:transform .05s linear,opacity .05s linear,-webkit-transform .05s linear;display:inline-block;vertical-align:middle;position:relative}.nova-c-button__label+.nova-c-button__icon{margin-left:.6em}.nova-c-button--align-left{text-align:left}.nova-c-button--align-center{text-align:center}.nova-c-button--radius-m{border-radius:2px}.nova-c-button--size-xs{font-size:.75rem;padding:.41667em .5em}.nova-c-button--size-xs .nova-c-button__label{line-height:1.33333}.nova-c-button--size-xs .nova-c-button__icon{min-height:1.33333em}.nova-c-button--size-s{font-size:.875rem;padding:.5em 1em}.nova-c-button--size-s .nova-c-button__label{line-height:1.28571}.nova-c-button--size-s .nova-c-button__icon{min-height:1.28571em}.nova-c-button--size-m{font-size:.875rem;padding:.78571em 2em}.nova-c-button--size-m .nova-c-button__label{line-height:1.28571}.nova-c-button--size-m .nova-c-button__icon{min-height:1.28571em}.nova-c-button--size-xs .nova-c-button__icon{width:1em;height:1em}.nova-c-button:disabled,.nova-c-button[disabled]{cursor:default;pointer-events:none}.nova-c-button:disabled .nova-c-button__label,.nova-c-button:disabled:hover,.nova-c-button[disabled] .nova-c-button__label,.nova-c-button[disabled]:hover{text-decoration:none}.nova-c-button--color-blue.nova-c-button--theme-solid{background-color:#0080ff;border-color:#0080ff;color:#fff}@supports (display:block){.nova-c-button--color-blue.nova-c-button--theme-solid:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,128,255,.4);box-shadow:0 0 0 3px rgba(0,128,255,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-blue.nova-c-button--theme-solid:active,.nova-c-button--color-blue.nova-c-button--theme-solid:hover{background-color:#034aa6;border-color:#034aa6}.nova-c-button--color-blue.nova-c-button--theme-solid[disabled]{background-color:#bbb;border-color:#bbb}.nova-c-button--color-blue.nova-c-button--theme-solid.is-selected{background-color:#034aa6;border-color:#034aa6}@supports (display:block){.nova-c-button--color-blue.nova-c-button--theme-solid.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(3,74,166,.4);box-shadow:0 0 0 3px rgba(3,74,166,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-blue.nova-c-button--theme-ghost{border-color:#0080ff;color:#0080ff}@supports (display:block){.nova-c-button--color-blue.nova-c-button--theme-ghost:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,128,255,.4);box-shadow:0 0 0 3px rgba(0,128,255,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-blue.nova-c-button--theme-ghost:active,.nova-c-button--color-blue.nova-c-button--theme-ghost:hover{background-color:#0080ff;border-color:#0080ff;color:#fff}.nova-c-button--color-blue.nova-c-button--theme-ghost[disabled]{border-color:#bbb;color:#bbb;background-color:transparent}.nova-c-button--color-blue.nova-c-button--theme-ghost.is-selected{background-color:#0080ff;border-color:#0080ff;color:#fff}@supports (display:block){.nova-c-button--color-blue.nova-c-button--theme-ghost.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,128,255,.4);box-shadow:0 0 0 3px rgba(0,128,255,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-blue.nova-c-button--theme-bare{color:#0080ff}@supports (display:block){.nova-c-button--color-blue.nova-c-button--theme-bare:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,128,255,.1);box-shadow:0 0 0 3px rgba(0,128,255,.1);background:rgba(0,128,255,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-blue.nova-c-button--theme-bare:active,.nova-c-button--color-blue.nova-c-button--theme-bare:hover{color:#034aa6}.nova-c-button--color-blue.nova-c-button--theme-bare:active .nova-c-button__label,.nova-c-button--color-blue.nova-c-button--theme-bare:hover .nova-c-button__label{text-decoration:underline}.nova-c-button--color-blue.nova-c-button--theme-bare[disabled]{color:#bbb}.nova-c-button--color-blue.nova-c-button--theme-bare[disabled] .nova-c-button__label{text-decoration:none}.nova-c-button--color-blue.nova-c-button--theme-bare.is-selected{color:#034aa6}@supports (display:block){.nova-c-button--color-blue.nova-c-button--theme-bare.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(3,74,166,.1);box-shadow:0 0 0 3px rgba(3,74,166,.1);background:rgba(3,74,166,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-blue.nova-c-button--theme-bare.is-selected .nova-c-button__label{text-decoration:underline}.nova-c-button--color-grey.nova-c-button--theme-solid{background-color:#888;border-color:#888;color:#fff}@supports (display:block){.nova-c-button--color-grey.nova-c-button--theme-solid:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,53.3%,.4);box-shadow:0 0 0 3px hsla(0,0%,53.3%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-grey.nova-c-button--theme-solid:active,.nova-c-button--color-grey.nova-c-button--theme-solid:hover{background-color:#555;border-color:#555}.nova-c-button--color-grey.nova-c-button--theme-solid[disabled]{background-color:#bbb;border-color:#bbb}.nova-c-button--color-grey.nova-c-button--theme-solid.is-selected{background-color:#555;border-color:#555}@supports (display:block){.nova-c-button--color-grey.nova-c-button--theme-solid.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(85,85,85,.4);box-shadow:0 0 0 3px rgba(85,85,85,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-grey.nova-c-button--theme-ghost{border-color:#888;color:#888}@supports (display:block){.nova-c-button--color-grey.nova-c-button--theme-ghost:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,53.3%,.4);box-shadow:0 0 0 3px hsla(0,0%,53.3%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-grey.nova-c-button--theme-ghost:active,.nova-c-button--color-grey.nova-c-button--theme-ghost:hover{background-color:#888;border-color:#888;color:#fff}.nova-c-button--color-grey.nova-c-button--theme-ghost[disabled]{border-color:#bbb;color:#bbb;background-color:transparent}.nova-c-button--color-grey.nova-c-button--theme-ghost.is-selected{background-color:#888;border-color:#888;color:#fff}@supports (display:block){.nova-c-button--color-grey.nova-c-button--theme-ghost.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,53.3%,.4);box-shadow:0 0 0 3px hsla(0,0%,53.3%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-grey.nova-c-button--theme-bare{color:#888}@supports (display:block){.nova-c-button--color-grey.nova-c-button--theme-bare:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,53.3%,.1);box-shadow:0 0 0 3px hsla(0,0%,53.3%,.1);background:hsla(0,0%,53.3%,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-grey.nova-c-button--theme-bare:active,.nova-c-button--color-grey.nova-c-button--theme-bare:hover{color:#555}.nova-c-button--color-grey.nova-c-button--theme-bare:active .nova-c-button__label,.nova-c-button--color-grey.nova-c-button--theme-bare:hover .nova-c-button__label{text-decoration:underline}.nova-c-button--color-grey.nova-c-button--theme-bare[disabled]{color:#bbb}.nova-c-button--color-grey.nova-c-button--theme-bare[disabled] .nova-c-button__label{text-decoration:none}.nova-c-button--color-grey.nova-c-button--theme-bare.is-selected{color:#555}@supports (display:block){.nova-c-button--color-grey.nova-c-button--theme-bare.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(85,85,85,.1);box-shadow:0 0 0 3px rgba(85,85,85,.1);background:rgba(85,85,85,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-grey.nova-c-button--theme-bare.is-selected .nova-c-button__label{text-decoration:underline}.nova-c-button--color-green.nova-c-button--theme-solid{background-color:#00a39e;border-color:#00a39e;color:#fff}@supports (display:block){.nova-c-button--color-green.nova-c-button--theme-solid:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,163,158,.4);box-shadow:0 0 0 3px rgba(0,163,158,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-green.nova-c-button--theme-solid:active,.nova-c-button--color-green.nova-c-button--theme-solid:hover{background-color:#00444d;border-color:#00444d}.nova-c-button--color-green.nova-c-button--theme-solid[disabled]{background-color:#bbb;border-color:#bbb}.nova-c-button--color-green.nova-c-button--theme-solid.is-selected{background-color:#00444d;border-color:#00444d}@supports (display:block){.nova-c-button--color-green.nova-c-button--theme-solid.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,68,77,.4);box-shadow:0 0 0 3px rgba(0,68,77,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-green.nova-c-button--theme-ghost{border-color:#00a39e;color:#00a39e}@supports (display:block){.nova-c-button--color-green.nova-c-button--theme-ghost:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,163,158,.4);box-shadow:0 0 0 3px rgba(0,163,158,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-green.nova-c-button--theme-ghost:active,.nova-c-button--color-green.nova-c-button--theme-ghost:hover{background-color:#00a39e;border-color:#00a39e;color:#fff}.nova-c-button--color-green.nova-c-button--theme-ghost[disabled]{border-color:#bbb;color:#bbb;background-color:transparent}.nova-c-button--color-green.nova-c-button--theme-ghost.is-selected{background-color:#00a39e;border-color:#00a39e;color:#fff}@supports (display:block){.nova-c-button--color-green.nova-c-button--theme-ghost.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,163,158,.4);box-shadow:0 0 0 3px rgba(0,163,158,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-green.nova-c-button--theme-bare{color:#00a39e}@supports (display:block){.nova-c-button--color-green.nova-c-button--theme-bare:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,163,158,.1);box-shadow:0 0 0 3px rgba(0,163,158,.1);background:rgba(0,163,158,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-green.nova-c-button--theme-bare:active,.nova-c-button--color-green.nova-c-button--theme-bare:hover{color:#00444d}.nova-c-button--color-green.nova-c-button--theme-bare:active .nova-c-button__label,.nova-c-button--color-green.nova-c-button--theme-bare:hover .nova-c-button__label{text-decoration:underline}.nova-c-button--color-green.nova-c-button--theme-bare[disabled]{color:#bbb}.nova-c-button--color-green.nova-c-button--theme-bare[disabled] .nova-c-button__label{text-decoration:none}.nova-c-button--color-green.nova-c-button--theme-bare.is-selected{color:#00444d}@supports (display:block){.nova-c-button--color-green.nova-c-button--theme-bare.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px rgba(0,68,77,.1);box-shadow:0 0 0 3px rgba(0,68,77,.1);background:rgba(0,68,77,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-green.nova-c-button--theme-bare.is-selected .nova-c-button__label{text-decoration:underline}.nova-c-button--color-white.nova-c-button--theme-solid{background-color:#fff;border-color:#fff;color:#111}@supports (display:block){.nova-c-button--color-white.nova-c-button--theme-solid:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,100%,.4);box-shadow:0 0 0 3px hsla(0,0%,100%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-white.nova-c-button--theme-solid:active,.nova-c-button--color-white.nova-c-button--theme-solid:hover{background-color:#fff;border-color:#fff}.nova-c-button--color-white.nova-c-button--theme-solid[disabled]{background-color:#bbb;border-color:#bbb}.nova-c-button--color-white.nova-c-button--theme-solid.is-selected{background-color:#fff;border-color:#fff}@supports (display:block){.nova-c-button--color-white.nova-c-button--theme-solid.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,100%,.4);box-shadow:0 0 0 3px hsla(0,0%,100%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-white.nova-c-button--theme-ghost{border-color:#fff;color:#fff}@supports (display:block){.nova-c-button--color-white.nova-c-button--theme-ghost:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,100%,.4);box-shadow:0 0 0 3px hsla(0,0%,100%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-white.nova-c-button--theme-ghost:active,.nova-c-button--color-white.nova-c-button--theme-ghost:hover{background-color:#fff;border-color:#fff;color:#111}.nova-c-button--color-white.nova-c-button--theme-ghost[disabled]{border-color:#bbb;color:#bbb;background-color:transparent}.nova-c-button--color-white.nova-c-button--theme-ghost.is-selected{background-color:#fff;border-color:#fff;color:#fff}@supports (display:block){.nova-c-button--color-white.nova-c-button--theme-ghost.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,100%,.4);box-shadow:0 0 0 3px hsla(0,0%,100%,.4);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocus;animation-name:fadeFocus;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-white.nova-c-button--theme-bare{color:#fff}@supports (display:block){.nova-c-button--color-white.nova-c-button--theme-bare:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,100%,.1);box-shadow:0 0 0 3px hsla(0,0%,100%,.1);background:hsla(0,0%,100%,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-white.nova-c-button--theme-bare:active,.nova-c-button--color-white.nova-c-button--theme-bare:hover{color:#fff}.nova-c-button--color-white.nova-c-button--theme-bare:active .nova-c-button__label,.nova-c-button--color-white.nova-c-button--theme-bare:hover .nova-c-button__label{text-decoration:underline}.nova-c-button--color-white.nova-c-button--theme-bare[disabled]{color:#bbb}.nova-c-button--color-white.nova-c-button--theme-bare[disabled] .nova-c-button__label{text-decoration:none}.nova-c-button--color-white.nova-c-button--theme-bare.is-selected{color:#fff}@supports (display:block){.nova-c-button--color-white.nova-c-button--theme-bare.is-selected:focus-visible{-webkit-box-shadow:0 0 0 3px hsla(0,0%,100%,.1);box-shadow:0 0 0 3px hsla(0,0%,100%,.1);background:hsla(0,0%,100%,.11);-webkit-transition:none;transition:none;-webkit-animation-name:fadeFocusBare;animation-name:fadeFocusBare;-webkit-animation-duration:.3s;animation-duration:.3s;-webkit-animation-timing-function:cubic-bezier(.25,.46,.45,.94);animation-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-animation-fill-mode:forwards;animation-fill-mode:forwards;-webkit-animation-delay:.6s;animation-delay:.6s}}.nova-c-button--color-white.nova-c-button--theme-bare.is-selected .nova-c-button__label{text-decoration:underline}.nova-c-button--theme-solid:hover .nova-c-button__label{text-decoration:none}.nova-c-button--theme-bare{padding:0;border-radius:0}.nova-c-button--theme-bare .nova-c-button__label{margin-top:0;margin-bottom:0}.nova-c-button--theme-bare .nova-c-button__icon{min-height:0}.nova-c-button--width-square{padding-left:0;padding-right:0;-webkit-box-sizing:content-box;box-sizing:content-box}.nova-c-button--width-square.nova-c-button--size-xs{width:2.16667em}.nova-c-button--width-square.nova-c-button--size-s{width:2.28571em}.nova-c-button--width-square.nova-c-button--size-m{width:2.85714em}.nova-c-button--width-square.nova-c-button--theme-bare{width:auto}.nova-c-button--width-full{width:100%}@-webkit-keyframes nova--spinner-rotate{0%{-webkit-transform:rotate(0deg);transform:rotate(0deg)}to{-webkit-transform:rotate(-1turn);transform:rotate(-1turn)}}@keyframes nova--spinner-rotate{0%{-webkit-transform:rotate(0deg);transform:rotate(0deg)}to{-webkit-transform:rotate(-1turn);transform:rotate(-1turn)}}@-webkit-keyframes nova--spinner-stroke{0%{stroke-dashoffset:44}50%{stroke-dashoffset:0}to{stroke-dashoffset:-44}}@keyframes nova--spinner-stroke{0%{stroke-dashoffset:44}50%{stroke-dashoffset:0}to{stroke-dashoffset:-44}}.nova-e-spinner{display:inline-block;width:1.1428571429em;height:1.1428571429em;vertical-align:top;line-height:1}.nova-e-spinner--size-m{width:1.5rem;height:1.5rem}.nova-e-spinner__asset{display:block;width:100%;height:100%;fill:none}.nova-e-spinner__circle{stroke:currentColor;opacity:.4}.nova-e-spinner__stroke{stroke-dasharray:44;stroke-dashoffset:33;-webkit-animation:nova--spinner-stroke 2s cubic-bezier(.455,.03,.515,.955) infinite both,nova--spinner-rotate 1s linear infinite both;animation:nova--spinner-stroke 2s cubic-bezier(.455,.03,.515,.955) infinite both,nova--spinner-rotate 1s linear infinite both;-webkit-transform-origin:center;transform-origin:center;stroke:currentColor}.nova-e-list{padding-left:0;list-style-position:outside;font-weight:400;color:inherit;font-family:Roboto,Arial,sans-serif;font-size:.875rem;margin:0 0 1.42857em 1.2em;line-height:1}.nova-e-list__item{line-height:1.3}.nova-e-list__item:empty{display:none!important}.nova-e-list__item:not(:last-child){margin-bottom:.5em}@media screen and (min-width:0px){.nova-e-list--size-s{font-size:.75rem}.nova-e-list--size-m{font-size:.875rem}}@media screen and (min-width:992px){.nova-e-list--size-s{font-size:.75rem}.nova-e-list--size-m{font-size:.875rem}}.nova-e-list--type-unordered,ul.nova-e-list{list-style-type:disc}ol.nova-e-list{list-style-type:decimal}.nova-e-list--type-bare,.nova-e-list--type-inline,ol.nova-e-list--type-bare,ol.nova-e-list--type-inline,ul.nova-e-list--type-bare,ul.nova-e-list--type-inline{list-style:none;margin-left:0}.nova-e-list--type-inline>.nova-e-list__item,ol.nova-e-list--type-inline>.nova-e-list__item,ul.nova-e-list--type-inline>.nova-e-list__item{display:inline}.nova-e-list--type-inline>.nova-e-list__item:not(:empty)~.nova-e-list__item:not(:empty):before,ol.nova-e-list--type-inline>.nova-e-list__item:not(:empty)~.nova-e-list__item:not(:empty):before,ul.nova-e-list--type-inline>.nova-e-list__item:not(:empty)~.nova-e-list__item:not(:empty):before{content:"\00A0\00B7\0020";vertical-align:middle}.nova-e-list--size-s{margin-bottom:1.25em}.nova-e-list--spacing-none{margin-bottom:0}.nova-e-list--spacing-s{margin-bottom:15px}.lite-page-ad-fallback{vertical-align:middle;padding:20px;border:1px solid #ddd}.lite-page-ad--has-label.lite-page-ad--show-label .rg-ad div{position:relative;display:inline-block}.lite-page-ad--has-label.lite-page-ad--show-label .rg-ad>div:before{position:absolute;text-align:center;top:-12px;width:100%;left:0;content:"Advertisement";color:#777;font-size:.6875rem;line-height:.6875rem}.lite-page-ad__inner{position:relative}.lite-page-ad__fallback{text-align:left;display:none}.lite-page-ad--align-center{text-align:center}.lite-page-ad--align-left{text-align:left}.lite-page-ad--vertical-spacing-xs{margin-top:10px;margin-bottom:10px}.lite-page-ad--vertical-spacing-xl{margin-top:30px;margin-bottom:30px}.lite-page-ad--sticky{position:-webkit-sticky;position:sticky;top:25px}.lite-page-ad--has-label .lite-page-ad__inner{padding-top:10px}.lite-page-ad--layout-break-sidebar{overflow:visible;position:relative;height:0}.lite-page-ad--layout-break-sidebar .lite-page-ad__inner{position:absolute;right:-335px}.lite-page-ad__close{position:absolute;top:10px;right:10px}@media (max-width:819px){.lite-page-ad__close{display:none}}.lite-page-ad--layout-header{margin:0;background-color:inherit}@media (min-width:820px){.lite-page-ad--layout-header{height:122px}}.lite-page-ad--layout-header .lite-page-ad__inner{padding:20px 0 10px;background-color:inherit}@media (min-width:820px){.lite-page-ad--layout-header .lite-page-ad__inner{min-height:91px}.lite-page-ad--layout-header .is-sticky{position:fixed;width:100%;z-index:91;top:0;border-bottom:1px solid #ddd}}.lite-page-ad--empty .lite-page-ad__fallback{display:block}.lite-page--wide .lite-page-ad--layout-break-sidebar .lite-page-ad__inner{right:-331px}.publication .lite-page-ad--sticky{top:190px}@media (min-width:820px){.publication .lite-page-ad .is-sticky{position:fixed;top:0;background-color:#fff}}.publication .lite-page--header-ad-collapsed .lite-page__main{padding-top:20px}.publication .lite-page--header-ad-collapsed .lite-page-ad--sticky{top:70px}.nova-c-card{-webkit-box-shadow:0 0 2px rgba(0,0,0,.18),0 1px 3px rgba(0,0,0,.12);box-shadow:0 0 2px rgba(0,0,0,.18),0 1px 3px rgba(0,0,0,.12);-webkit-box-sizing:border-box;box-sizing:border-box;background:#fff;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;max-width:100%;width:100%}@supports (display:flex){.nova-c-card{width:auto}}.nova-c-card--elevation-none{-webkit-box-shadow:0 0 0 0 transparent;box-shadow:0 0 0 0 transparent;border:1px solid #ddd;border-radius:.125rem}@media screen and (min-width:0px){.nova-c-card--spacing-xs>.nova-c-card__header{padding:20px 10px}.nova-c-card--spacing-xs>.nova-c-card__body{padding:10px}}@media screen and (min-width:0px){.nova-c-card--spacing-m>.nova-c-card__header{padding:20px 15px}.nova-c-card--spacing-m>.nova-c-card__body{padding:15px}}@media screen and (min-width:992px){.nova-c-card--spacing-m>.nova-c-card__body,.nova-c-card--spacing-m>.nova-c-card__header{padding:20px}}.nova-c-card__header{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;border-bottom:1px solid #ddd}.nova-c-card__body{padding:30px;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column}.lite-page__header-search{height:62px}.lite-page__header-search-input-wrapper{position:relative}.lite-page__header-search-input{margin:0;transition:none;border:0;width:390px;max-width:500px;height:34px;line-height:26px;border-radius:.125rem;background-color:#a0ded9;position:relative;padding:0 30px 0 10px;font-size:.875rem}.lite-page__header-search-input::-webkit-input-placeholder{color:#00a39e}.lite-page__header-search-input::-moz-placeholder{color:#00a39e}.lite-page__header-search-input:-ms-input-placeholder{color:#00a39e}.lite-page__header-search-input::-ms-input-placeholder{color:#00a39e}.lite-page__header-search-input::placeholder{color:#00a39e}.lite-page__header-search-input:focus{background-color:#fff}.lite-page__header-search-or{vertical-align:text-bottom}.lite-page__header-search .lite-page__header-search-button{position:absolute;height:100%;width:30px!important;right:0;top:0}.lite-page__header-search :-ms-input-placeholder,.lite-page__header-search ::-webkit-input-placeholder{color:#ddf0ee}@media (max-width:1199px){.lite-page__header-search-input{width:46px}}@media (max-width:991px){.lite-page .lite-page__header-search>.item{display:none}.lite-page .lite-page__header-search>.item:first-child{display:table-cell}.lite-page .lite-page__header-search .lite-page__header-search-button{position:relative;background-color:#00a39e;color:#fff;height:30px}.lite-page .lite-page__header-search .lite-page__header-search-button:hover{background-color:#00444d;color:#fff}.lite-page__header-search{height:40px}.lite-page__header-search-input{display:none}}.nova-o-pack{display:table;width:100%;table-layout:fixed;border-spacing:0}.nova-o-pack>.nova-o-pack__item{display:table-cell;vertical-align:top}.nova-o-pack>.nova-o-pack__item:empty{display:none!important}.nova-o-pack--gutter-m{margin-left:-10px;margin-right:-10px}.nova-o-pack--gutter-m>.nova-o-pack__item{padding-left:10px;padding-right:10px}.nova-o-pack--gutter-l{margin-left:-12.5px;margin-right:-12.5px}.nova-o-pack--gutter-l>.nova-o-pack__item{padding-left:12.5px;padding-right:12.5px}.nova-o-pack--gutter-xl{margin-left:-15px;margin-right:-15px}.nova-o-pack--gutter-xl>.nova-o-pack__item{padding-left:15px;padding-right:15px}.nova-o-pack--width-auto{width:auto}.nova-o-pack--vertical-align-middle>.nova-o-pack__item{vertical-align:middle}.lite-page-tooltip{display:inline-block;position:relative}.lite-page-tooltip:hover .lite-page-tooltip__content{display:inline-block;z-index:7000}.lite-page-tooltip__content{box-shadow:0 1px 6px rgba(0,0,0,.13),0 3px 6px rgba(0,0,0,.09);display:none;position:absolute;background:#fff;padding:10px;min-width:250px}.lite-page-tooltip__content--middle{top:-100%;transform:translate(-50%,-100%);left:50%}.lite-page-tooltip__arrow{position:absolute;overflow:hidden;width:3em;height:1.5em}.lite-page-tooltip__arrow--middle{transform:translate(-50%,10px);left:50%}.lite-page-tooltip__arrow-tip{background:#fff;width:1.5em;height:1.5em;transform:translate(-50%,-50%) rotate(45deg);left:50%}.lite-page-tooltip__arrow-tip,.lite-page__trusted-device-promo{box-shadow:0 1px 6px rgba(0,0,0,.13),0 3px 6px rgba(0,0,0,.09);position:absolute}.lite-page__trusted-device-promo{top:120%;right:25px;min-width:320px;z-index:1000;background-color:#fff;padding:20px;text-align:left}.lite-page__trusted-device-promo:after,.lite-page__trusted-device-promo:before{content:"";position:absolute;right:18px;top:-12px;display:block;width:0;height:0;border-left:12px solid transparent;border-right:12px solid transparent;border-bottom:12px solid #f4f4f4}.lite-page__trusted-device-promo:after{z-index:1001;right:20px;top:-10px;border-width:0 10px 10px;border-bottom-color:#fff}.lite-page__trusted-device-promo-counter{position:absolute;top:20px;left:55px;background-color:#f50}.lite-page__trusted-device-promo-avatar{padding-right:20px}@media (max-width:991px){.lite-page__trusted-device-promo{right:10px}}@media screen and (max-width:399px){.lite-page__trusted-device-promo{max-width:250px;min-width:auto}}.nova-e-badge{outline:none;border:0;font-size:.875em;font-family:Roboto,Arial,sans-serif;font-weight:400;line-height:1.3;min-height:1em;color:#fff;display:inline-block;padding:.16em .5714em;margin:-.16em 0;border-radius:624.9375rem;white-space:nowrap;min-width:1em;text-align:center;text-decoration:none;vertical-align:.125rem}a.nova-e-badge:hover,button.nova-e-badge:hover{text-decoration:underline;cursor:pointer}.nova-e-badge--display-block{display:block}.nova-e-badge--color-green.nova-e-badge--theme-solid.nova-e-badge--luminosity-medium{background-color:#00a39e;color:#fff}.nova-e-badge--color-green.nova-e-badge--theme-solid.nova-e-badge--luminosity-high{background-color:#c5e8e5;color:#007478}.nova-e-badge--color-green.nova-e-badge--theme-ghost.nova-e-badge--luminosity-medium{background-color:transparent;color:#00a39e;-webkit-box-shadow:inset 0 0 0 1px #00a39e;box-shadow:inset 0 0 0 1px #00a39e}.nova-e-badge--color-green.nova-e-badge--theme-ghost.nova-e-badge--luminosity-high{background-color:transparent;color:#a0ded9;-webkit-box-shadow:inset 0 0 0 1px #a0ded9;box-shadow:inset 0 0 0 1px #a0ded9}.nova-e-badge--radius-m{border-radius:2px}.nova-e-badge--size-l.nova-e-badge--theme-ghost,.nova-e-badge--size-l.nova-e-badge--theme-solid{font-size:.875rem;padding-top:.286em;padding-bottom:.286em;margin-top:-.286em;margin-bottom:-.286em}.nova-e-badge--size-l.nova-e-badge--display-block{margin-top:0;margin-bottom:0}.lite-page-modal{position:relative;z-index:100}.lite-page-modal--hidden{display:none}.lite-page-modal__close{margin:10px}.lite-page-modal__close--top-right{position:absolute;top:0;right:0}.lite-page-modal__login{text-align:left}.lite-page-modal__forgot{position:relative}.lite-page-modal__forgot-link{position:absolute;right:0}.lite-page .lite-page-modal .lite-page-modal__header{margin-bottom:0}.nova-c-modal{position:fixed;top:0;left:0;right:0;bottom:0;background-color:hsla(0,0%,95.7%,.93);z-index:9000;overflow:auto}@supports (display:flex){.nova-c-modal{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center}}@media screen and (min-width:768px){.nova-c-modal--position-center{-webkit-box-align:center;-ms-flex-align:center;align-items:center}}.nova-c-modal--width-s>.nova-c-modal__container>.nova-c-modal__window{max-width:34.375rem}.nova-c-modal--color-green .nova-c-modal__header{background:#00a39e;color:#fff}@media screen and (min-width:0px){.nova-c-modal--spacing-xl .nova-c-modal__window{padding:20px}.nova-c-modal--spacing-xl .nova-c-modal__header{padding:20px;margin:-20px -20px 20px}.nova-c-modal--spacing-xl .nova-c-modal__body{padding:20px;margin:-20px}.nova-c-modal--spacing-xl .nova-c-modal__footer{padding:10px 20px;margin:20px -20px -20px}}@media screen and (min-width:768px){.nova-c-modal--spacing-xl .nova-c-modal__window{padding:25px}.nova-c-modal--spacing-xl .nova-c-modal__header{padding:25px;margin:-25px -25px 25px}.nova-c-modal--spacing-xl .nova-c-modal__body{padding:25px;margin:-25px}.nova-c-modal--spacing-xl .nova-c-modal__footer{padding:10px 25px;margin:25px -25px -25px}}@media screen and (min-width:992px){.nova-c-modal--spacing-xl .nova-c-modal__window{padding:30px}.nova-c-modal--spacing-xl .nova-c-modal__header{padding:30px;margin:-30px -30px 30px}.nova-c-modal--spacing-xl .nova-c-modal__body{padding:30px;margin:-30px}.nova-c-modal--spacing-xl .nova-c-modal__footer{padding:10px 30px;margin:30px -30px -30px}}@media screen and (min-width:0px){.nova-c-modal--spacing-xxl .nova-c-modal__window{padding:25px}.nova-c-modal--spacing-xxl .nova-c-modal__header{padding:25px;margin:-25px -25px 25px}.nova-c-modal--spacing-xxl .nova-c-modal__body{padding:25px;margin:-25px}.nova-c-modal--spacing-xxl .nova-c-modal__footer{padding:10px 25px;margin:25px -25px -25px}}@media screen and (min-width:768px){.nova-c-modal--spacing-xxl .nova-c-modal__window{padding:30px}.nova-c-modal--spacing-xxl .nova-c-modal__header{padding:30px;margin:-30px -30px 30px}.nova-c-modal--spacing-xxl .nova-c-modal__body{padding:30px;margin:-30px}.nova-c-modal--spacing-xxl .nova-c-modal__footer{padding:10px 30px;margin:30px -30px -30px}}@media screen and (min-width:992px){.nova-c-modal--spacing-xxl .nova-c-modal__window{padding:40px}.nova-c-modal--spacing-xxl .nova-c-modal__header{padding:40px;margin:-40px -40px 40px}.nova-c-modal--spacing-xxl .nova-c-modal__body{padding:40px;margin:-40px}.nova-c-modal--spacing-xxl .nova-c-modal__footer{padding:10px 40px;margin:40px -40px -40px}}@media screen and (min-width:0px){.nova-c-modal--spacing-xxxl .nova-c-modal__window{padding:30px}.nova-c-modal--spacing-xxxl .nova-c-modal__header{padding:30px;margin:-30px -30px 30px}.nova-c-modal--spacing-xxxl .nova-c-modal__body{padding:30px;margin:-30px}.nova-c-modal--spacing-xxxl .nova-c-modal__footer{padding:10px 30px;margin:30px -30px -30px}}@media screen and (min-width:768px){.nova-c-modal--spacing-xxxl .nova-c-modal__window{padding:40px}.nova-c-modal--spacing-xxxl .nova-c-modal__header{padding:40px;margin:-40px -40px 40px}.nova-c-modal--spacing-xxxl .nova-c-modal__body{padding:40px;margin:-40px}.nova-c-modal--spacing-xxxl .nova-c-modal__footer{padding:10px 40px;margin:40px -40px -40px}}@media screen and (min-width:992px){.nova-c-modal--spacing-xxxl .nova-c-modal__window{padding:60px}.nova-c-modal--spacing-xxxl .nova-c-modal__header{padding:60px;margin:-60px -60px 60px}.nova-c-modal--spacing-xxxl .nova-c-modal__body{padding:60px;margin:-60px}.nova-c-modal--spacing-xxxl .nova-c-modal__footer{padding:10px 60px;margin:60px -60px -60px}}.nova-c-modal__container{max-height:100%;padding:0 10px}.nova-c-modal__container,.nova-c-modal__window{position:relative;width:100%;-webkit-box-sizing:border-box;box-sizing:border-box}.nova-c-modal__window{-webkit-box-shadow:0 3px 10px rgba(0,0,0,.15),0 14px 28px rgba(0,0,0,.15);box-shadow:0 3px 10px rgba(0,0,0,.15),0 14px 28px rgba(0,0,0,.15);background:#fff;margin:10px auto;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column}@media screen and (min-width:768px){.nova-c-modal__window{margin:60px auto}}.nova-c-modal__spinner{position:absolute;top:50%;left:50%;opacity:0;-webkit-transform:translate(-50%,-50%);transform:translate(-50%,-50%);color:#777}.nova-c-modal__header,.nova-c-modal__spinner{-webkit-box-sizing:border-box;box-sizing:border-box}.nova-c-modal__header{background-color:#00a39e;color:#fff;position:relative;padding:30px}.nova-c-modal__header-content{position:relative;z-index:2;width:100%}.nova-c-modal__header-image{position:absolute;top:0;right:0;bottom:0;left:0;z-index:1;background-size:cover;background-position:50% 50%;opacity:.2;-webkit-transition:opacity 1s linear;transition:opacity 1s linear}@media screen and (min-width:0px){.nova-c-modal .nova-c-modal__header--spacing-none{padding:0}.nova-c-modal .nova-c-modal__header--spacing-none:before{margin-top:0;margin-right:0}}.nova-c-modal__body{-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;-webkit-box-sizing:border-box;box-sizing:border-box}.nova-c-modal__body-content{position:relative}.nova-c-modal__footer{min-height:4rem;-webkit-box-sizing:border-box;box-sizing:border-box;background:#fafafa;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row;-webkit-box-pack:end;-ms-flex-pack:end;justify-content:flex-end;-webkit-box-align:center;-ms-flex-align:center;align-items:center;border-top:1px solid #ddd}.nova-l-flex{display:-webkit-box;display:-ms-flexbox;display:flex}.nova-l-flex--gutter-none{margin:0}.nova-l-flex--gutter-none>.nova-l-flex__item{padding:0}.nova-l-flex--gutter-xxs{margin:-2.5px}.nova-l-flex--gutter-xxs>.nova-l-flex__item{padding:2.5px}.nova-l-flex--gutter-xs{margin:-5px}.nova-l-flex--gutter-xs>.nova-l-flex__item{padding:5px}.nova-l-flex--gutter-m{margin:-10px}.nova-l-flex--gutter-m>.nova-l-flex__item{padding:10px}.nova-l-flex--gutter-xxxxl{margin:-50px}.nova-l-flex--gutter-xxxxl>.nova-l-flex__item{padding:50px}@media screen and (min-width:0px){.nova-l-flex--direction-column\@s-up{-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column}.nova-l-flex--direction-row\@s-up{-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row}}@media screen and (min-width:0px){.nova-l-flex--align-items-center\@s-up{-webkit-box-align:center;-ms-flex-align:center;align-items:center}.nova-l-flex--align-items-stretch\@s-up{-webkit-box-align:stretch;-ms-flex-align:stretch;align-items:stretch}}@media screen and (min-width:0px){.nova-l-flex--justify-content-center\@s-up{-webkit-box-pack:center;-ms-flex-pack:center;justify-content:center}.nova-l-flex--justify-content-flex-start\@s-up{-webkit-box-pack:start;-ms-flex-pack:start;justify-content:flex-start}}@media screen and (min-width:0px){.nova-l-flex--wrap-nowrap\@s-up{-ms-flex-wrap:nowrap;flex-wrap:nowrap}.nova-l-flex--wrap-wrap\@s-up{-ms-flex-wrap:wrap;flex-wrap:wrap}}.nova-l-flex__item{-ms-flex-preferred-size:auto;flex-basis:auto}.nova-l-flex__item--grow{-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;min-width:0}.nova-l-flex__item--shrink{-ms-flex-negative:1;flex-shrink:1}@media screen and (min-width:0px){.nova-l-flex__item--align-self-center\@s-up{-ms-flex-item-align:center;align-self:center}}.nova-l-form-group{display:-webkit-box;display:-ms-flexbox;display:flex;-ms-flex-wrap:wrap;flex-wrap:wrap;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;margin-left:-15px;margin-bottom:-15px}.nova-l-form-group__item{-webkit-box-sizing:border-box;box-sizing:border-box;padding-left:15px;margin-bottom:15px}.nova-l-form-group--layout-stack{-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-ms-flex-wrap:nowrap;flex-wrap:nowrap}.nova-l-form-group--gutter-s{margin-left:-15px;margin-bottom:-15px}.nova-l-form-group--gutter-s>.nova-l-form-group__item{padding-left:15px;margin-bottom:15px}.nova-e-label{vertical-align:middle;display:inline-block}.nova-e-label__text{font-weight:700}.nova-c-tooltip{-webkit-box-shadow:0 1px 6px rgba(0,0,0,.13),0 3px 6px rgba(0,0,0,.09);box-shadow:0 1px 6px rgba(0,0,0,.13),0 3px 6px rgba(0,0,0,.09);-webkit-box-sizing:border-box;box-sizing:border-box;position:relative;z-index:7000;max-width:350px;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column}.nova-c-tooltip--spacing-m .nova-c-tooltip__body{padding:20px}.nova-c-tooltip--position-above-left .nova-c-tooltip__arrow{top:100%}.nova-c-tooltip--position-above-left .nova-c-tooltip__arrow-tip{top:0;-webkit-box-shadow:2px 2px 5px rgba(0,0,0,.15);box-shadow:2px 2px 5px rgba(0,0,0,.15)}.nova-c-tooltip--position-above-left .nova-c-tooltip__arrow{left:0}.nova-c-tooltip--color-white.nova-c-tooltip--elevation-none{-webkit-box-shadow:0 0 0 0 transparent;box-shadow:0 0 0 0 transparent;border:1px solid #ddd;border-radius:.125rem}.nova-c-tooltip--color-white.nova-c-tooltip--elevation-none .nova-c-tooltip__arrow-tip{-webkit-box-shadow:0 0 0 0 transparent;box-shadow:0 0 0 0 transparent;border:1px solid #ddd}.nova-c-tooltip--elevation-none,.nova-c-tooltip--elevation-none .nova-c-tooltip__arrow-tip{-webkit-box-shadow:0 0 0 0 transparent;box-shadow:0 0 0 0 transparent}.nova-c-tooltip--color-white.nova-c-tooltip--luminosity-medium{color:#111}.nova-c-tooltip--color-white.nova-c-tooltip--luminosity-medium .nova-c-tooltip__body{background-color:#fff}.nova-c-tooltip--color-white.nova-c-tooltip--luminosity-medium .nova-c-tooltip__arrow{color:#fff}.nova-c-tooltip__arrow{position:absolute;width:48.3px;height:48.3px;overflow:hidden;-webkit-transform-origin:center center;transform-origin:center center;pointer-events:none;display:block}.nova-c-tooltip__arrow-tip{-webkit-box-shadow:2px 2px 14px rgba(0,0,0,.3);box-shadow:2px 2px 14px rgba(0,0,0,.3);position:absolute;left:50%;top:50%;width:13px;height:13px;-webkit-transform:translate(-50%,-50%) rotate(45deg);transform:translate(-50%,-50%) rotate(45deg);background:currentColor;overflow:hidden;-webkit-transform-origin:center center;transform-origin:center center}.nova-c-tooltip__arrow-tip img{position:absolute;width:368.3px;height:auto;display:block;-webkit-transform:rotate(-45deg);transform:rotate(-45deg);-webkit-transform-origin:center center;transform-origin:center center}.nova-c-tooltip__body{padding:10px;-webkit-box-sizing:border-box;box-sizing:border-box;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1}.nova-e-input__ambient{position:absolute;top:0;left:0;right:0;bottom:0;z-index:0;border:1px solid #bbb;border-radius:.125rem;background:#fafafa}.nova-e-input__field.nova-e-input__ambient{z-index:auto;position:static}.nova-e-input__field+.nova-e-input__ambient,.nova-e-input__field.nova-e-input__ambient{border:1px solid #bbb;background:#fafafa}.nova-e-input__field:focus+.nova-e-input__ambient,.nova-e-input__field:focus.nova-e-input__ambient{background:#fff;border:1px solid #0080ff;-webkit-box-shadow:inset 0 0 0 1px #0080ff;box-shadow:inset 0 0 0 1px #0080ff}.nova-e-input__field[disabled]+.nova-e-input__ambient,.nova-e-input__field[disabled].nova-e-input__ambient{border:1px solid #ddd;cursor:not-allowed}.nova-e-input__field[readonly]+.nova-e-input__ambient,.nova-e-input__field[readonly].nova-e-input__ambient{background:#fafafa;border:1px solid #bbb;-webkit-box-shadow:none;box-shadow:none}.nova-e-input__field[maxlength]:focus+.nova-e-input__ambient,.nova-e-input__field[maxlength]:focus.nova-e-input__ambient{padding-right:55px}.nova-e-input__field{position:relative;z-index:1}.nova-e-input__field{font-family:Roboto,Arial,sans-serif;font-weight:400;-webkit-appearance:none;-moz-appearance:none;appearance:none;outline:none;border:0;background:transparent;-webkit-box-sizing:border-box;box-sizing:border-box;padding:0;width:100%}.nova-e-input__field[disabled].nova-e-input__field::-webkit-input-placeholder{color:#bbb}.nova-e-input__field[disabled].nova-e-input__field::-moz-placeholder{color:#bbb}.nova-e-input__field[disabled].nova-e-input__field:-ms-input-placeholder{color:#bbb}.nova-e-input__field[readonly]{color:#333}.nova-e-input__field--size-m{font-size:1rem;line-height:1.25;padding:.625em .5em;height:2.625em}@media screen and (min-width:768px){.nova-e-input__field--size-m{font-size:.875rem;line-height:1.28571;padding:.625em .5em;height:3em}}@-webkit-keyframes LinkFadeFocusBare{to{-webkit-box-shadow:0 2px 0 0 transparent;box-shadow:0 2px 0 0 transparent;background:transparent}}@keyframes LinkFadeFocusBare{to{-webkit-box-shadow:0 2px 0 0 transparent;box-shadow:0 2px 0 0 transparent;background:transparent}}.nova-e-link{color:inherit;outline:none;padding:0;margin:0;font-size:inherit;font-family:inherit;font-weight:inherit;border:0;background:none;text-align:inherit;display:inline}.nova-e-link,.nova-e-link:hover{text-decoration:underline;cursor:pointer}@supports (display:block){.nova-e-link:focus-visible{position:relative}.nova-e-link:focus-visible:before{content:"";position:absolute;top:0;left:0;right:0;bottom:0;display:block;opacity:.11;background:currentColor;-webkit-box-shadow:0 0 0 .1875rem currentColor;box-shadow:0 0 0 .1875rem currentColor;-webkit-animation:LinkFadeFocusBare .3s cubic-bezier(.25,.46,.45,.94) .6s forwards;animation:LinkFadeFocusBare .3s cubic-bezier(.25,.46,.45,.94) .6s forwards}}.nova-e-link--theme-decorated,.nova-e-link--theme-decorated:hover{text-decoration:underline}.nova-e-link--theme-bare{text-decoration:none}.nova-e-link--theme-bare:hover{text-decoration:underline}.nova-e-link--theme-silent,.nova-e-link--theme-silent:hover{text-decoration:none}.nova-e-link--color-blue,.nova-e-link--color-blue:hover{color:#0080ff}.nova-e-checkbox{position:relative;font-family:Roboto,Arial,sans-serif;font-size:.875rem;font-weight:400;line-height:1.3;border:1px solid transparent;display:inline-block;padding:0}.nova-e-checkbox[disabled] .nova-e-checkbox__label{color:#bbb;cursor:not-allowed}.nova-e-checkbox[disabled] .nova-e-checkbox__label:before{background-color:#fafafa;border:1px solid #ddd}.nova-e-checkbox__input{opacity:0;z-index:-1;position:absolute}.nova-e-checkbox__input:focus+.nova-e-checkbox__label:before{background-color:#fafafa;border:1px solid #0080ff}.nova-e-checkbox__input:checked+.nova-e-checkbox__label:before,.nova-e-checkbox__input:checked:focus+.nova-e-checkbox__label:before{background-color:#0080ff;background-image:url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTIiIGhlaWdodD0iOSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJNMTAuMDM3IDBMNC4yMjkgNi44MDcgMS44ODMgMy40NjEgMSA0LjM0NWwzLjIzIDQuMjNMMTAuOTIuODgzeiIgc3Ryb2tlPSIjRkZGIiBmaWxsPSIjRkZGIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz48L3N2Zz4=");background-position:50% 50%;background-repeat:no-repeat;border:1px solid #0080ff}.nova-e-checkbox__input:checked:focus+.nova-e-checkbox__label:before{-webkit-box-shadow:0 0 0 3px #d7e8f5;box-shadow:0 0 0 3px #d7e8f5}.nova-e-checkbox__input:indeterminate:checked+.nova-e-checkbox__label:before,.nova-e-checkbox__input[indeterminate]:checked+.nova-e-checkbox__label:before{background-color:#0080ff;background-image:url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIGhlaWdodD0iMTYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTMgN2gxMHYySDN6IiBmaWxsPSIjRkZGIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz48L3N2Zz4=");background-position:50% 50%;background-repeat:no-repeat;border:1px solid #0080ff}.nova-e-checkbox__label{cursor:pointer;position:relative;color:#111;-webkit-box-sizing:border-box;box-sizing:border-box;display:inline-block;min-height:.75rem}.nova-e-checkbox__label:before{content:"";display:inline-block;width:14px;height:14px;vertical-align:-.1875rem;background-color:#fafafa;border:1px solid #bbb;border-radius:.125rem}.lite-page__header{background-color:#0cb;position:absolute;top:0;width:100%;height:62px}.lite-page__header-wrapper{height:100%}.lite-page__header-logo-link{display:block;height:100%;line-height:62px;vertical-align:middle;padding:0 20px 0 30px;margin-right:10px}.lite-page__header-logo-link .brand{vertical-align:text-bottom}.lite-page__header-links{text-align:right;height:100%}.lite-page__header-links li{display:inline-block;height:100%}.lite-page__header-link{display:block;padding:0 30px;height:100%;line-height:62px;vertical-align:middle;font-weight:700;background-color:#0cb;color:#fff;border-left:1px solid #00a39e;text-decoration:none;font-size:.875rem;cursor:pointer}.lite-page__header-link:hover{background-color:#00a39e}.lite-page__header-login{position:relative;height:100%}.lite-page__header-login-container{position:absolute;right:0;width:100vw;max-width:310px;max-height:580px;z-index:1000;padding:30px;background-color:#fff;box-shadow:0 0 2px rgba(0,0,0,.18),0 1px 3px rgba(0,0,0,.12);box-sizing:border-box;text-align:left}.lite-page__header-login-item{margin-bottom:10px;position:relative}.lite-page__header-login-checkbox{padding:5px 0 10px;font-size:.75rem}.lite-page__header-login-label{color:#555;font-size:.875rem}.lite-page__header-login-forgot{position:absolute;right:0;top:2px}@media (max-width:1398px){.lite-page__header-link{padding:0 15px}}@media (max-width:1199px){.lite-page__header-logo-link{padding:0 20px}}@media (max-width:991px){.lite-page__header{height:40px}.lite-page__header-links{position:absolute;right:0;top:0}.lite-page__header-link{border:0;padding:0 15px;line-height:40px;font-weight:400}.lite-page__header-logo-link{line-height:40px;padding:0 10px 0 20px}.lite-page__header-logo-link .brand{width:100px}}@media (max-width:767px){.lite-page__header-link,.lite-page__header-logo-link{padding:0 10px}}@media screen and (max-width:767px){.lite-page__footer{padding:40px 10px 0}}@media screen and (min-width:768px) and (max-width:991px){.lite-page__footer{padding:40px 20px 0}}@media screen and (min-width:992px){.lite-page__footer{padding-top:40px;margin:0 auto;max-width:980px}}.lite-page__footer-index{border-top:1px solid #ddd;padding:40px 0}.lite-page__footer-index-title{font-weight:700;margin-top:10px;margin-bottom:15px;color:#111}.lite-page__footer-index-column{color:#555;font-size:.875rem}.lite-page__footer-native-app{margin-bottom:20px}.lite-page__footer-ad-slot{min-width:308px}@media screen and (max-width:767px){.lite-page__footer-ad-slot{display:none}}.lite-page__footer-base-right{text-align:right}.lite-page__footer .constrained-box-sizing{box-sizing:border-box;max-width:100%}@media screen and (min-width:992px){.lite-page--wide .lite-page__footer{max-width:1346px}}.nova-o-stack{list-style:none;margin:0;padding:0}.nova-o-stack__item{display:block;margin:0;padding:0}.nova-o-stack__item:empty{display:none!important;display:none}.nova-o-stack--show-divider>.nova-o-stack__item:not(:last-child){border-bottom:1px solid hsla(0,0%,86.7%,.5)}@media screen and (min-width:0px){.nova-o-stack--spacing-none>.nova-o-stack__item{padding-left:0;padding-right:0}}@media screen and (min-width:0px){.nova-o-stack--gutter-xs>.nova-o-stack__item,.nova-o-stack--gutter-xs>.nova-o-stack__item:not(:empty)~.nova-o-stack__item{padding-top:5px}.nova-o-stack--gutter-xs>.nova-o-stack__item{padding-bottom:5px}}@media screen and (min-width:0px){.nova-o-stack--gutter-s>.nova-o-stack__item,.nova-o-stack--gutter-s>.nova-o-stack__item:not(:empty)~.nova-o-stack__item{padding-top:7.5px}.nova-o-stack--gutter-s>.nova-o-stack__item{padding-bottom:7.5px}}@media screen and (min-width:0px){.nova-o-stack--gutter-m>.nova-o-stack__item,.nova-o-stack--gutter-m>.nova-o-stack__item:not(:empty)~.nova-o-stack__item{padding-top:7.5px}.nova-o-stack--gutter-m>.nova-o-stack__item{padding-bottom:7.5px}}@media screen and (min-width:992px){.nova-o-stack--gutter-m>.nova-o-stack__item,.nova-o-stack--gutter-m>.nova-o-stack__item:not(:empty)~.nova-o-stack__item{padding-top:10px}.nova-o-stack--gutter-m>.nova-o-stack__item{padding-bottom:10px}}.nova-o-stack--no-gutter-outside>.nova-o-stack__item:first-child,.nova-o-stack--no-gutter-outside>.nova-o-stack__item:not(:empty){padding-top:0}.nova-o-stack--no-gutter-outside>.nova-o-stack__item:last-child{padding-bottom:0}.lite-cookie-consent{display:block;text-align:center;min-height:38px;background:#333;line-height:24px;padding:5px 20px;box-sizing:border-box}.lite-cookie-consent__description{margin-right:20px;display:inline-block;max-width:80%}.lite-cookie-consent--hidden{display:none}.lite-cookie-consent-modal__header{position:absolute;top:-40px;left:-40px;right:-40px;background:#00a39e;color:#fff;padding:30px 40px}.lite-cookie-consent-modal__copy{padding-top:80px}@media (max-width:991px){.lite-cookie-consent-modal__header{top:-30px;left:-30px;right:-30px;padding:20px 30px}.lite-cookie-consent-modal__copy{padding-top:60px}}@media (max-width:767px){.lite-cookie-consent-modal__header{top:-25px;left:-25px;right:-25px;padding:20px 30px;font-size:1rem}.lite-cookie-consent-modal__copy{padding-top:50px}}:focus{outline:0}abbr[title]{border-bottom:1px dotted}ol,ul{list-style:none;margin:0}fieldset,ol,td,th,ul{padding:0}fieldset{margin:0;border:0}img{vertical-align:middle;border:0;-ms-interpolation-mode:bicubic}html{font-size:100%;-webkit-text-size-adjust:100%;-moz-text-size-adjust:100%;-ms-text-size-adjust:100%;text-size-adjust:100%}a:active,a:focus,a:hover{outline:0}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}button,input,select,textarea{margin:0;font-size:100%;vertical-align:middle;outline:0}button,input{line-height:normal}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;-moz-appearance:button;appearance:button;cursor:pointer}button,input[type=button],input[type=checkbox],input[type=radio],input[type=reset],input[type=submit],label,select{cursor:pointer}input[type=search]{box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none;appearance:none}textarea{overflow:auto;vertical-align:top}blockquote{quotes:none}blockquote:after,blockquote:before{content:none}q{quotes:"" "" "" ""}q:before{content:open-quote}q:after{content:close-quote}input,textarea{box-shadow:none}body,html{height:100%}body{position:relative;margin:0;font-size:.75rem;line-height:1.5;font-family:Roboto,Arial,sans-serif;color:#111;text-align:left;word-wrap:break-word}.lite-page{min-height:100%}@-webkit-keyframes loading{0%{opacity:.4}50%{opacity:1}to{opacity:.4}}@keyframes loading{0%{opacity:.4}50%{opacity:1}to{opacity:.4}}.lite-page{position:relative}.lite-page--lock-view{overflow:hidden!important}.lite-page:before{content:"";display:block;height:62px;background-color:#0cb}@media (max-width:991px){.lite-page:before{height:40px}}.lite-page__main{display:flex;justify-content:center;padding:0 20px}.lite-page__above{margin:20px 0}.lite-page__content{width:100%;flex-grow:1;max-width:585px;margin-right:395px}.lite-page__side{width:365px;flex-shrink:0;margin-left:-365px}.lite-page--wide .lite-page__content{max-width:975px;margin-right:371px}.lite-page--wide .lite-page__side{width:351px;margin-left:-351px}.lite-page .lite-page-center{text-align:center}.lite-page .lite-page-hidden{display:none}.lite-page .lite-page-visible{display:block}@media screen and (max-width:1199px){.lite-page .hide-l{display:none}.lite-page .lite-page__above{margin-top:0}.lite-page .lite-page__content{margin-right:0}.lite-page .lite-page__main{flex-wrap:wrap;padding:0 5px;margin-top:10px}.lite-page .lite-page__side{max-width:585px;width:100%;margin-left:0;margin-top:20px}}@media screen and (max-width:767px){.lite-page .hide-m{display:none}}@media (-ms-high-contrast:none),screen and (-ms-high-contrast:active){.lite-page__content{flex:1 0 100%}}.lite-page .lite-dropdown{position:relative;display:inline-block}.lite-page .lite-dropdown__dropdown-wrapper{top:100%;position:absolute;right:0;margin-top:15px;z-index:7000}.lite-page .lite-dropdown__dropdown-close{display:none;text-align:right}@media screen and (max-width:767px){.lite-page .lite-dropdown__dropdown-wrapper{position:fixed;top:50%;margin-top:-200px;right:20px;left:20px}.lite-page .lite-dropdown__dropdown{width:100%}.lite-page .lite-dropdown__dropdown-close{display:block}}.nova-c-dropdown{-webkit-box-shadow:0 1px 6px rgba(0,0,0,.13),0 3px 6px rgba(0,0,0,.09);box-shadow:0 1px 6px rgba(0,0,0,.13),0 3px 6px rgba(0,0,0,.09);z-index:7000;background:#fff;display:inline-block;min-width:230px;overflow-y:auto;max-height:400px}.nova-c-dropdown__body{position:relative;padding-top:5px;padding-bottom:5px;background:#fff}.nova-c-dropdown__divider{display:block;height:1px;background:#ddd;margin-top:5px;margin-bottom:5px}.nova-c-dropdown__action{position:relative;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-ms-flex-negative:0;flex-shrink:0;width:100%;-webkit-box-sizing:border-box;box-sizing:border-box;padding:.9375rem 1.25rem;border:0;outline:none;cursor:pointer;background:#fff;color:#555;overflow:hidden;text-align:left;text-decoration:none}.nova-c-dropdown__action:before{content:"";position:absolute;background:#bbb;-webkit-transform:translate(0);transform:translate(0);-webkit-transition:.15s cubic-bezier(.455,.03,.515,.955);transition:.15s cubic-bezier(.455,.03,.515,.955);right:100%;top:0;bottom:0;width:2px}.nova-c-dropdown__action:focus,.nova-c-dropdown__action:hover{text-decoration:none}.nova-c-dropdown__action:focus:before,.nova-c-dropdown__action:hover:before{-webkit-transform:translate(100%);transform:translate(100%)}.nova-c-dropdown__action[disabled]{cursor:default;pointer-events:none;color:#bbb}.nova-c-dropdown__label-container{max-width:100%}.nova-e-radio{position:relative;border:1px solid transparent;display:inline-block;padding:0}.nova-e-radio[disabled] .nova-e-radio__label{color:#bbb;cursor:not-allowed}.nova-e-radio[disabled] .nova-e-radio__label:before{background-color:#fafafa;border:1px solid #ddd}.nova-e-radio__input{opacity:0;z-index:-1;position:absolute}.nova-e-radio__input:checked+.nova-e-radio__label:before,.nova-e-radio__input:focus+.nova-e-radio__label:before{border:1px solid #0080ff}.nova-e-radio__input:checked+.nova-e-radio__label:after{display:inline-block}.nova-e-radio__input:checked:focus+.nova-e-radio__label:before{border:1px solid #0080ff;-webkit-box-shadow:0 0 0 3px #d7e8f5;box-shadow:0 0 0 3px #d7e8f5}.nova-e-radio__input:checked:focus+.nova-e-radio__label:after{display:inline-block;background-color:#0080ff}.nova-e-radio__label{font-family:Roboto,Arial,sans-serif;font-size:.875rem;font-weight:400;line-height:1.3;cursor:pointer;position:relative;-webkit-box-sizing:border-box;box-sizing:border-box;color:#111;display:inline-block;min-height:.75rem}.nova-e-radio__label:before{content:"";display:inline-block;width:14px;height:14px;vertical-align:top;background-color:#fafafa;border:1px solid #bbb;border-radius:624.9375rem}.nova-e-radio__label:after{content:"";display:none;width:8px;height:8px;position:absolute;left:0;top:0;background-color:#0080ff;border-radius:624.9375rem;margin:4px}.publication-resources-summary{position:relative;height:46px}.publication-resources-summary:before{content:"";display:block}.publication-resources-summary__to-top{display:none}.publication-resources-summary__inner{position:relative;left:0;top:0;right:0;z-index:90;box-sizing:border-box;padding:0 20px;background:hsla(0,0%,100%,.95);transform:translateZ(0)}.publication-resources-summary__inner.is-sticky{position:fixed;padding-top:5px;top:122px;box-shadow:0 0 2px rgba(0,0,0,.18),0 1px 3px rgba(0,0,0,.12)}@media (max-width:819px){.publication-resources-summary__inner.is-sticky{top:0}}.publication-resources-summary__content{max-width:1346px;height:100%;margin:0 auto}.publication-resources-summary__cta-box{text-align:right}.publication-resources-summary__cta-box-inner{display:inline-block}.publication-resources-summary .publication-resources-summary__see-all:hover .publication-resources-summary__see-all-label{text-decoration:none}.publication-resources-summary .publication-resources-summary__see-all:hover .publication-resources-summary__see-all-label .publication-resources-summary__see-all-count{text-decoration:underline}.publication-resources-summary .lite-dropdown__dropdown-wrapper{margin-top:0;top:70%}@media screen and (max-width:767px){.publication-resources-summary__inner.is-sticky .publication-resources-summary__to-top{display:block}}.lite-page--header-ad-collapsed .publication-resources-summary .is-sticky{top:0}.publication-meta{margin-bottom:20px}.lite-page-avatar{position:relative;overflow:hidden;display:flex;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;align-items:center}.lite-page-avatar--size-xs{width:16px;height:16px}.lite-page-avatar--size-m{width:48px;height:48px}.lite-page-avatar--radius-full,.lite-page-avatar--radius-full .lite-image{border-radius:624.9375rem}.lite-page-avatar .lite-image{width:100%;height:100%}.nova-v-person-list-item{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;position:relative}.nova-v-person-list-item__body{overflow:hidden}.nova-v-person-list-item__footer{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-ms-flex-wrap:wrap-reverse;flex-wrap:wrap-reverse;margin:0 -15px}.nova-v-person-list-item__footer-metrics{margin:10px 15px 0}.nova-v-person-list-item__footer-metrics:empty{display:none!important}.nova-v-person-list-item__align{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.nova-v-person-list-item__align:before{content:"";width:1px;height:48px}.nova-v-person-list-item__align-content{overflow:hidden}.nova-v-person-list-item__meta{color:#777;white-space:nowrap;overflow:hidden;text-overflow:ellipsis}.nova-v-person-list-item__title{font-weight:700;color:#111}.nova-v-person-list-item__stack-item:empty{display:none!important}.nova-v-person-list-item__stack--gutter-s .nova-v-person-list-item__stack-item:not(:empty)~.nova-v-person-list-item__stack-item:not(:empty){margin-top:5px}.publication-author-list{display:flex;flex-flow:wrap}@media screen and (min-width:768px){.publication-author-list__item{box-sizing:border-box;width:50%}}@media screen and (max-width:767px){.publication-author-list__item{width:100%}}@media screen and (min-width:768px){.publication-author-list__item:nth-child(2n){padding-left:10px}}.show-more-less-authors__button--hidden{display:none}.request-fulltext-promo{background-color:#fafafa;padding:25px;margin-top:40px}.request-fulltext-promo__image{display:inline-block}@media screen and (max-width:767px){.request-fulltext-promo__image{display:none}}@media screen and (min-width:768px){.request-fulltext-promo{padding:30px}}.nova-o-media-object{display:-webkit-box;display:-ms-flexbox;display:flex}.nova-o-media-object__item{-webkit-box-sizing:border-box;box-sizing:border-box;-webkit-box-flex:0;-ms-flex:0 0 auto;flex:0 0 auto;min-width:1%}.nova-o-media-object__item--width-full{-webkit-box-flex:1;-ms-flex:1;flex:1;-ms-flex-preferred-size:auto;flex-basis:auto}.nova-o-media-object__item:empty{display:none!important}.nova-o-media-object--gutter-m{margin-left:-10px;margin-right:-10px}.nova-o-media-object--gutter-m>.nova-o-media-object__item{padding-left:10px;padding-right:10px}.nova-o-media-object--vertical-align-middle{-webkit-box-align:center;-ms-flex-align:center;align-items:center}.nova-o-grid{margin:0;padding:0;list-style:none;letter-spacing:-.31em}.nova-o-grid__column{display:inline-block;width:100%;-webkit-box-sizing:border-box;box-sizing:border-box;padding-left:0;vertical-align:top;letter-spacing:normal;text-align:left}.nova-o-grid--vertical-align-top>.nova-o-grid__column{vertical-align:top}.nova-o-grid--horizontal-align-left{text-align:left}.nova-o-grid--gutter-m{margin-left:-10px;margin-right:-10px}.nova-o-grid--gutter-m>.nova-o-grid__column{padding-left:10px;padding-right:10px}@media screen and (min-width:0px){.nova-o-grid__column--width-12\/12\@s-up{width:100%}}@media screen and (min-width:768px){.nova-o-grid__column--width-6\/12\@m-up{width:50%}}.nova-v-citation-item__context{max-width:none;z-index:auto}.nova-e-text-highlight{line-height:inherit;padding:0 .1875rem;color:#000}.nova-e-text-highlight--color-yellow{background-color:#fce3ac}.lite-person-inline-item__image{display:inline-block;line-height:inherit;width:1.1428571429em;height:1.1428571429em;vertical-align:middle;margin-top:-2px}.nova-v-person-inline-item{text-decoration:none;outline:none;border:0;background:none;padding:0;margin:0;font-size:inherit;font-family:inherit;font-weight:inherit;color:inherit}a.nova-v-person-inline-item:hover,button.nova-v-person-inline-item:hover{text-decoration:underline;cursor:pointer}.nova-v-person-inline-item__fullname{display:inline}.nova-v-person-inline-item__fullname:not(:first-child){margin-left:.357em}.nova-v-publication-item{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1}.nova-v-publication-item__body{overflow:hidden}.nova-v-publication-item__footer{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-ms-flex-wrap:wrap-reverse;flex-wrap:wrap-reverse;margin:0 -15px}.nova-v-publication-item__footer-actions{-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;margin:10px 15px 0}.nova-v-publication-item__footer-actions:empty{display:none!important}.nova-v-publication-item__footer-button-group{overflow:hidden}.nova-v-publication-item__meta{display:-webkit-box;display:-ms-flexbox;display:flex;-ms-flex-wrap:wrap;flex-wrap:wrap;-webkit-box-align:center;-ms-flex-align:center;align-items:center;margin-bottom:-5px}.nova-v-publication-item__meta:empty{display:none!important}.nova-v-publication-item__meta-left{margin-bottom:5px;margin-right:5px}.nova-v-publication-item__meta-left:empty{display:none!important}.nova-v-publication-item__meta-right{margin-bottom:5px;-webkit-box-flex:1;-ms-flex:1 0 0px;flex:1 0 0;overflow:hidden;min-width:100%}.nova-v-publication-item__meta-right:empty{display:none!important}@media screen and (min-width:768px){.nova-v-publication-item__meta-right{min-width:auto}.nova-v-publication-item__meta-left:nth-child(n+3)~.nova-v-publication-item__meta-right{min-width:100%}}.nova-v-publication-item__title{font-weight:700;color:#111}.nova-v-publication-item__description{color:#111}.nova-v-publication-item__meta-data{color:#777;white-space:nowrap;overflow:hidden;text-overflow:ellipsis}.nova-v-publication-item__stack-item:empty{display:none!important}.nova-v-publication-item__stack--gutter-m .nova-v-publication-item__stack-item:not(:empty)~.nova-v-publication-item__stack-item:not(:empty){margin-top:5px}@media screen and (min-width:768px){.nova-v-publication-item__stack--gutter-m .nova-v-publication-item__stack-item:not(:empty)~.nova-v-publication-item__stack-item:not(:empty){margin-top:10px}}.nova-v-publication-item__person-list{color:#111}@-webkit-keyframes flash{0%,50%,to{opacity:.4}25%,75%{opacity:1}}@keyframes flash{0%,50%,to{opacity:.4}25%,75%{opacity:1}}.lite-tabs__tab{display:none}.nova-c-nav{background:#fff;height:100%;width:100%;position:relative;overflow:hidden}.nova-c-nav--direction-horizontal{height:3.875rem}.nova-c-nav--direction-horizontal .nova-c-nav__items{-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row;max-height:7.75rem;height:auto}.nova-c-nav--direction-horizontal .nova-c-nav__item{height:3.875rem}.nova-c-nav--direction-horizontal .nova-c-nav__item:after{top:100%;left:.625rem;right:.625rem;height:2px}.nova-c-nav--direction-horizontal .nova-c-nav__item.is-selected:after,.nova-c-nav--direction-horizontal .nova-c-nav__item:focus:after,.nova-c-nav--direction-horizontal .nova-c-nav__item:hover:after{-webkit-transform:translateY(-100%);transform:translateY(-100%)}.nova-c-nav--theme-tabs{background:#fafafa}.nova-c-nav--theme-tabs:before{content:"";background:#ddd;position:absolute}.nova-c-nav--theme-tabs .nova-c-nav__item{position:relative;border-color:hsla(0,0%,86.7%,.25);border-style:solid}.nova-c-nav--theme-tabs .nova-c-nav__item:after{display:none}.nova-c-nav--theme-tabs .nova-c-nav__item:focus{background:#f4f4f4}.nova-c-nav--theme-tabs .nova-c-nav__item.is-selected{background:#fff;border-color:#ddd}.nova-c-nav--theme-tabs .nova-c-nav__plus-overflow-trigger:focus{background:#f4f4f4}.nova-c-nav--theme-tabs.nova-c-nav--direction-horizontal:before{height:.0625rem;width:100%;bottom:0;left:0}.nova-c-nav--theme-tabs.nova-c-nav--direction-horizontal .nova-c-nav__item{border-left-width:1px;border-right-width:1px;border-bottom:1px solid #ddd}.nova-c-nav--theme-tabs.nova-c-nav--direction-horizontal .nova-c-nav__item.is-selected{border-bottom:1px solid #fff}.nova-c-nav--theme-tabs.nova-c-nav--direction-horizontal .nova-c-nav__item:not(:last-child){margin-right:-1px}.nova-c-nav__wrapper{overflow:hidden}.nova-c-nav__items,.nova-c-nav__wrapper{display:-webkit-box;display:-ms-flexbox;display:flex}.nova-c-nav__items{-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;-ms-flex-wrap:wrap;flex-wrap:wrap}.nova-c-nav--theme-tabs .nova-c-nav__items:after{background:-webkit-gradient(linear,left top,right top,from(hsla(0,0%,98%,0)),color-stop(50%,#fafafa));background:linear-gradient(90deg,hsla(0,0%,98%,0),#fafafa 50%)}@media (pointer:coarse){.nova-c-nav__items{display:-webkit-box;display:-ms-flexbox;display:flex;-ms-flex-wrap:nowrap;flex-wrap:nowrap;overflow:scroll}.nova-c-nav__items:after{content:"";position:relative;width:3.75rem;z-index:2;-ms-flex-negative:0;flex-shrink:0;margin:.0625rem 0 .125rem auto}}.nova-c-nav__scroll-overflow{position:absolute;top:0;right:0;bottom:.125rem;width:3.75rem;-webkit-box-sizing:border-box;box-sizing:border-box;color:#111;display:none;z-index:1}.nova-c-nav--theme-tabs .nova-c-nav__scroll-overflow{background:-webkit-gradient(linear,left top,right top,from(hsla(0,0%,98%,0)),color-stop(50%,#fafafa));background:linear-gradient(90deg,hsla(0,0%,98%,0),#fafafa 50%)}@media (pointer:coarse){.nova-c-nav__scroll-overflow{display:-webkit-box;display:-ms-flexbox;display:flex}}.nova-c-nav__scroll-overflow-trigger{color:#555;white-space:nowrap;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;overflow:hidden;background:none;border:0;text-decoration:none;cursor:pointer;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;padding:0 0 0 1.875rem}.nova-c-nav__scroll-overflow-trigger:hover{color:#111}.nova-c-nav__plus-overflow-mount{position:absolute;bottom:0;right:0}.nova-c-nav__plus-overflow{display:-webkit-box;display:-ms-flexbox;display:flex;width:3.875rem;height:3.875rem;-ms-flex-negative:0;flex-shrink:0;-ms-flex-item-align:end;align-self:flex-end;-webkit-transform:translateY(-100%);transform:translateY(-100%)}@media (pointer:coarse){.nova-c-nav__plus-overflow{display:none}}.nova-c-nav__plus-overflow-trigger{color:#555;white-space:nowrap;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-ms-flex-pack:center;justify-content:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;overflow:hidden;background:none;border:0;text-decoration:none;cursor:pointer;-webkit-box-flex:1;-ms-flex-positive:1;flex-grow:1;padding:0;outline:none}.nova-c-nav--theme-tabs .nova-c-nav__plus-overflow-trigger{border-bottom:1px solid #ddd}.nova-c-nav__plus-overflow-trigger:hover{color:#111}.nova-c-nav__item{padding:.9375rem 1.25rem;-webkit-box-sizing:border-box;box-sizing:border-box;color:#555;white-space:nowrap;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;overflow:hidden;text-align:left;background:none;border:0;-ms-flex-negative:0;flex-shrink:0;margin:0;text-decoration:none;cursor:pointer;outline:none}.nova-c-nav__item:after{content:"";position:absolute;background:#bbb;-webkit-transform:translate(0);transform:translate(0);-webkit-transition:.15s cubic-bezier(.455,.03,.515,.955);transition:.15s cubic-bezier(.455,.03,.515,.955)}.nova-c-nav__item.is-selected,.nova-c-nav__item:hover{color:#111;text-decoration:none;background:none}.nova-c-nav__item:disabled,.nova-c-nav__item[disabled]{cursor:default;pointer-events:none;color:#bbb}.nova-c-nav--direction-horizontal .nova-c-nav__item:last-child{margin-right:-3.875rem}@media (pointer:coarse){.nova-c-nav--direction-horizontal .nova-c-nav__item:last-child{margin-right:-.625rem}}.nova-c-nav--theme-tabs .nova-c-nav__item-label{font-size:1rem}.publication-citations__item{border-top:1px solid #ddd;margin-top:15px;padding-top:15px}.publication-citations__item:first-child{border-top:0;margin-top:0}.publication-citations__item--load-more{border-top:0}@media screen and (min-width:768px){.publication-citations__item{margin-top:25px;padding-top:25px}}.publication-citations__item .publication-citations__item:first-child{padding-top:0}.publication-citations__more{text-align:right}.publication-citations__menubar{margin-left:-1px}@-webkit-keyframes progressValueIdle{0%,to{background-color:#83b7e6}50%{background-color:#0080ff}}@keyframes progressValueIdle{0%,to{background-color:#83b7e6}50%{background-color:#0080ff}}@-webkit-keyframes progressValueIndeterminate{0%{-webkit-transform:translateX(-100%);transform:translateX(-100%)}to{-webkit-transform:translateX(303.0303%);transform:translateX(303.0303%)}}@keyframes progressValueIndeterminate{0%{-webkit-transform:translateX(-100%);transform:translateX(-100%)}to{-webkit-transform:translateX(303.0303%);transform:translateX(303.0303%)}}::-moz-selection{background:rgba(127,255,255,.4)}::selection{background:rgba(127,255,255,.4)}.publication-details__section{margin-bottom:60px}.mobile-catfish-adslot{display:none}@media screen and (max-width:767px){.mobile-catfish-adslot{display:block;position:fixed;top:auto;bottom:0;left:0;right:0;z-index:7001}}@media screen and (max-width:768px){.publication-details__title{font-size:1.375rem;font-weight:400}}@-webkit-keyframes bump{0%{-webkit-transform:scale(1);transform:scale(1)}35%{-webkit-transform:scale(1.2);transform:scale(1.2)}to{-webkit-transform:scale(1);transform:scale(1)}}@keyframes bump{0%{-webkit-transform:scale(1);transform:scale(1)}35%{-webkit-transform:scale(1.2);transform:scale(1.2)}to{-webkit-transform:scale(1);transform:scale(1)}}.publication-bottom{width:100%}.publication-recommendation-item{position:relative;padding:20px 20px 40px;min-height:270px;background-color:#fff;box-shadow:1px 1px 1px 0 rgba(0,0,0,.075)}.publication-recommendation-item__item{margin-bottom:20px}.publication-recommendation-item__item-badges{margin-bottom:10px}.publication-recommendation-item__more{position:absolute;left:20px;bottom:20px}.publication-recommendations{background-color:#fafafa;margin-bottom:20px;padding:0 20px}.publication-recommendations__inner{max-width:955px;padding:40px 0}.publication-recommendations__outer{max-width:1326px;margin:0 auto}.publication-recommendations__items{margin-top:20px;margin-bottom:20px}</style>
<title>Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera | Request PDF</title>
<meta name="description" content="Request PDF | Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera | Bristly millipedes (subclass Penicillata, order Polyxenida) are minute diplopods characterised by uncalcified cuticle and a body covered with... | Find, read and cite all the research you need on ResearchGate"/>
</head><body>
<div class="lite-cookie-consent lite-cookie-consent--hidden js-target-cookie-consent"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-100 lite-cookie-consent__description">We use cookies to make interactions with our website easy and meaningful, to better understand the use of our services, and to tailor advertising. For further information, including about cookie settings, please read our <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-decorated" target="_blank" href="privacy-policy#cookies" rel="noopener">Cookie Policy</a> . By continuing to use this site, you consent to the use of cookies.</div><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-xs nova-c-button--color-white nova-c-button--theme-ghost nova-c-button--width-auto js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-cookie-consent&quot;,&quot;a&quot;:&quot;ac(lite-cookie-consent--hidden)&quot;}]}"><span class="nova-c-button__label">Got it</span></button></div><div id="lite-page" class="lite-page lite-page--wide publication"><div id="js-target-cookie-consent-modal" class="lite-page-modal lite-page-modal--hidden"><span><div class="nova-c-modal nova-c-modal--color-green nova-c-modal--position-center nova-c-modal--spacing-xxl nova-c-modal--width-s lite-cookie-consent-modal" style="z-index:9998"><div class="nova-c-modal__overlay"></div><div class="nova-c-modal__container"><div class="nova-c-modal__window" role="dialog" aria-modal="true" tabindex="-1"><header class="nova-c-modal__header nova-c-modal__header--spacing-none lite-page-modal__header"><div class="nova-c-modal__header-content"></div><div class="nova-c-modal__header-image"></div></header><div class="nova-c-modal__body nova-c-modal__body--spacing-inherit"><div class="nova-c-modal__body-content"><div class="lite-cookie-consent-modal__header"><div class="nova-e-text nova-e-text--size-xxl nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit">We value your privacy</div></div><div class="lite-cookie-consent-modal__copy"><p class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-m nova-e-text--color-inherit">We use cookies to offer you a better experience, personalize content, tailor advertising, provide social media features, and better understand the use of our services.</p><p class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit">To learn more or modify/prevent the use of cookies, see our <a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated gtm-open-cookie-policy-link js-lite-click" target="_blank" rel="noopener" href="cookie-consent-policy" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.gtm-open-cookie-policy-link&quot;,&quot;a&quot;:&quot;tgtm(gtm-ev-cookie-policy)&quot;}]}">Cookie Policy</a> and <a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated gtm-open-privacy-policy-link js-lite-click" target="_blank" rel="noopener" href="privacy-policy" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.gtm-open-privacy-policy-link&quot;,&quot;a&quot;:&quot;tgtm(gtm-ev-privacy-policy)&quot;}]}">Privacy Policy</a>.</p></div></div></div><footer class="nova-c-modal__footer nova-c-modal__footer--align-right nova-c-modal__footer--spacing-inherit"><div class="nova-c-modal__footer-content"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-auto js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;#js-target-cookie-consent-modal&quot;,&quot;a&quot;:&quot;ac(lite-page-modal--hidden)&quot;},{&quot;t&quot;:&quot;window&quot;,&quot;a&quot;:&quot;tcl(cookie-consented)&quot;},{&quot;t&quot;:&quot;.gtm-cookie-consented&quot;,&quot;a&quot;:&quot;tgtm(gtm-ev-cookie-consented)&quot;}]}"><span class="nova-c-button__label gtm-cookie-consented">Accept Cookies</span></button></div></footer><div class="nova-e-spinner nova-e-spinner--size-m nova-e-spinner--color-inherit nova-c-modal__spinner"><div class="nova-e-spinner__container"><svg width="16" height="16" viewBox="0 0 16 16" class="nova-e-spinner__asset"><g stroke-width="2"><path class="nova-e-spinner__circle" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path><path class="nova-e-spinner__stroke" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path></g></svg></div></div></div></div></div></span></div><div id="lite-ad-y65dvdr7jd" class="lite-page-ad lite-page-ad--align-center lite-page-ad--vertical-spacing-xs lite-page-ad--layout-header lite-page-ad--has-label js-lite-sticky"><div class="lite-page-ad__inner js-lite-sticky-target"><button class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-grey nova-c-button--theme-bare nova-c-button--width-square lite-page-ad__close" type="button" id="lite-ad-y65dvdr7jd-close"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#close-circle-s"></use></svg></button><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.querySelector('#lite-ad-y65dvdr7jd-close').onclick = function(e) {document.querySelector('#lite-ad-y65dvdr7jd').style.display = 'none'; RGDom.className.addClass(document.querySelector('#lite-page'), 'lite-page--header-ad-collapsed');};</script><div id="y65dvdr7jd"><rg-ad id="spvr9bih6l" ad-unit="LoggedOut_PublicationWithoutFulltext_Top" ad-unit-path="/83500759/LoggedOut_PublicationWithoutFulltext_Top" ad-location="Top" refresh-timeout="-1" load-on-init="false" collapse="never" class="rg-ad" ad-unit-sizes="[[728,90],[970,90],&quot;fluid&quot;]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="spvr9bih6l" targeting-sequence="1"><rg-ad-size size="[728,90]"></rg-ad-size><rg-ad-size size="[970,90]"></rg-ad-size><rg-ad-size size="&quot;fluid&quot;"></rg-ad-size><rg-ad-sizeset viewport-size="[0,0]" slot-sizes="[]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[300,0]" slot-sizes="[[300,100],[300,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[320,0]" slot-sizes="[[320,100],[320,50],[300,100],[300,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[728,0]" slot-sizes="[[728,90],&quot;fluid&quot;]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[970,0]" slot-sizes="[[970,90],[728,90],&quot;fluid&quot;]"></rg-ad-sizeset></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("spvr9bih6l").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-y65dvdr7jd').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-y65dvdr7jd");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-y65dvdr7jd').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-y65dvdr7jd");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div><div class="lite-page__above"><section class="js-lite-sticky publication-resources-summary" role="navigation"><div class="js-lite-sticky-target publication-resources-summary__inner"><div class="publication-resources-summary__ad-slot"></div><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-none nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up publication-resources-summary__content"><div class="nova-l-flex__item publication-resources-summary__to-top"><button class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-xs nova-c-button--color-grey nova-c-button--theme-bare nova-c-button--width-auto js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;body&quot;,&quot;a&quot;:&quot;st&quot;}]}"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#arrow-up-s"></use></svg><span class="nova-c-button__label">top</span></button></div><div class="nova-l-flex__item hide-m"><div class="nova-o-pack nova-o-pack--gutter-xl nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item"><a class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto publication-resources-summary__see-all js-lite-click" role="button" href="publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera#citations" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-citation&quot;,&quot;a&quot;:&quot;st(64)&quot;,&quot;d&quot;:100},{&quot;t&quot;:&quot;.js-target-citationReferences button.citations&quot;,&quot;a&quot;:&quot;tcl(click)&quot;}]}"><span class="nova-c-button__label publication-resources-summary__see-all-label"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-900">See all </div><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit publication-resources-summary__see-all-count"><strong>2</strong> Citations</div></span></a></div><div class="nova-o-pack__item"><a class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto publication-resources-summary__see-all js-lite-click" role="button" href="publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera#references" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-reference&quot;,&quot;a&quot;:&quot;st(64)&quot;,&quot;d&quot;:100},{&quot;t&quot;:&quot;.js-target-citationReferences button.references&quot;,&quot;a&quot;:&quot;tcl(click)&quot;}]}"><span class="nova-c-button__label publication-resources-summary__see-all-label"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-900">See all </div><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit publication-resources-summary__see-all-count"><strong>66</strong> References</div></span></a></div></div></div><div class="nova-l-flex__item nova-l-flex__item--grow publication-resources-summary__cta-box"><div class="publication-resources-summary__cta-box-inner"><div class="nova-o-pack nova-o-pack--gutter-l nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item hide-m"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto" href="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera/citation/download"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#download-s"></use></svg><span class="nova-c-button__label gtm-download-citation-btn">Download citation</span></a></div><div class="nova-o-pack__item"><div class="lite-dropdown"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto lite-share js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-share-content&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;,&quot;r&quot;:&quot;rco&quot;}]}"><span class="nova-c-button__label">Share <svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium lite-share__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#arrow-down-s"></use></svg></span></button><div class="lite-dropdown__dropdown-wrapper lite-page-hidden js-target-share-content"><div role="menu" class="nova-c-dropdown lite-dropdown__dropdown"><div class="nova-c-dropdown__body"><button type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-share-content&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}" class="nova-c-dropdown__action lite-dropdown__dropdown-close js-lite-click" role="menuitem"><span class="nova-c-dropdown__label-container"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-black nova-e-icon--luminosity-low"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#close-s"></use></svg></div></span></button><span class="nova-c-dropdown__divider lite-dropdown__dropdown-close" aria-hidden="true"></span><a href="http://www.facebook.com/share.php?u=https%3A%2F%2Fwww.researchgate.net%2Fpublication%2F320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera" data-provider-context="facebook" class="nova-c-dropdown__action share-facebook" role="menuitem"><span class="nova-c-dropdown__label-container"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#social-facebook-s"></use></svg> Facebook</div></span></a><a href="http://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.researchgate.net%2Fpublication%2F320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera" data-provider-context="twitter" class="nova-c-dropdown__action share-twitter" role="menuitem"><span class="nova-c-dropdown__label-container"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#social-twitter-s"></use></svg> Twitter</div></span></a><a href="http://www.linkedin.com/shareArticle?mini=true&amp;source=ResearchGate&amp;url=https%3A%2F%2Fwww.researchgate.net%2Fpublication%2F320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera" data-provider-context="linkedin" class="nova-c-dropdown__action share-linkedin" role="menuitem"><span class="nova-c-dropdown__label-container"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#social-linkedin-s"></use></svg> LinkedIn</div></span></a><a href="https://www.reddit.com/submit?url=https%3A%2F%2Fwww.researchgate.net%2Fpublication%2F320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera" data-provider-context="reddit" class="nova-c-dropdown__action share-reddit" role="menuitem"><span class="nova-c-dropdown__label-container"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#social-reddit-s"></use></svg> Reddit</div></span></a></div></div></div></div></div><div class="nova-o-pack__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-auto js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;#login-modal&quot;,&quot;a&quot;:&quot;tc(lite-page-modal--hidden)&quot;,&quot;c&quot;:1},{&quot;t&quot;:&quot;.lite-page&quot;,&quot;a&quot;:&quot;ft(https://www.researchgate.net/lite.publication.PublicationResourcesSummary.requestFulltext.html?publicationUid=320321240&amp;ev=su_requestFulltext)&quot;}]}"><span class="nova-c-button__label gtm-request-fulltext-btn-header">Request full-text</span></button></div></div></div></div></div></div></section></div><main class="lite-page__main" role="main"><section class="lite-page__content"><section class="publication-details__section"><h1 class="nova-e-text nova-e-text--size-xxxl nova-e-text--family-sans-serif nova-e-text--spacing-l nova-e-text--color-grey-900 publication-details__title" itemProp="headline">Phylogenetic relationships of millipedes in the subclass Penicillata (Diplopoda) with a key to the genera</h1><div class="publication-meta"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-xxs nova-e-text--color-grey-700"><span class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-900"><strong><span class="publication-meta__type">Article</span></strong></span><em class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-500">in</em><span><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-bare" href="journal/0022-2933_Journal_of_Natural_History" target="_blank" rel="noopener">Journal of Natural History</a> 51(41-42):2443-2461</span><span>October 2017</span><em class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-500">with</em>77 Reads<div class="lite-page-tooltip"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#info-circle-s"></use></svg><div class="lite-page-tooltip__content lite-page-tooltip__content--middle"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><strong>How we measure &#x27;reads&#x27;</strong></div><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit">A &#x27;read&#x27; is counted each time someone views a publication summary (such as the title, abstract, and list of authors), clicks on a figure, or views or downloads the full-text. <a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="https://explore.researchgate.net/display/support/Reads" target="_blank" rel="noopener">Learn more</a></div><div class="lite-page-tooltip__arrow lite-page-tooltip__arrow--middle"><div class="lite-page-tooltip__arrow-tip"></div></div></div></div></div><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-500">DOI: <a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="https://www.researchgate.net/deref/http%3A%2F%2Fdx.doi.org%2F10.1080%2F00222933.2017.1380241" target="_blank" rel="noopener">10.1080/00222933.2017.1380241</a>  </div><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto" href="https://www.researchgate.net/publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera/citation/download"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#download-s"></use></svg><span class="nova-c-button__label gtm-cite-publication-btn">Cite this publication</span></a></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-bare nova-e-list--spacing-s publication-author-list load-more-authors"><li class="nova-e-list__item publication-author-list__item"><div class="nova-c-card nova-c-card--spacing-xs nova-c-card--elevation-none"><div class="nova-c-card__body nova-c-card__body--spacing-inherit"><div itemprop="author" itemscope="" itemType="http://schema.org/Person" class="nova-v-person-list-item has-image"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xs nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item"><a href="https://www.researchgate.net/profile/Megan_Short3"><picture class="lite-page-avatar lite-page-avatar--size-m lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" data-srcset="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg 1x, https://c5.rgstatic.net/m/437738464651637/images/template/default/profile/profile_default_l.jpg 2x" class="lite-image" alt="Megan Short at Deakin University"/></picture></a></div><div class="nova-l-flex__item nova-l-flex__item--grow nova-v-person-list-item__body"><div class="nova-v-person-list-item__stack nova-v-person-list-item__stack--gutter-s"><div class="nova-v-person-list-item__stack-item"><div class="nova-v-person-list-item__align"><div class="nova-v-person-list-item__align-content"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp nova-v-person-list-item__title" itemProp="name"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Megan_Short3">Megan Short</a></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-person-list-item__meta"><li class="nova-e-list__item nova-v-person-list-item__meta-item"><span><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Megan_Short3"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#rg-score-s"></use></svg>9.12</a></span></li><li class="nova-e-list__item nova-v-person-list-item__meta-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="institution/Deakin_University">Deakin University</a></span></li></ul></div></div></div></div><footer class="nova-v-person-list-item__footer"><div class="nova-v-person-list-item__footer-metrics"></div></footer></div><div class="nova-l-flex__item nova-l-flex__item--shrink"><div class="nova-v-person-list-item__align"><div class="nova-v-person-list-item__align-content"></div></div></div></div><span><meta itemProp="image" content="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg"/><meta itemProp="url" content="https://www.researchgate.net/profile/Megan_Short3"/><span itemProp="memberOf" itemscope="" itemType="http://schema.org/Organization"><meta itemProp="name" content="Deakin University"/></span></span></div></div></div></li><li class="nova-e-list__item publication-author-list__item"><div class="nova-c-card nova-c-card--spacing-xs nova-c-card--elevation-none"><div class="nova-c-card__body nova-c-card__body--spacing-inherit"><div itemprop="author" itemscope="" itemType="http://schema.org/Person" class="nova-v-person-list-item has-image"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xs nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item"><a href="https://www.researchgate.net/profile/Varpu_Vahtera"><picture class="lite-page-avatar lite-page-avatar--size-m lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272488118616066-1441977638089_Q64/Varpu_Vahtera.jpg" data-srcset="https://i1.rgstatic.net/ii/profile.image/272488118616066-1441977638089_Q64/Varpu_Vahtera.jpg 1x, https://i1.rgstatic.net/ii/profile.image/272488118616066-1441977638089_Q128/Varpu_Vahtera.jpg 2x" class="lite-image" alt="Varpu Vahtera at University of Turku"/></picture></a></div><div class="nova-l-flex__item nova-l-flex__item--grow nova-v-person-list-item__body"><div class="nova-v-person-list-item__stack nova-v-person-list-item__stack--gutter-s"><div class="nova-v-person-list-item__stack-item"><div class="nova-v-person-list-item__align"><div class="nova-v-person-list-item__align-content"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp nova-v-person-list-item__title" itemProp="name"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Varpu_Vahtera">Varpu Vahtera</a></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-person-list-item__meta"><li class="nova-e-list__item nova-v-person-list-item__meta-item"><span><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Varpu_Vahtera"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#rg-score-s"></use></svg>25.97</a></span></li><li class="nova-e-list__item nova-v-person-list-item__meta-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="institution/University_of_Turku">University of Turku</a></span></li></ul></div></div></div></div><footer class="nova-v-person-list-item__footer"><div class="nova-v-person-list-item__footer-metrics"></div></footer></div><div class="nova-l-flex__item nova-l-flex__item--shrink"><div class="nova-v-person-list-item__align"><div class="nova-v-person-list-item__align-content"></div></div></div></div><span><meta itemProp="image" content="https://i1.rgstatic.net/ii/profile.image/272488118616066-1441977638089_Q64/Varpu_Vahtera.jpg"/><meta itemProp="url" content="https://www.researchgate.net/profile/Varpu_Vahtera"/><span itemProp="memberOf" itemscope="" itemType="http://schema.org/Organization"><meta itemProp="name" content="University of Turku"/></span></span></div></div></div></li></ul><div class="nova-c-card nova-c-card--spacing-m nova-c-card--elevation-none"><div class="nova-c-card__header nova-c-card__header--spacing-inherit"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-700">Abstract</div></div><div class="nova-c-card__body nova-c-card__body--spacing-inherit"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-auto nova-e-text--color-inherit" itemProp="description">Bristly millipedes (subclass Penicillata, order Polyxenida) are minute diplopods characterised by uncalcified cuticle and a body covered with unique tufts of bristles. The order is found worldwide and comprises less than 200 described species divided into three families, with many of the species and genera being poorly known. The first evolutionary analysis of the order presented here utilises both molecular (COI, 18S rRNA and 16S rRNA) and morphological data to examine monophyly of the families and subfamilies and the evolutionary relationships between them. Maximum likelihood analysis was based on molecular data only, whereas parsimony analyses were based on molecular data as well as combined morphological and molecular data. The results of these analyses with two different optimality criteria were incongruent in many aspects. Unlike parsimony, the likelihood result found strong support for a basal position of the family Synxenidae and separation of the order into two monophyletic clades corresponding to the two superfamilies Synxenoidea, containing the family Synxenidae, and Polyxenoidea, containing the families Polyxenidae and Lophoproctidae. Parsimony results did not support the existence of the two superfamilies. Both analyses resolved the family Synxenidae as monophyletic and Polyxenidae as polyphyletic, whereas the family Lophoproctidae was shown to be paraphyletic in likelihood and monophyletic in parsimony analysis. The subfamilies Monographinae and Polyxeninae were found to be monophyletic in the likelihood tree but parsimony suggested paraphyly of both. The results suggest that further revision of the systematics of the Polyxenida may be necessary. However, a much larger molecular data set will be necessary to clarify and provide stronger nodal support for phylogenetic trees and to confirm the relationships, particularly of the families Polyxenidae and Lophoproctidae. Molecular identification is likely to be an important tool for this taxonomically challenging order in future. A simplified key to the genera is provided.</div></div></div><div id="lite-ad-ib2r4ss7ss" class="lite-page-ad lite-page-ad--align-left lite-page-ad--vertical-spacing-xl"><div class="lite-page-ad__inner"><div id="ib2r4ss7ss"><rg-ad id="ek1i9pdwdge" ad-unit="Instream_Video_Mobile" ad-unit-path="/83500759/Instream_Video_Mobile" refresh-timeout="0" load-on-init="true" collapse="never" class="rg-ad" ad-unit-sizes="[[1,1]]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="ek1i9pdwdge" targeting-sequence="1"><rg-ad-size size="[1,1]"></rg-ad-size></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("ek1i9pdwdge").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-ib2r4ss7ss').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-ib2r4ss7ss");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-ib2r4ss7ss').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-ib2r4ss7ss");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div><div class="request-fulltext-promo"><div class="nova-o-media-object nova-o-media-object--gutter-m nova-o-media-object--vertical-align-middle"><div class="nova-o-media-object__item nova-o-media-object__item--width-full"><p class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-auto nova-e-text--color-inherit">Do you want to <strong>read the rest</strong> of this article?</p><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-auto js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;#login-modal&quot;,&quot;a&quot;:&quot;tc(lite-page-modal--hidden)&quot;,&quot;c&quot;:1},{&quot;t&quot;:&quot;.lite-page&quot;,&quot;a&quot;:&quot;ft(https://www.researchgate.net/lite.publication.PublicationRequestFulltextPromo.requestFulltext.html?publicationUid=320321240&amp;ev=su_requestFulltext)&quot;}]}"><span class="nova-c-button__label gtm-request-fulltext-btn-promo">Request full-text</span></button></div><div class="nova-o-media-object__item nova-o-media-object__item--width-auto request-fulltext-promo__image"><picture width="60" height="82"><source data-srcset="images/symbols/publication/publication_pdf_icon.png" type="image/png" width="60" height="82"/> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="images/symbols/publication/publication_pdf_icon.png" class="lite-image" alt="Request Full-text Paper PDF" width="60" height="82"/></picture></div></div></div></section><div id="lite-ad-bfkpdgmtgog" class="lite-page-ad lite-page-ad--align-center lite-page-ad--has-label"><div class="lite-page-ad__inner"><div id="bfkpdgmtgog"><rg-ad id="zpj5d24y1v" ad-unit="LoggedOut_PublicationWithoutFulltext_Middle" ad-unit-path="/83500759/LoggedOut_PublicationWithoutFulltext_Middle" ad-location="Middle" refresh-timeout="-1" load-on-init="false" collapse="never" class="rg-ad" ad-unit-sizes="[[728,90],[300,250],&quot;fluid&quot;]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="zpj5d24y1v" targeting-sequence="1"><rg-ad-size size="[728,90]"></rg-ad-size><rg-ad-size size="[300,250]"></rg-ad-size><rg-ad-size size="&quot;fluid&quot;"></rg-ad-size><rg-ad-sizeset viewport-size="[0,0]" slot-sizes="[]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[250,0]" slot-sizes="[[250,250]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[300,0]" slot-sizes="[[250,250],[300,250],[300,100],[300,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[320,0]" slot-sizes="[[250,250],[300,250],[300,100],[300,50],[320,100],[320,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[336,0]" slot-sizes="[[336,280],[300,250],[250,250],[300,100],[300,50],[320,100],[320,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[992,0]" slot-sizes="[[728,90],&quot;fluid&quot;]"></rg-ad-sizeset></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("zpj5d24y1v").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-bfkpdgmtgog').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-bfkpdgmtgog");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-bfkpdgmtgog').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-bfkpdgmtgog");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div><section class="publication-bottom"><section class="publication-details__section"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none publication-file-list"></div></section><section class="publication-details__section"><div class="js-target-citationReferences lite-tabs"><div style="position:relative;width:100%;height:100%" class="publication-citations__menubar"><nav class="nova-c-nav nova-c-nav--direction-horizontal nova-c-nav--theme-tabs nova-c-nav--spread-auto"><div class="nova-c-nav__wrapper"><div class="nova-c-nav__items" role="tablist"><button class="nova-c-nav__item is-selected citations js-lite-click" aria-disabled="false" aria-selected="true" role="tab" type="menu" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-citationReferences .lite-page-visible&quot;,&quot;a&quot;:&quot;rc(lite-page-visible)&quot;},{&quot;t&quot;:&quot;.js-target-citationReferences .is-selected&quot;,&quot;a&quot;:&quot;rc(is-selected)&quot;},{&quot;t&quot;:&quot;.js-target-citations&quot;,&quot;a&quot;:&quot;ac(lite-page-visible)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(is-selected)&quot;}]}" tabindex="-1"><span><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-c-nav__item-label">Citations (2)</div></span></button><button class="nova-c-nav__item references js-lite-click" aria-disabled="false" role="tab" type="menu" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-citationReferences .lite-page-visible&quot;,&quot;a&quot;:&quot;rc(lite-page-visible)&quot;},{&quot;t&quot;:&quot;.js-target-citationReferences .is-selected&quot;,&quot;a&quot;:&quot;rc(is-selected)&quot;},{&quot;t&quot;:&quot;.js-target-references&quot;,&quot;a&quot;:&quot;ac(lite-page-visible)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(is-selected)&quot;}]}"><span><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-c-nav__item-label">References (66)</div></span></button></div><div class="nova-c-nav__scroll-overflow" aria-hidden="true"><button class="nova-c-nav__scroll-overflow-trigger"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#arrow-right-s"></use></svg></button></div><div class="nova-c-nav__plus-overflow" aria-hidden="true"><button class="nova-c-nav__plus-overflow-trigger" tabindex="-1"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#ellipsis-s"></use></svg></button></div></div></nav><span><span><div class="nova-c-nav__plus-overflow-mount" aria-hidden="true" aria-haspopup="true" aria-expanded="false"></div></span></span></div><div class="lite-tabs__tab js-target-citations lite-page-visible"><div class="js-target-citation" id="citations"><div class="nova-o-stack nova-o-stack--gutter-s nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-bare nova-e-list--spacing-none publication-citations"><div><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div role="tooltip" id="tooltip-209779" class="nova-c-tooltip nova-c-tooltip--position-above-left nova-c-tooltip--elevation-none nova-c-tooltip--color-white nova-c-tooltip--spacing-m nova-c-tooltip--luminosity-medium nova-v-citation-item__context"><div class="nova-c-tooltip__body"><div class="nova-c-tooltip__body-content"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--show-divider nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--show-divider nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--show-divider nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-citation-item__context-body">... This idea was mentioned previously with statements of restricted distribution and high embryonic mortality in the parthenogenetic population (Sarvala 1979a). In some species of millipedes, parthenogenetic and dioecious populations were also mixed in one clade of the phyloge- netic tree <mark class="nova-e-text-highlight nova-e-text-highlight--color-yellow">(Short and Vahtera 2017)</mark>, suggesting that parthenogenesis may have evolved more recently as a specific adaptation to certain environmental conditions without chang- ing the structure of nucleotide sequences.  ...</div></div></div></div></div></div></div></div><div class="nova-c-tooltip__arrow nova-c-tooltip__arrow nova-c-tooltip__arrow--body"><div class="nova-c-tooltip__arrow-tip"></div></div></div></div></div><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-citation-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/327352347_Phylogenetic_Study_of_Dioecious_and_Parthenogenetic_Populations_of_Canthocamptus_staphylinus_Crustacea_Copepoda_Harpacticoida">Phylogenetic Study of Dioecious and Parthenogenetic Populations of Canthocamptus staphylinus (Crustacea, Copepoda, Harpacticoida)</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Aug 2018</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0038-3872_Bulletin_Southern_California_Academy_of_Sciences">Bull South Calif Acad Sci</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Elena_Kochanova"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/477857798463489-1490941585181_Q64/Elena_Kochanova.jpg" class="lite-image" alt="Elena Kochanova"/></picture></div><span class="nova-v-person-inline-item__fullname">Elena Kochanova</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Jouko_Sarvala"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273630126931972-1442249914236_Q64/Jouko_Sarvala.jpg" class="lite-image" alt="Jouko Sarvala"/></picture></div><span class="nova-v-person-inline-item__fullname">Jouko Sarvala</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/E_Fefilova"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/638419080343553-1529222378747_Q64/E_Fefilova.jpg" class="lite-image" alt="E. B. Fefilova"/></picture></div><span class="nova-v-person-inline-item__fullname">E. B. Fefilova</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a7192 lite-page-hidden">The phylogenetic relationships of four dioecious populations and one parthenogenetic population of the harpacticoid Canthocamptus staphylinus (Jurine, 1820) were studied. Analysis of the mtCOI gene revealed two main clades as a phylogenetic tree and a network of haplotypes: a clade with Fennoscandian populations in Lake Pjrvi (Finland) and Lake Vnern (Sweden), and a second clade with populations in Lake Vrtsjrv (Estonia), Orlov Pond in Saint Petersburg (Russia), and the type locality of the species in Lake Geneva (Switzerland). The parthenogenetic population of C. staphylinus showed the smallest nucleotide and haplotype polymorphisms and could have evolved as a reaction to the changing environmental conditions following the Last Glacial Maximum, 20K YBP</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="The phylogenetic relationships of four dioecious populations and one parthenogenetic population of the harpacticoid Canthocamptus staphylinus (Jurine, 1820) were studied. Analysis of the mtCOI gene revealed two main clades as a phylogenetic tree and a network of haplotypes: a clade with Fennoscandian populations in Lake Pjrvi (Finland) and Lake Vnern (Sweden), and a second clade with populations in Lake Vrtsjrv (Estonia), Orlov Pond in Saint Petersburg (Russia), and the type locality of the species in Lake Geneva (Switzerland). The parthenogenetic population of C. staphylinus showed the smallest nucleotide and haplotype polymorphisms and could have evolved as a reaction to the changing environmental conditions following the Last Glacial Maximum, 20K YBP" publicationUrl="publication/327352347_Phylogenetic_Study_of_Dioecious_and_Parthenogenetic_Populations_of_Canthocamptus_staphylinus_Crustacea_Copepoda_Harpacticoida" abstractClassName="js-target-abstract-5e40a6c4a7192"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/327352347_Phylogenetic_Study_of_Dioecious_and_Parthenogenetic_Populations_of_Canthocamptus_staphylinus_Crustacea_Copepoda_Harpacticoida"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a7192&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-citation-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/323352482_Two_new_species_of_Phryssonotus_DiplopodaSynxenidae_from_southern_and_Western_Australia">Two new species of Phryssonotus (Diplopoda:Synxenidae) from southern and Western Australia</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2018</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0004-959X_Australian_Journal_of_Zoology">AUST J ZOOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Cuong_Huynh"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273615451062279-1442246415718_Q64/Cuong_Huynh.jpg" class="lite-image" alt="Cuong Huynh"/></picture></div><span class="nova-v-person-inline-item__fullname">Cuong Huynh</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Anneke_Veenstra"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/324488602816512-1454375519265_Q64/Anneke_Veenstra.jpg" class="lite-image" alt="Anneke Veenstra"/></picture></div><span class="nova-v-person-inline-item__fullname">Anneke Veenstra</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a73c1 lite-page-hidden">For many years Phryssonotus novaehollandiae Silvestri, 1923 was thought to be the only species of this genus present in Australia. Specimens collected from three geographically separated populations had similar taxonomically important morphological characters and body lengths. However, their body scale trichomes exhibited three distinctive patterns: a trapezoid, T-shape, or dark banding, and the length-to-width ratios of their trichomes also differed. These differences, as well as results of phylogenetic analysis using 18S and COI gene regions from representatives of the three populations, suggested that more than one Phryssonotus species were present. Specimens of Phryssonotus preserved in ethanol can prove difficult to distinguish because the colour patterning gradually fades. Examination of the colour patterns of live specimens, other morphological characters and DNA analysis together, were found to be the most reliable method of correctly distinguishing between these three species. P. novaehollandiae Silvestri, 1923, the first described species collected from South Australia, remains, with its range extended to inland Victoria, and two new species (P. australis, sp. nov. and P. occidentalis, sp. nov.) are described.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="For many years Phryssonotus novaehollandiae Silvestri, 1923 was thought to be the only species of this genus present in Australia. Specimens collected from three geographically separated populations had similar taxonomically important morphological characters and body lengths. However, their body scale trichomes exhibited three distinctive patterns: a trapezoid, T-shape, or dark banding, and the length-to-width ratios of their trichomes also differed. These differences, as well as results of phylogenetic analysis using 18S and COI gene regions from representatives of the three populations, suggested that more than one Phryssonotus species were present. Specimens of Phryssonotus preserved in ethanol can prove difficult to distinguish because the colour patterning gradually fades. Examination of the colour patterns of live specimens, other morphological characters and DNA analysis together, were found to be the most reliable method of correctly distinguishing between these three species. P. novaehollandiae Silvestri, 1923, the first described species collected from South Australia, remains, with its range extended to inland Victoria, and two new species (P. australis, sp. nov. and P. occidentalis, sp. nov.) are described." publicationUrl="publication/323352482_Two_new_species_of_Phryssonotus_DiplopodaSynxenidae_from_southern_and_Western_Australia" abstractClassName="js-target-abstract-5e40a6c4a73c1"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/323352482_Two_new_species_of_Phryssonotus_DiplopodaSynxenidae_from_southern_and_Western_Australia"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a73c1&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li></div></ul></div></div></div></div><div class="lite-tabs__tab js-target-references"><div class="js-target-reference" id="references"><div class="nova-o-stack nova-o-stack--gutter-s nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-bare nova-e-list--spacing-auto publication-citations"><div><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/285818651_Systema_Naturae_per_Regna_Tria_Naturae_Secundum_Classes_Ordines_Genera_Species_cum_Characteribus_Differentiis_Synonymis_Locis">Systema Naturae per Regna Tria Naturae, Secundum Classes, Ordines, Genera, Species, cum Characteribus, Differentiis, Synonymis, Locis</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1758</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2087348717_C_Linnaeus"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">C. Linnaeus</span></span></a></li></ul></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" publicationUrl="publication/285818651_Systema_Naturae_per_Regna_Tria_Naturae_Secundum_Classes_Ordines_Genera_Species_cum_Characteribus_Differentiis_Synonymis_Locis" abstractClassName="js-target-abstract-5e40a6c4a84cd"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/285818651_Systema_Naturae_per_Regna_Tria_Naturae_Secundum_Classes_Ordines_Genera_Species_cum_Characteribus_Differentiis_Synonymis_Locis"><span class="nova-c-button__label">View</span></a></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Cours d&#x27;Entomologie et d&#x27;histoire naturelle des Crustacs des Arachnides, des Myriapodes et des Insectes  l&#x27;usage des lves de l&#x27;cole du Musum d&#x27;Histoire Naturelle. Paris: Musum d</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1831</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Pa Latreille</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Latreille PA. 1831. Cours d&#x27;Entomologie et d&#x27;histoire naturelle des Crustacs des Arachnides, des 
Myriapodes et des Insectes  l&#x27;usage des lves de l&#x27;cole du Musum d&#x27;Histoire Naturelle. Paris: 
Musum d&#x27;Histoire Naturelle. Roret; 1-568. doi:10.5962/bhl.title.4854</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Pauropsxenus (3 species)-3 ommatidia each side in adult Miopsxenus (2 species)-0 ommatidia each side in adult</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">........................................................................................................................... Ommatidia Each Side In Adult</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">ommatidia each side in adult................................................ Pauropsxenus (3 species)-3 ommatidia each side in adult.................................................... Miopsxenus (2 species)-0 ommatidia each side in adult................................................... Anopsxenus (2 species)</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Javanische Myriopoden, gesammelt von direktor K. Kraepelin im jahre 1903</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1907</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">77-142</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">C Attems</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Attems C. 1907. Javanische Myriopoden, gesammelt von direktor K. Kraepelin im jahre 1903. Mitt 
Naturhist Mus Hamburg. 24:77-142.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Macroxenus, nouveau genre de Myriapodes Pslaphognathes</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1917</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">114-118</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Hw Brlemann</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Brlemann HW. 1917. Macroxenus, nouveau genre de Myriapodes Pslaphognathes. Bull Soc Hist 
Nat Afr Noire. 8(6):114-118.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Myriapodes recueillis en Nouvelle-Caldonie et aux Loyality</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1931</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">275-316</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Hw Brlemann</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Brlemann HW. 1931. Myriapodes recueillis en Nouvelle-Caldonie et aux Loyality. Arch Zool Exp 
Gen. 72(2):275-316.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">On a diplopod collection from Barro Colorado Island</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1940</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">1-16</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Rv Chamberlin</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Chamberlin RV. 1940. On a diplopod collection from Barro Colorado Island. Panama. Bull Univ Utah 
Biol Ser. 5(6):1-16.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">On some millipeds from Micronesia</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1947</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">ENTOMOL NEWS</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">41-47</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">V Chamberlin</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Chamberlin V. 1947. On some millipeds from Micronesia. Entomol News. 58:41-47.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Un Pnicillate indit de Nouvelle Zlande (Myriapodes, Diplopodes)</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1951</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">9-14</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B Cond</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cond B. 1951a. Un Pnicillate indit de Nouvelle Zlande (Myriapodes, Diplopodes). RecCant Mus. 
6(1):9-14.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Pnicillates de Basse-Egypte (Myriapodes Diplopodes) Bull Soc Zool Fr</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1951</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">55-63</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B Cond</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cond B. 1951b. Pnicillates de Basse-Egypte (Myriapodes Diplopodes). Bull Soc Zool Fr. 76:55-63.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Un Polyxnid aveugle (Diplopodes Pnicillates) Bull Soc Zool Fr</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1954</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">415-418</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B Cond</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cond B. 1954. Un Polyxnid aveugle (Diplopodes Pnicillates). Bull Soc Zool Fr. 79:415-418.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Essai sur l&#x27;volution des Diplopodes Pnicillates. Bull Mus Nat Hist Nat 2e srie</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1970</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">48-52</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B Cond</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cond B. 1970. Essai sur l&#x27;volution des Diplopodes Pnicillates. Bull Mus Nat Hist Nat 2e srie. 41:48-52.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Une approche des Diplopoda Penicillata de L&#x27;amrique du Nord</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1996</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">127-135</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B Cond</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cond B. 1996. Une approche des Diplopoda Penicillata de L&#x27;amrique du Nord. In: Geoffroy J-J, 
Mauris J-P, Nguyen Duy-Jacquemin M, editors. Acta Myriapodologica. Memoir Mus Natl Hist. 
169:127-135.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">Diplopodes Pnicillates rcolts  Bombay parPA</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1963</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">68-78</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B Cond</span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">M Jacquemin</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cond B, Jacquemin M. 1963. Diplopodes Pnicillates rcolts  Bombay parPA. Remy. Rev Fr 
Entomol. 30:68-78.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">A new diplopod fauna in Liberia</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">AM NAT</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">593-595</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Of Cook</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Cook OF. 1896. A new diplopod fauna in Liberia. Am Nat. 30:593-595.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">tudes sur les Myriapodes</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">51-80</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">P Gervais</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Gervais P. 1844. tudes sur les Myriapodes. Ann Sci Nat sr 3. 2:51-80.</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/329566653_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jul 2016</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0737-4038_Molecular_Biology_and_Evolution">MOL BIOL EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Sudhir_Kumar35"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Sudhir Kumar"/></picture></div><span class="nova-v-person-inline-item__fullname">Sudhir Kumar</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Glen_Stecher"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273740978192404-1442276343506_Q64/Glen_Stecher.jpg" class="lite-image" alt="Glen Stecher"/></picture></div><span class="nova-v-person-inline-item__fullname">Glen Stecher</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Koichiro_Tamura"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273705918005258-1442267984412_Q64/Koichiro_Tamura.jpg" class="lite-image" alt="Koichiro Tamura"/></picture></div><span class="nova-v-person-inline-item__fullname">Koichiro Tamura</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a779e lite-page-hidden">We present the latest version of the Molecular Evolutionary Genetics Analysis (M ega ) software, which contains many sophisticated methods and tools for phylogenomics and phylomedicine. In this major upgrade, M ega has been optimized for use on 64-bit computing systems for analyzing larger datasets. Researchers can now explore and analyze tens of thousands of sequences in M ega . The new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict gene duplication events in gene family trees. The 64-bit M ega is made available in two interfaces: graphical and command line. The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OS X. The command line M ega is available as native applications for Windows, Linux, and Mac OS X. They are intended for use in high-throughput and scripted analysis. Both versions are available from www.megasoftware.net free of charge.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We present the latest version of the Molecular Evolutionary Genetics Analysis (M ega ) software, which contains many sophisticated methods and tools for phylogenomics and phylomedicine. In this major upgrade, M ega has been optimized for use on 64-bit computing systems for analyzing larger datasets. Researchers can now explore and analyze tens of thousands of sequences in M ega . The new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict gene duplication events in gene family trees. The 64-bit M ega is made available in two interfaces: graphical and command line. The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OS X. The command line M ega is available as native applications for Windows, Linux, and Mac OS X. They are intended for use in high-throughput and scripted analysis. Both versions are available from www.megasoftware.net free of charge." publicationUrl="publication/329566653_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets" abstractClassName="js-target-abstract-5e40a6c4a779e"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/329566653_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a779e&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/318502387_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jul 2016</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0737-4038_Molecular_Biology_and_Evolution">MOL BIOL EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Sudhir_Kumar35"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Sudhir Kumar"/></picture></div><span class="nova-v-person-inline-item__fullname">Sudhir Kumar</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Glen_Stecher"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273740978192404-1442276343506_Q64/Glen_Stecher.jpg" class="lite-image" alt="Glen Stecher"/></picture></div><span class="nova-v-person-inline-item__fullname">Glen Stecher</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Koichiro_Tamura"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273705918005258-1442267984412_Q64/Koichiro_Tamura.jpg" class="lite-image" alt="Koichiro Tamura"/></picture></div><span class="nova-v-person-inline-item__fullname">Koichiro Tamura</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a79e4 lite-page-hidden">We present the latest version of the Molecular Evolutionary Genetics Analysis (M ega ) software, which contains many sophisticated methods and tools for phylogenomics and phylomedicine. In this major upgrade, M ega has been optimized for use on 64-bit computing systems for analyzing larger datasets. Researchers can now explore and analyze tens of thousands of sequences in M ega . The new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict gene duplication events in gene family trees. The 64-bit M ega is made available in two interfaces: graphical and command line. The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OS X. The command line M ega is available as native applications for Windows, Linux, and Mac OS X. They are intended for use in high-throughput and scripted analysis. Both versions are available from www.megasoftware.net free of charge.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We present the latest version of the Molecular Evolutionary Genetics Analysis (M ega ) software, which contains many sophisticated methods and tools for phylogenomics and phylomedicine. In this major upgrade, M ega has been optimized for use on 64-bit computing systems for analyzing larger datasets. Researchers can now explore and analyze tens of thousands of sequences in M ega . The new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict gene duplication events in gene family trees. The 64-bit M ega is made available in two interfaces: graphical and command line. The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OS X. The command line M ega is available as native applications for Windows, Linux, and Mac OS X. They are intended for use in high-throughput and scripted analysis. Both versions are available from www.megasoftware.net free of charge." publicationUrl="publication/318502387_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets" abstractClassName="js-target-abstract-5e40a6c4a79e4"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/318502387_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a79e4&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/313391996_MEGA7_Molecular_evolutionary_genetics_analysis_version_70_for_bigger_datasets">MEGA7: Molecular evolutionary genetics analysis version 7.0 for bigger datasets</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2015</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0737-4038_Molecular_Biology_and_Evolution">MOL BIOL EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Sumitxl_Kumar"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Sumitxl Kumar"/></picture></div><span class="nova-v-person-inline-item__fullname">Sumitxl Kumar</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Glen_Stecher"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273740978192404-1442276343506_Q64/Glen_Stecher.jpg" class="lite-image" alt="Glen Stecher"/></picture></div><span class="nova-v-person-inline-item__fullname">Glen Stecher</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Koichiro_Tamura"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273705918005258-1442267984412_Q64/Koichiro_Tamura.jpg" class="lite-image" alt="Koichiro Tamura"/></picture></div><span class="nova-v-person-inline-item__fullname">Koichiro Tamura</span></span></a></li></ul></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" publicationUrl="publication/313391996_MEGA7_Molecular_evolutionary_genetics_analysis_version_70_for_bigger_datasets" abstractClassName="js-target-abstract-5e40a6c4a7be4"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/313391996_MEGA7_Molecular_evolutionary_genetics_analysis_version_70_for_bigger_datasets"><span class="nova-c-button__label">View</span></a></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/301627335_TNT_version_15_including_a_full_implementation_of_phylogenetic_morphometrics">TNT version 1.5, including a full implementation of phylogenetic morphometrics</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Apr 2016</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1096-0031_Cladistics">Cladistics</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Pablo_Goloboff"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/277069004066826-1443069806230_Q64/Pablo_Goloboff.jpg" class="lite-image" alt="Pablo A. Goloboff"/></picture></div><span class="nova-v-person-inline-item__fullname">Pablo A. Goloboff</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Santiago_Catalano"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273674053877789-1442260387933_Q64/Santiago_Catalano.jpg" class="lite-image" alt="Santiago A. Catalano"/></picture></div><span class="nova-v-person-inline-item__fullname">Santiago A. Catalano</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a7df8 lite-page-hidden">Version 1.5 of the computer program TNT completely integrates landmark data into phylogenetic analysis. Landmark data consist of coordinates (in two or three dimensions) for the terminal taxa; TNT reconstructs shapes for the internal nodes such that the difference between ancestor and descendant shapes for all tree branches sums up to a minimum; this sum is used as tree score. Landmark data can be analysed alone or in combination with standard characters; all the applicable commands and options in TNT can be used transparently after reading a landmark data set. The program continues implementing all the types of analyses in former versions, including discrete and continuous characters (which can now be read at any scale, and automatically rescaled by TNT). Using algorithms described in this paper, searches for landmark data can be made tens to hundreds of times faster than it was possible before (from T to 3T times faster, where T is the number of taxa), thus making phylogenetic analysis of landmarks feasible even on standard personal computers.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Version 1.5 of the computer program TNT completely integrates landmark data into phylogenetic analysis. Landmark data consist of coordinates (in two or three dimensions) for the terminal taxa; TNT reconstructs shapes for the internal nodes such that the difference between ancestor and descendant shapes for all tree branches sums up to a minimum; this sum is used as tree score. Landmark data can be analysed alone or in combination with standard characters; all the applicable commands and options in TNT can be used transparently after reading a landmark data set. The program continues implementing all the types of analyses in former versions, including discrete and continuous characters (which can now be read at any scale, and automatically rescaled by TNT). Using algorithms described in this paper, searches for landmark data can be made tens to hundreds of times faster than it was possible before (from T to 3T times faster, where T is the number of taxa), thus making phylogenetic analysis of landmarks feasible even on standard personal computers." publicationUrl="publication/301627335_TNT_version_15_including_a_full_implementation_of_phylogenetic_morphometrics" abstractClassName="js-target-abstract-5e40a6c4a7df8"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/301627335_TNT_version_15_including_a_full_implementation_of_phylogenetic_morphometrics"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a7df8&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/299371757_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Mar 2016</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0737-4038_Molecular_Biology_and_Evolution">MOL BIOL EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Sudhir_Kumar35"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Sudhir Kumar"/></picture></div><span class="nova-v-person-inline-item__fullname">Sudhir Kumar</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Glen_Stecher"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273740978192404-1442276343506_Q64/Glen_Stecher.jpg" class="lite-image" alt="Glen Stecher"/></picture></div><span class="nova-v-person-inline-item__fullname">Glen Stecher</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Koichiro_Tamura"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273705918005258-1442267984412_Q64/Koichiro_Tamura.jpg" class="lite-image" alt="Koichiro Tamura"/></picture></div><span class="nova-v-person-inline-item__fullname">Koichiro Tamura</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a7fc1 lite-page-hidden">We present the latest version of the Molecular Evolutionary Genetics Analysis (MEGA) software, which contains many sophisticated
methods and tools for phylogenomics and phylomedicine. In this major upgrade, MEGA has been optimized for use on 64-bit computing
systems for analyzing bigger datasets. Researchers can now explore and analyze tens of thousands of sequences in MEGA. The
new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict
gene duplication events in gene family trees. The 64-bit MEGA is made available in two interfaces: graphical and command line.
The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OSX. The command
line MEGA is available as native applications for Windows, Linux, and Mac OSX. They are intended for use in high-throughput
and scripted analysis. Both versions are available from www.megasoftware.net free of charge.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We present the latest version of the Molecular Evolutionary Genetics Analysis (MEGA) software, which contains many sophisticated
methods and tools for phylogenomics and phylomedicine. In this major upgrade, MEGA has been optimized for use on 64-bit computing
systems for analyzing bigger datasets. Researchers can now explore and analyze tens of thousands of sequences in MEGA. The
new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict
gene duplication events in gene family trees. The 64-bit MEGA is made available in two interfaces: graphical and command line.
The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OSX. The command
line MEGA is available as native applications for Windows, Linux, and Mac OSX. They are intended for use in high-throughput
and scripted analysis. Both versions are available from www.megasoftware.net free of charge." publicationUrl="publication/299371757_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets" abstractClassName="js-target-abstract-5e40a6c4a7fc1"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/299371757_MEGA7_Molecular_Evolutionary_Genetics_Analysis_Version_70_for_Bigger_Datasets"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a7fc1&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/291022395_Histoire_naturelle_des_animaux_articules_Crustaces_Arachnides_Myriapodes_et_Hexapodes">Histoire naturelle des animaux articuls. Crustacs, Arachnides, Myriapodes et Hexapodes</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2094834961_H_Lucas"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">H. Lucas</span></span></a></li></ul></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" publicationUrl="publication/291022395_Histoire_naturelle_des_animaux_articules_Crustaces_Arachnides_Myriapodes_et_Hexapodes" abstractClassName="js-target-abstract-5e40a6c4a812c"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/291022395_Histoire_naturelle_des_animaux_articules_Crustaces_Arachnides_Myriapodes_et_Hexapodes"><span class="nova-c-button__label">View</span></a></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/290185348_Present_classification_of_Diplopoda_Penicillata_Myriapoda_with_new_definitions_of_taxa">Present classification of Diplopoda Penicillata (Myriapoda), with new definitions of taxa</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2008</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2093527069_B_Conde"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">B. Cond</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2094409504_M_Nguyen_Duy-Jacquemin"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">M. Nguyen Duy-Jacquemin</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a827c lite-page-hidden">An overview of the classification of Penicillata is presented, based on an unpublished manuscript written by B. Conde some years before his death, updated to take into account recent work on the group. It revises the descriptions of Silvestri&#x27;s (1948a) superfamilies and families, and proposes a subdivision of the family Polyxenidae into four sub-families: Polyxeninae Lucas, 1840, Monographinae Conde&#x27; nov., Macroxeninae Condi nov. and Hypogexeninae Schubart, 1947. The first three subfamilies are characterised by the structure of the telson, while the fourth is of uncertain status. The creation of a family-rank taxon for the succinctly described species H. pusillus Schubart in 1947 appears unwarranted and we therefore reduce Hypogexenidae to the rank of a subfamily, Hypogexeninae, within the Polyxenidae.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="An overview of the classification of Penicillata is presented, based on an unpublished manuscript written by B. Conde some years before his death, updated to take into account recent work on the group. It revises the descriptions of Silvestri&#x27;s (1948a) superfamilies and families, and proposes a subdivision of the family Polyxenidae into four sub-families: Polyxeninae Lucas, 1840, Monographinae Conde&#x27; nov., Macroxeninae Condi nov. and Hypogexeninae Schubart, 1947. The first three subfamilies are characterised by the structure of the telson, while the fourth is of uncertain status. The creation of a family-rank taxon for the succinctly described species H. pusillus Schubart in 1947 appears unwarranted and we therefore reduce Hypogexenidae to the rank of a subfamily, Hypogexeninae, within the Polyxenidae." publicationUrl="publication/290185348_Present_classification_of_Diplopoda_Penicillata_Myriapoda_with_new_definitions_of_taxa" abstractClassName="js-target-abstract-5e40a6c4a827c"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/290185348_Present_classification_of_Diplopoda_Penicillata_Myriapoda_with_new_definitions_of_taxa"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a827c&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/287356655_Results_of_Dr_E_Mjoberg&#x27;s_scientific_expeditions_to_Australia_1910-1913_34_Myriapoda_Diplopoda">Results of Dr. E. Mjberg&#x27;s scientific expeditions to Australia 1910-1913. 34. Myriapoda: Diplopoda</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2089661383_KW_Verhoeff"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">K.W. Verhoeff</span></span></a></li></ul></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" publicationUrl="publication/287356655_Results_of_Dr_E_Mjoberg&#x27;s_scientific_expeditions_to_Australia_1910-1913_34_Myriapoda_Diplopoda" abstractClassName="js-target-abstract-5e40a6c4a8332"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/287356655_Results_of_Dr_E_Mjoberg&#x27;s_scientific_expeditions_to_Australia_1910-1913_34_Myriapoda_Diplopoda"><span class="nova-c-button__label">View</span></a></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/286669749_Fossil_giants_and_surviving_dwarfs_Arthropleurida_and_Pselaphognatha_Atelocerata_Diplopoda_Characters_phylogenetic_relationships_and_construction">Fossil giants and surviving dwarfs. Arthropleurida and Pselaphognatha (Atelocerata, Diplopoda): Characters, phylogenetic relationships and construction</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2003</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2088463375_O_Kraus"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">O. Kraus</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Carsten_Brauckmann"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272913618173959-1442079085304_Q64/Carsten_Brauckmann.jpg" class="lite-image" alt="Carsten Brauckmann"/></picture></div><span class="nova-v-person-inline-item__fullname">Carsten Brauckmann</span></span></a></li></ul></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" publicationUrl="publication/286669749_Fossil_giants_and_surviving_dwarfs_Arthropleurida_and_Pselaphognatha_Atelocerata_Diplopoda_Characters_phylogenetic_relationships_and_construction" abstractClassName="js-target-abstract-5e40a6c4a8420"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/286669749_Fossil_giants_and_surviving_dwarfs_Arthropleurida_and_Pselaphognatha_Atelocerata_Diplopoda_Characters_phylogenetic_relationships_and_construction"><span class="nova-c-button__label">View</span></a></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title">12-Absence of ommatidia</div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-right"></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Ommatidia....................................................................................................................................................... Presence</span></span></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-v-publication-item__description">Presence of ommatidia.............................................................................................................. 12-Absence of ommatidia..................................................................... Typhloxenus (2 species)</div></div></div></div><footer class="nova-v-publication-item__footer"></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/284671294_Histoire_naturelle_generale_et_particuliere_des_Crustaces_et_des_Insectes">Histoire naturelle gnrale et particulire des Crustacs et des Insectes</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 1805</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2085740287_PA_Latreille"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">P.A. Latreille</span></span></a></li></ul></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" publicationUrl="publication/284671294_Histoire_naturelle_generale_et_particuliere_des_Crustaces_et_des_Insectes" abstractClassName="js-target-abstract-5e40a6c4a8563"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/284671294_Histoire_naturelle_generale_et_particuliere_des_Crustaces_et_des_Insectes"><span class="nova-c-button__label">View</span></a></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/265925424_Phryssonotus_novaehollandiae_Silvestri_1923_the_sole_Australian_representative_of_the_millipede_Family_Synxenidae">Phryssonotus novaehollandiae Silvestri, 1923: the sole Australian representative of the millipede Family Synxenidae</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2009</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Megan_Short3"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Megan Short"/></picture></div><span class="nova-v-person-inline-item__fullname">Megan Short</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Cuong_Huynh"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273615451062279-1442246415718_Q64/Cuong_Huynh.jpg" class="lite-image" alt="Cuong Huynh"/></picture></div><span class="nova-v-person-inline-item__fullname">Cuong Huynh</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a85ba lite-page-hidden">Examination of synxenid millipedes from a number of collections confirms that Phryssonotus novaehollandiae is the sole representative of the genus and family in Australia. P. novaehollandiae was found to have the most widespread distribution of any native Australian millipede species. It occurs in a range of well-drained habitats including heathlands, woodlands and coastal scrub. Several thelytokous (female only) populations were found in coastal areas of south eastern Australia.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Examination of synxenid millipedes from a number of collections confirms that Phryssonotus novaehollandiae is the sole representative of the genus and family in Australia. P. novaehollandiae was found to have the most widespread distribution of any native Australian millipede species. It occurs in a range of well-drained habitats including heathlands, woodlands and coastal scrub. Several thelytokous (female only) populations were found in coastal areas of south eastern Australia." publicationUrl="publication/265925424_Phryssonotus_novaehollandiae_Silvestri_1923_the_sole_Australian_representative_of_the_millipede_Family_Synxenidae" abstractClassName="js-target-abstract-5e40a6c4a85ba"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/265925424_Phryssonotus_novaehollandiae_Silvestri_1923_the_sole_Australian_representative_of_the_millipede_Family_Synxenidae"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a85ba&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/260241916_Molecular_phylogeny_of_Myriapoda_provides_insights_into_evolutionary_patterns_of_the_mode_in_post-embryonic_development">Molecular phylogeny of Myriapoda provides insights into evolutionary patterns of the mode in post-embryonic development</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Feb 2014</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Hideyuki_Miyazawa"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Hideyuki Miyazawa"/></picture></div><span class="nova-v-person-inline-item__fullname">Hideyuki Miyazawa</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2043967183_Chiaki_Ueda"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Chiaki Ueda</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Kensuke_Yahata"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/292014248742912-1446633030236_Q64/Kensuke_Yahata.jpg" class="lite-image" alt="Kensuke Yahata"/></picture></div><span class="nova-v-person-inline-item__fullname">Kensuke Yahata</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Zhi-Hui_Su"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272441553453059-1441966536441_Q64/Zhi-Hui_Su.jpg" class="lite-image" alt="Zhi-Hui Su"/></picture></div><span class="nova-v-person-inline-item__fullname">Zhi-Hui Su</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a87c3 lite-page-hidden">Myriapoda, a subphylum of Arthropoda, comprises four classes, Chilopoda, Diplopoda, Pauropoda, and Symphyla. While recent molecular evidence has shown that Myriapoda is monophyletic, the internal phylogeny, which is pivotal for understanding the evolutionary history of myriapods, remains unresolved. Here we report the results of phylogenetic analyses and estimations of divergence time and ancestral state of myriapods. Phylogenetic analyses were performed based on three nuclear protein-coding genes determined from 19 myriapods representing the four classes (17 orders) and 11 outgroup species. The results revealed that Symphyla whose phylogenetic position has long been debated is the sister lineage to all other myriapods, and that the interordinal relationships within classes were consistent with traditional classifications. Ancestral state estimation based on the tree topology suggests that myriapods evolved from an ancestral state that was characterized by a hemianamorphic mode of post-embryonic development and had a relatively low number of body segments and legs.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Myriapoda, a subphylum of Arthropoda, comprises four classes, Chilopoda, Diplopoda, Pauropoda, and Symphyla. While recent molecular evidence has shown that Myriapoda is monophyletic, the internal phylogeny, which is pivotal for understanding the evolutionary history of myriapods, remains unresolved. Here we report the results of phylogenetic analyses and estimations of divergence time and ancestral state of myriapods. Phylogenetic analyses were performed based on three nuclear protein-coding genes determined from 19 myriapods representing the four classes (17 orders) and 11 outgroup species. The results revealed that Symphyla whose phylogenetic position has long been debated is the sister lineage to all other myriapods, and that the interordinal relationships within classes were consistent with traditional classifications. Ancestral state estimation based on the tree topology suggests that myriapods evolved from an ancestral state that was characterized by a hemianamorphic mode of post-embryonic development and had a relatively low number of body segments and legs." publicationUrl="publication/260241916_Molecular_phylogeny_of_Myriapoda_provides_insights_into_evolutionary_patterns_of_the_mode_in_post-embryonic_development" abstractClassName="js-target-abstract-5e40a6c4a87c3"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/260241916_Molecular_phylogeny_of_Myriapoda_provides_insights_into_evolutionary_patterns_of_the_mode_in_post-embryonic_development"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a87c3&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/259876540_RAxML_Version_8_A_Tool_for_Phylogenetic_Analysis_and_Post-Analysis_of_Large_Phylogenies">RAxML Version 8: A Tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2014</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1367-4811_Bioinformatics">BIOINFORMATICS</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Alexandros_Stamatakis"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273584429989890-1442239019544_Q64/Alexandros_Stamatakis.jpg" class="lite-image" alt="Alexandros Stamatakis"/></picture></div><span class="nova-v-person-inline-item__fullname">Alexandros Stamatakis</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a894d lite-page-hidden">Phylogenies are increasingly used in all fields of medical and biological research. Moreover, because of the next generation sequencing revolution, datasets used for conducting phylogenetic analyses grow at an unprecedented pace. RAxML (Randomized Axelerated Maximum Likelihood) is a popular program for phylogenetic analyses of large datasets under maximum likelihood. Since the last RAxML paper in 2006, it has been continuously maintained and extended to accommodate the increasingly growing input datasets and to serve the needs of the user community.
I present some of the most notable new features and extensions of RAxML, such as, a substantial extension of substitution models and supported data types, the introduction of SSE3, AVX, and AVX2 vector intrinsics, techniques for reducing the memory requirements of the code and a plethora of operations for conducting post-analyses on sets of trees. In addition, an up-to-date, 50 page user manual covering all new RAxML options is available.
The code is available under GNU GPL at https://github.com/stamatak/standard-RAxML.
Alexandros.Stamatakis@h-its.org.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Phylogenies are increasingly used in all fields of medical and biological research. Moreover, because of the next generation sequencing revolution, datasets used for conducting phylogenetic analyses grow at an unprecedented pace. RAxML (Randomized Axelerated Maximum Likelihood) is a popular program for phylogenetic analyses of large datasets under maximum likelihood. Since the last RAxML paper in 2006, it has been continuously maintained and extended to accommodate the increasingly growing input datasets and to serve the needs of the user community.
I present some of the most notable new features and extensions of RAxML, such as, a substantial extension of substitution models and supported data types, the introduction of SSE3, AVX, and AVX2 vector intrinsics, techniques for reducing the memory requirements of the code and a plethora of operations for conducting post-analyses on sets of trees. In addition, an up-to-date, 50 page user manual covering all new RAxML options is available.
The code is available under GNU GPL at https://github.com/stamatak/standard-RAxML.
Alexandros.Stamatakis@h-its.org." publicationUrl="publication/259876540_RAxML_Version_8_A_Tool_for_Phylogenetic_Analysis_and_Post-Analysis_of_Large_Phylogenies" abstractClassName="js-target-abstract-5e40a6c4a894d"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/259876540_RAxML_Version_8_A_Tool_for_Phylogenetic_Analysis_and_Post-Analysis_of_Large_Phylogenies"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a894d&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/248899738_The_systematics_of_the_south-east_Asian_genus_Fangensis_Rambla_OpilioneCyphophthalmiStylocellidae">The systematics of the south-east Asian genus Fangensis Rambla (Opilione:Cyphophthalmi:Stylocellidae)</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2005</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1445-5226_Invertebrate_Systematics">INVERTEBR SYST</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Gonzalo_Giribet"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272480971522060-1441975934974_Q64/Gonzalo_Giribet.jpg" class="lite-image" alt="Gonzalo Giribet"/></picture></div><span class="nova-v-person-inline-item__fullname">Gonzalo Giribet</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Peter_Schwendinger2"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Peter Schwendinger"/></picture></div><span class="nova-v-person-inline-item__fullname">Peter Schwendinger</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a8ada lite-page-hidden">The hitherto monotypic opilionid genus Fangensis Rambla, 1994 is re-evaluated from new material collected in northern, western and southern Thailand. The original description of F. leclerci Rambla (type material not traceable) was based on specimens belonging to at least three distinct species. Fangensis leclerci is thus redefined and the binomen restricted to specimens from a cave in northern Thailand only. Three new species are described, F. spelaeus, sp. nov. and F. cavernarus, sp. nov. from two caves in western Thailand and F. insulanus, sp. nov. from forests on two islands in southern Thailand. Taxonomic characters are discussed and relationships among Fangensis species and between Fangensis and other cyphophthalmids are evaluated using morphological data and 4 Kb of molecular data from three loci, two nuclear ribosomal genes and a mitochondrial coding gene. Overall, data indicate monophyly of Stylocellidae and morphological and ribosomal data suggest monophyly of Fangensis, which is in turn divided into two clades, one composed of the three species from northern and western Thailand and the other represented by the species from southern Thailand. Observations on mating in F. leclerci are provided.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="The hitherto monotypic opilionid genus Fangensis Rambla, 1994 is re-evaluated from new material collected in northern, western and southern Thailand. The original description of F. leclerci Rambla (type material not traceable) was based on specimens belonging to at least three distinct species. Fangensis leclerci is thus redefined and the binomen restricted to specimens from a cave in northern Thailand only. Three new species are described, F. spelaeus, sp. nov. and F. cavernarus, sp. nov. from two caves in western Thailand and F. insulanus, sp. nov. from forests on two islands in southern Thailand. Taxonomic characters are discussed and relationships among Fangensis species and between Fangensis and other cyphophthalmids are evaluated using morphological data and 4 Kb of molecular data from three loci, two nuclear ribosomal genes and a mitochondrial coding gene. Overall, data indicate monophyly of Stylocellidae and morphological and ribosomal data suggest monophyly of Fangensis, which is in turn divided into two clades, one composed of the three species from northern and western Thailand and the other represented by the species from southern Thailand. Observations on mating in F. leclerci are provided." publicationUrl="publication/248899738_The_systematics_of_the_south-east_Asian_genus_Fangensis_Rambla_OpilioneCyphophthalmiStylocellidae" abstractClassName="js-target-abstract-5e40a6c4a8ada"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/248899738_The_systematics_of_the_south-east_Asian_genus_Fangensis_Rambla_OpilioneCyphophthalmiStylocellidae"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a8ada&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/247673644_Parsimony_Jackknifing_outperforms_Neighbor-Joining">Parsimony Jackknifing outperforms Neighbor-Joining</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jun 1996</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0748-3007_Cladistics">CLADISTICS</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/James_Farris2"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/278441460027398-1443397025624_Q64/James_Farris2.jpg" class="lite-image" alt="James Farris"/></picture></div><span class="nova-v-person-inline-item__fullname">James Farris</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Victor_Albert4"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/403483216891904-1473209303323_Q64/Victor_Albert4.jpg" class="lite-image" alt="Victor A. Albert"/></picture></div><span class="nova-v-person-inline-item__fullname">Victor A. Albert</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Mari_Kaellersjoe"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Mari Kllersj"/></picture></div><span class="nova-v-person-inline-item__fullname">Mari Kllersj</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/18011096_Arnold_G_Kluge"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Arnold G. Kluge</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a8e99 lite-page-hidden">Because they are designed to produced just one tree, neighbor-joining programs can obscure ambiguities in data. Ambiguities can be uncovered by resampling, but existing neighbor-joining programs may give misleading bootstrap frequencies because they do not suppress zero-length branches and/or are sensitive to the order of terminals in the data. A new procedure, parsimony jackknifing, overcomes these problems while running hundreds of times faster than existing programs for neighbor-joining bootstrapping. For analysis of large matrices, parsimony jackknifing is hundreds of thousands of times faster than extensive branch-swapping, yet is better able to screen out poorly-supported groups.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Because they are designed to produced just one tree, neighbor-joining programs can obscure ambiguities in data. Ambiguities can be uncovered by resampling, but existing neighbor-joining programs may give misleading bootstrap frequencies because they do not suppress zero-length branches and/or are sensitive to the order of terminals in the data. A new procedure, parsimony jackknifing, overcomes these problems while running hundreds of times faster than existing programs for neighbor-joining bootstrapping. For analysis of large matrices, parsimony jackknifing is hundreds of thousands of times faster than extensive branch-swapping, yet is better able to screen out poorly-supported groups." publicationUrl="publication/247673644_Parsimony_Jackknifing_outperforms_Neighbor-Joining" abstractClassName="js-target-abstract-5e40a6c4a8e99"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/247673644_Parsimony_Jackknifing_outperforms_Neighbor-Joining"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a8e99&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/238620771_The_oldest_records_of_Polyxenida_Myriapoda_Diplopoda_New_discoveries_from_the_Cretaceous_ambers_of_Lebanon_and_France">The oldest records of Polyxenida (Myriapoda, Diplopoda): New discoveries from the Cretaceous ambers of Lebanon and France</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Dec 2004</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1280-9659_GEODIVERSITAS">Geodiversitas</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2127884200_M_Nguyen_Duy-Jaquemin"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">M Nguyen Duy-Jaquemin</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Dany_Azar"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/298027618193409-1448066729225_Q64/Dany_Azar.jpg" class="lite-image" alt="Dany Azar"/></picture></div><span class="nova-v-person-inline-item__fullname">Dany Azar</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a90fe lite-page-hidden">Electroxenus jezzinensis n. gen., n. sp. and Libanoxenus hammanaensis n. gen., n. sp. are described from the Lower Cretaceous amber of Lebanon. These are the oldest known records of Penicillata because Phryssonotus burmiticus (Cockerell, 1917), from Burmese amber, is dated as being from upper Albian. They belong to the family Polyxenidae. This family contains the recent genus Polyxenus Latreille, 1803, which is known from Eocene Baltic amber. Electroxenus n. gen. and Libanoxenus n. gen. are very close to the recent genera of Polyxenidae. The first French fossil Penicillata, discovered in the Cretaceous amber of Haute-Provence, is also described and referred to the genus Phryssonotus Scudder, 1885 (sole genus of the family Synxenidae). The recent polyxenid families Polyxenidae and Synxenidae therefore already existed during the Cretaceous.  Publications Scientifiques du Musum national d&#x27;Histoire naturelle, Paris.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Electroxenus jezzinensis n. gen., n. sp. and Libanoxenus hammanaensis n. gen., n. sp. are described from the Lower Cretaceous amber of Lebanon. These are the oldest known records of Penicillata because Phryssonotus burmiticus (Cockerell, 1917), from Burmese amber, is dated as being from upper Albian. They belong to the family Polyxenidae. This family contains the recent genus Polyxenus Latreille, 1803, which is known from Eocene Baltic amber. Electroxenus n. gen. and Libanoxenus n. gen. are very close to the recent genera of Polyxenidae. The first French fossil Penicillata, discovered in the Cretaceous amber of Haute-Provence, is also described and referred to the genus Phryssonotus Scudder, 1885 (sole genus of the family Synxenidae). The recent polyxenid families Polyxenidae and Synxenidae therefore already existed during the Cretaceous.  Publications Scientifiques du Musum national d&#x27;Histoire naturelle, Paris." publicationUrl="publication/238620771_The_oldest_records_of_Polyxenida_Myriapoda_Diplopoda_New_discoveries_from_the_Cretaceous_ambers_of_Lebanon_and_France" abstractClassName="js-target-abstract-5e40a6c4a90fe"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/238620771_The_oldest_records_of_Polyxenida_Myriapoda_Diplopoda_New_discoveries_from_the_Cretaceous_ambers_of_Lebanon_and_France"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a90fe&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/237226404_Analyzing_Large_Data_Sets_in_Reasonable_Times_Solutions_for_Composite_Optima">Analyzing Large Data Sets in Reasonable Times: Solutions for Composite Optima</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Dec 1999</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0748-3007_Cladistics">CLADISTICS</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Pablo_Goloboff"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/277069004066826-1443069806230_Q64/Pablo_Goloboff.jpg" class="lite-image" alt="Pablo A. Goloboff"/></picture></div><span class="nova-v-person-inline-item__fullname">Pablo A. Goloboff</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a9243 lite-page-hidden">New methods for parsimony analysis of large data sets are presented. The new methods are sectorial searches, tree-drifting, and tree-fusing. For Chase et al.&#x27;s 500-taxon data set these methods (on a 266-MHz Pentium II) find a shortest tree in less than 10 min (i.e., over 15,000 times faster than PAUP and 1000 times faster than PAUP*). Making a complete parsimony analysis requires hitting minimum length several times independently, but not necessarily all islands for Chase et al.&#x27;s data set, this can be done in 4 to 6 h. The new methods also perform well in other cases analyzed (which range from 170 to 854 taxa).</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="New methods for parsimony analysis of large data sets are presented. The new methods are sectorial searches, tree-drifting, and tree-fusing. For Chase et al.&#x27;s 500-taxon data set these methods (on a 266-MHz Pentium II) find a shortest tree in less than 10 min (i.e., over 15,000 times faster than PAUP and 1000 times faster than PAUP*). Making a complete parsimony analysis requires hitting minimum length several times independently, but not necessarily all islands for Chase et al.&#x27;s data set, this can be done in 4 to 6 h. The new methods also perform well in other cases analyzed (which range from 170 to 854 taxa)." publicationUrl="publication/237226404_Analyzing_Large_Data_Sets_in_Reasonable_Times_Solutions_for_Composite_Optima" abstractClassName="js-target-abstract-5e40a6c4a9243"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/237226404_Analyzing_Large_Data_Sets_in_Reasonable_Times_Solutions_for_Composite_Optima"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a9243&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/232674368_Revision_of_the_genera_of_Polyxenidae_bearing_pseudoarticulated_gnathochilarial_sensilla_Diplopoda_Polyxenida_Polyxenidae_with_the_description_of_two_new_species">Revision of the genera of Polyxenidae bearing pseudoarticulated gnathochilarial sensilla (Diplopoda, Polyxenida, Polyxenidae), with the description of two new species</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jan 2011</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1280-9551_Zoosystema">ZOOSYSTEMA</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2094409584_Monique_Nguyen_Duy-Jacquemin"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Monique Nguyen Duy-Jacquemin</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a9408 lite-page-hidden">Rvision des genres de Polyxenidae porteurs de sensilles gnathochilariaux pseudoarticuls (Diplopoda, Polyxenida, Polyxenidae), avec description de deux espces nouvelles. L&#x27;examen des syntypes des espces Chilexenus rosendinus (Silvestri, 1903) et Macroxenodes meinerti (Silvestri, 1898) a permis la rvision des quatre genres porteurs de sensilles gnathochilariaux pseudoarticuls appartenant  la sous-famille Macroxeninae Cond, 2008. Les caractres propres  chacun de ces quatre genres tropicaux ou sub-tropicaux : Chilexenus, Macroxenodes, Macroxenus Brlemann, 1917 et Afraustraloxenodes Nguyen Duy-Jacquemin, 2003 sont redfinis et deux nouvelles espces : Macroxenodes navassaensis n. sp. from Navassa Island et M. jahynyi n. sp. d&#x27;Uberlndia au Brsil sont dcrites. Des cls d&#x27;identification sont donnes pour les quatre genres et pour les espces  l&#x27;intrieur de chaque genre,  l&#x27;exception de Chilexenus monospcifique.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Rvision des genres de Polyxenidae porteurs de sensilles gnathochilariaux pseudoarticuls (Diplopoda, Polyxenida, Polyxenidae), avec description de deux espces nouvelles. L&#x27;examen des syntypes des espces Chilexenus rosendinus (Silvestri, 1903) et Macroxenodes meinerti (Silvestri, 1898) a permis la rvision des quatre genres porteurs de sensilles gnathochilariaux pseudoarticuls appartenant  la sous-famille Macroxeninae Cond, 2008. Les caractres propres  chacun de ces quatre genres tropicaux ou sub-tropicaux : Chilexenus, Macroxenodes, Macroxenus Brlemann, 1917 et Afraustraloxenodes Nguyen Duy-Jacquemin, 2003 sont redfinis et deux nouvelles espces : Macroxenodes navassaensis n. sp. from Navassa Island et M. jahynyi n. sp. d&#x27;Uberlndia au Brsil sont dcrites. Des cls d&#x27;identification sont donnes pour les quatre genres et pour les espces  l&#x27;intrieur de chaque genre,  l&#x27;exception de Chilexenus monospcifique." publicationUrl="publication/232674368_Revision_of_the_genera_of_Polyxenidae_bearing_pseudoarticulated_gnathochilarial_sensilla_Diplopoda_Polyxenida_Polyxenidae_with_the_description_of_two_new_species" abstractClassName="js-target-abstract-5e40a6c4a9408"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/232674368_Revision_of_the_genera_of_Polyxenidae_bearing_pseudoarticulated_gnathochilarial_sensilla_Diplopoda_Polyxenida_Polyxenidae_with_the_description_of_two_new_species"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a9408&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/231953819_The_identity_of_Phryssonotus_burmiticus_Cockerell_1917_Diplopoda_Polyxenida_Synxenidae_in_Cretaceous_amber_from_Myanmar">The identity of Phryssonotus burmiticus (Cockerell, 1917) (Diplopoda, Polyxenida, Synxenidae) in Cretaceous amber from Myanmar</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jun 2004</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1478-0941_Journal_of_Systematic_Palaeontology">J Syst Palaeontology</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2000906118_S_I_Golovatch"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">S. I. Golovatch</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a9577 lite-page-hidden">Based on a study of the holotype and of five presumably topotypic and conspecific juveniles, the millipede Phryssonotus
burmiticus (Cockerell, 1917) from Lower Cretaceous (Albian) Myanmar amber (Burmite) is revised and redescribed. All relevant millipede material from Burmite can now be unequivocally assigned to Phryssonotus and represents the geologically oldest member of the order Polyxenida known to date.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Based on a study of the holotype and of five presumably topotypic and conspecific juveniles, the millipede Phryssonotus
burmiticus (Cockerell, 1917) from Lower Cretaceous (Albian) Myanmar amber (Burmite) is revised and redescribed. All relevant millipede material from Burmite can now be unequivocally assigned to Phryssonotus and represents the geologically oldest member of the order Polyxenida known to date." publicationUrl="publication/231953819_The_identity_of_Phryssonotus_burmiticus_Cockerell_1917_Diplopoda_Polyxenida_Synxenidae_in_Cretaceous_amber_from_Myanmar" abstractClassName="js-target-abstract-5e40a6c4a9577"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/231953819_The_identity_of_Phryssonotus_burmiticus_Cockerell_1917_Diplopoda_Polyxenida_Synxenidae_in_Cretaceous_amber_from_Myanmar"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a9577&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/230265240_SequenceMatrix_Concatenation_software_for_the_fast_assembly_of_multi-gene_datasets_with_character_set_and_codon_information">SequenceMatrix: Concatenation software for the fast assembly of multi-gene datasets with character set and codon information</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Jul 2010</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1096-0031_Cladistics">Cladistics</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/29727043_Gaurav_Vaidya"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Gaurav Vaidya</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/David_Lohman"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272669006364674-1442020765048_Q64/David_Lohman.jpg" class="lite-image" alt="David J Lohman"/></picture></div><span class="nova-v-person-inline-item__fullname">David J Lohman</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Rudolf_Meier"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/278615083241491-1443438420692_Q64/Rudolf_Meier.jpg" class="lite-image" alt="Rudolf Meier"/></picture></div><span class="nova-v-person-inline-item__fullname">Rudolf Meier</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a98c1 lite-page-hidden">We present SequenceMatrix, software that is designed to facilitate the assembly and analysis of multi-gene datasets. Genes are concatenated by dragging and dropping FASTA, NEXUS, or TNT files with aligned sequences into the program window. A multi-gene dataset is concatenated and displayed in a spreadsheet; each sequence is represented by a cell that provides information on sequence length, number of indels, the number of ambiguous bases (Ns), and the availability of codon information. Alternatively, GenBank numbers for the sequences can be displayed and exported. Matrices with hundreds of genes and taxa can be concatenated within minutes and exported in TNT, NEXUS, or PHYLIP formats, preserving both character set and codon information for TNT and NEXUS files. SequenceMatrix also creates taxon sets listing taxa with a minimum number of characters or gene fragments, which helps assess preliminary datasets. Entire taxa, whole gene fragments, or individual sequences for a particular gene and species can be excluded from export. Data matrices can be re-split into their component genes and the gene fragments can be exported as individual gene files. SequenceMatrix also includes two tools that help to identify sequences that may have been compromised through laboratory contamination or data management error. One tool lists identical or near-identical sequences within genes, while the other compares the pairwise distance pattern of one gene against the pattern for all remaining genes combined. SequenceMatrix is Java-based and compatible with the Microsoft Windows, Apple MacOS X and Linux operating systems. The software is freely available from http://code.google.com/p/sequencematrix/.
 The Willi Hennig Society 2010.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We present SequenceMatrix, software that is designed to facilitate the assembly and analysis of multi-gene datasets. Genes are concatenated by dragging and dropping FASTA, NEXUS, or TNT files with aligned sequences into the program window. A multi-gene dataset is concatenated and displayed in a spreadsheet; each sequence is represented by a cell that provides information on sequence length, number of indels, the number of ambiguous bases (Ns), and the availability of codon information. Alternatively, GenBank numbers for the sequences can be displayed and exported. Matrices with hundreds of genes and taxa can be concatenated within minutes and exported in TNT, NEXUS, or PHYLIP formats, preserving both character set and codon information for TNT and NEXUS files. SequenceMatrix also creates taxon sets listing taxa with a minimum number of characters or gene fragments, which helps assess preliminary datasets. Entire taxa, whole gene fragments, or individual sequences for a particular gene and species can be excluded from export. Data matrices can be re-split into their component genes and the gene fragments can be exported as individual gene files. SequenceMatrix also includes two tools that help to identify sequences that may have been compromised through laboratory contamination or data management error. One tool lists identical or near-identical sequences within genes, while the other compares the pairwise distance pattern of one gene against the pattern for all remaining genes combined. SequenceMatrix is Java-based and compatible with the Microsoft Windows, Apple MacOS X and Linux operating systems. The software is freely available from http://code.google.com/p/sequencematrix/.
 The Willi Hennig Society 2010." publicationUrl="publication/230265240_SequenceMatrix_Concatenation_software_for_the_fast_assembly_of_multi-gene_datasets_with_character_set_and_codon_information" abstractClassName="js-target-abstract-5e40a6c4a98c1"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/230265240_SequenceMatrix_Concatenation_software_for_the_fast_assembly_of_multi-gene_datasets_with_character_set_and_codon_information"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a98c1&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/227883849_Phylogeny_of_millipedes_-_A_cladistic_analysis">Phylogeny of millipedes - A cladistic analysis</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Mar 1984</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1439-0469_Journal_of_Zoological_Systematics_and_Evolutionary_Research">J Zool Systemat Evol Res</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Henrik_Enghoff"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/400704050089993-1472546698833_Q64/Henrik_Enghoff.jpg" class="lite-image" alt="Henrik Enghoff"/></picture></div><span class="nova-v-person-inline-item__fullname">Henrik Enghoff</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a9a7c lite-page-hidden">Application of the principles of cladistics, or phylogenetic systematics, to the class Diplopoda indicates monophyly (in the strict sense) of the taxa Penicillata, Pentazonia, Colobognatha (minus Siphoniulus?), Nemathophora, Merocheta, and Juliformia. The result agrees with traditional classifications but stands in contrast to the classification by Hoffman (R. L. Hoffman, 1979: Classification of the Diplopoda. 237 pp. Musum d&#x27;Histoire naturelle, Genve).At higher levels, the analysis indicates strict monophyly of the taxa Diplopoda, Chilognatha, Helminthomorpha, and Eugnatha. The following sister-group relationships are proposed:Penicillata  ChilognathaPentazonia  HelminthomorphaColobognatha (minus Siphoniulus?)  EugnathaThe interrelationships of the eugnathan groups: Nematophora, Merocheta and Juliformia, remain unsettled, as does the position of Siphoniulus.A formal classification expressing the results is presented.ZusammenfassungPhylogenie der Tausendfer- eine kladistische AnalyseDie Anwendung von den Prinzipien der kladistischen  oder phylogenetischen  Systematik an der Klasse Diplopoda zeigt die Monophylie (sensu strictu) der Taxa Penicillata, Pentazonia, Colobognatha (ohne Siphoniulida?), Nematophora, Merocheta and Juliformia. Diese Ergebnisse stimmen mit traditionellen Klassifikationen berein, stehen aber in Widerspruch zur Klassifikation von R. L. Hoffman N (1979). Auf hheren taxonomischen Niveaus zeigt die Analyse Monophylie (sensu strictu) fr die Taxa Diplopoda, Chilognatha, Helminthomorpha and Eugnatha. Die folgenden Schwester-Gruppen-Verhaltnisse werden vorgeschlagen:Penicillata  ChilognathaPentazonia  HehninthomorphaColobognatha (ohne Siphoniulida?)  Eugnatha.Die verwandschaftlichen Beziehungen der eugnathen Gruppen: Nematophora, Merocheta and Juliformia sind nosh unsicher, ebenso die Stellung von Siphoniulida.Die Ergebnisse werden in foreteller Klassifikation dargelegt.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Application of the principles of cladistics, or phylogenetic systematics, to the class Diplopoda indicates monophyly (in the strict sense) of the taxa Penicillata, Pentazonia, Colobognatha (minus Siphoniulus?), Nemathophora, Merocheta, and Juliformia. The result agrees with traditional classifications but stands in contrast to the classification by Hoffman (R. L. Hoffman, 1979: Classification of the Diplopoda. 237 pp. Musum d&#x27;Histoire naturelle, Genve).At higher levels, the analysis indicates strict monophyly of the taxa Diplopoda, Chilognatha, Helminthomorpha, and Eugnatha. The following sister-group relationships are proposed:Penicillata  ChilognathaPentazonia  HelminthomorphaColobognatha (minus Siphoniulus?)  EugnathaThe interrelationships of the eugnathan groups: Nematophora, Merocheta and Juliformia, remain unsettled, as does the position of Siphoniulus.A formal classification expressing the results is presented.ZusammenfassungPhylogenie der Tausendfer- eine kladistische AnalyseDie Anwendung von den Prinzipien der kladistischen  oder phylogenetischen  Systematik an der Klasse Diplopoda zeigt die Monophylie (sensu strictu) der Taxa Penicillata, Pentazonia, Colobognatha (ohne Siphoniulida?), Nematophora, Merocheta and Juliformia. Diese Ergebnisse stimmen mit traditionellen Klassifikationen berein, stehen aber in Widerspruch zur Klassifikation von R. L. Hoffman N (1979). Auf hheren taxonomischen Niveaus zeigt die Analyse Monophylie (sensu strictu) fr die Taxa Diplopoda, Chilognatha, Helminthomorpha and Eugnatha. Die folgenden Schwester-Gruppen-Verhaltnisse werden vorgeschlagen:Penicillata  ChilognathaPentazonia  HehninthomorphaColobognatha (ohne Siphoniulida?)  Eugnatha.Die verwandschaftlichen Beziehungen der eugnathen Gruppen: Nematophora, Merocheta and Juliformia sind nosh unsicher, ebenso die Stellung von Siphoniulida.Die Ergebnisse werden in foreteller Klassifikation dargelegt." publicationUrl="publication/227883849_Phylogeny_of_millipedes_-_A_cladistic_analysis" abstractClassName="js-target-abstract-5e40a6c4a9a7c"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/227883849_Phylogeny_of_millipedes_-_A_cladistic_analysis"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a9a7c&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/221803809_Two_remarkable_species_of_Penicillata_Diplopoda_Polyxenida_from_Table_Mountain_National_Park_Cape_Town_South_Africa">Two remarkable species of Penicillata (Diplopoda, Polyxenida) from Table Mountain National Park (Cape Town, South Africa)</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Dec 2011</span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2094409584_Monique_Nguyen_Duy-Jacquemin"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Monique Nguyen Duy-Jacquemin</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/36034677_Charmaine_Uys"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Charmaine Uys</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/75653811_Jean-Jacques_Geoffroy"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Jean-Jacques Geoffroy</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4a9e0c lite-page-hidden">Two new species of the families Polyxenidae and Synxenidae, are described from Table Mountain National Park, South Africa. Propolyxenus squamatussp. n. (Polyxenidae) has tergites I-X mostly covered by scale-shaped trichomes directed caudally, a character previously known only in Synxenidae. The structure of scale-shaped dorsal trichomes is different to that of the scales in Phryssonotus and Condexenus species. Phryssonotus brevicapensissp. n. (Synxenidae) is the only known species of the genus Phryssonotus having 11 tergites, (including collum and telson) and 15 pairs of legs, as in Condexenus biramipalpus Nguyen Duy-Jacquemin, 2006. These two species therefore appear to occupy an intermediate position between Phryssonotus (12 tergites) and Polyxenoidea (maximum of 11 tergites).</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Two new species of the families Polyxenidae and Synxenidae, are described from Table Mountain National Park, South Africa. Propolyxenus squamatussp. n. (Polyxenidae) has tergites I-X mostly covered by scale-shaped trichomes directed caudally, a character previously known only in Synxenidae. The structure of scale-shaped dorsal trichomes is different to that of the scales in Phryssonotus and Condexenus species. Phryssonotus brevicapensissp. n. (Synxenidae) is the only known species of the genus Phryssonotus having 11 tergites, (including collum and telson) and 15 pairs of legs, as in Condexenus biramipalpus Nguyen Duy-Jacquemin, 2006. These two species therefore appear to occupy an intermediate position between Phryssonotus (12 tergites) and Polyxenoidea (maximum of 11 tergites)." publicationUrl="publication/221803809_Two_remarkable_species_of_Penicillata_Diplopoda_Polyxenida_from_Table_Mountain_National_Park_Cape_Town_South_Africa" abstractClassName="js-target-abstract-5e40a6c4a9e0c"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/221803809_Two_remarkable_species_of_Penicillata_Diplopoda_Polyxenida_from_Table_Mountain_National_Park_Cape_Town_South_Africa"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4a9e0c&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/39916314_Caroli_Linnaei_Entomologia_faunae_Suecicae_descriptionibus_aucta_5">Caroli Linnaei Entomologia, faunae Suecicae descriptionibus aucta. [5] /</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/64869666_Linne"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Linn</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/61256226_Carl_von"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Carl von</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/20744519_Villers"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Villers</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/20744520_Charles_Joseph_de"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Charles Joseph de</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4aa158 lite-page-hidden">Obsahuje obrazov plohy ze vech ty svazk</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Obsahuje obrazov plohy ze vech ty svazk" publicationUrl="publication/39916314_Caroli_Linnaei_Entomologia_faunae_Suecicae_descriptionibus_aucta_5" abstractClassName="js-target-abstract-5e40a6c4aa158"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/39916314_Caroli_Linnaei_Entomologia_faunae_Suecicae_descriptionibus_aucta_5"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4aa158&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/31449100_The_Stresiptera_Problem_Phylogeny_of_the_Holometabolous_Insect_Orders_Inferred_from_18S_and_28S_Ribosomal_DNA_Sequences_and_Morphology">The Stresiptera Problem: Phylogeny of the Holometabolous Insect Orders Inferred from 18S and 28S Ribosomal DNA Sequences and Morphology</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Apr 1997</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1063-5157_Systematic_Biology">SYST BIOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Wc_Wheeler"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="W.C. Wheeler"/></picture></div><span class="nova-v-person-inline-item__fullname">W.C. Wheeler</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2163230455_Michael_F_Whiting"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Michael F. Whiting</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/James_Carpenter3"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272315615281174-1441936510438_Q64/James_Carpenter3.jpg" class="lite-image" alt="James Michael Carpenter"/></picture></div><span class="nova-v-person-inline-item__fullname">James Michael Carpenter</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Quentin_Wheeler"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272309273493504-1441934998264_Q64/Quentin_Wheeler.jpg" class="lite-image" alt="Quentin Wheeler"/></picture></div><span class="nova-v-person-inline-item__fullname">Quentin Wheeler</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4aa426 lite-page-hidden">Phylogenetic relationships among the holometabolous insect orders were inferred from cladistic analysis of nucleotide sequences of 18S ribosomal DNA (rDNA) (85 exemplars) and 28S rDNA (52 exemplars) and morphological characters. Exemplar outgroup taxa were Collembola (1 sequence), Archaeognatha (1), Ephemerida (1), Odonata (2), Plecoptera (2), Blattodea (1), Mantodea (1), Dermaptera (1), Orthoptera (1), Phasmatodea (1), Embioptera (1), Psocoptera (1), Phthiraptera (1), Hemiptera (4), and Thysanoptera (1). Exemplar ingroup taxa were Coleoptera: Archostemata (1), Adephaga (2), and Polyphaga (7); Megaloptera (1); Raphidioptera (1); Neuroptera (sensu stricto = Planipennia): Mantispoidea (2), Hemerobioidea (2), and Myrmeleontoidea (2); Hymenoptera: Symphyta (4) and Apocrita (19); Trichoptera: Hydropsychoidea (1) and Limnephiloidea (2); Lepidoptera: Ditrysia (3); Siphonaptera: Pulicoidea (1) and Ceratophylloidea (2); Mecoptera: Meropeidae (1), Boreidae (1), Panorpidae (1), and Bittacidae (2); Diptera: Nematocera (1), Brachycera (2), and Cyclorrhapha (1); and Strepsiptera: Corioxenidae (1), Myrmecolacidae (1), Elenchidae (1), and Stylopidae (3). We analyzed 1 kilobase of 18S rDNA, starting 398 nucleotides downstream of the 5 end, and 400 bp of 28S rDNA in expansion segment D3. Multiple alignment of the 18S and 28S sequences resulted in 1,116 nucleotide positions with 24 insert regions and 398 positions with 14 insert regions, respectively. All Strepsiptera and Neuroptera have large insert regions in 18S and 28S. The secondary structure of 18S insert 23 is composed of long stems that are GC rich in the basal Strepsiptera and AT rich in the more derived Strepsiptera. A matrix of 176 morphological characters was analyzed for holometabolous orders. Incongruence length difference tests indicate that the 28S + morphological data sets are incongruent but that 28S + 18S, 18S + morphology, and 28S + 18S + morphology fail to reject the hypothesis of congruence. Phylogenetic trees were generated by parsimony analysis, and clade robustness was evaluated by branch length, Bremer support, percentage of extra steps required to force paraphyly, and sensitivity analysis using the following parameters: gap weights, morphological character weights, methods of data set combination, removal of key taxa, and alignment region. The following are monophyletic under most or all combinations of parameter values: Holometabola, Polyphaga, Megaloptera + Raphidioptera, Neuroptera, Hymenoptera, Trichoptera, Lepidoptera, Amphiesmenoptera (Trichoptera + Lepidoptera), Siphonaptera, Siphonaptera + Mecoptera, Strepsiptera, Diptera, and Strepsiptera + Diptera (Halteria). Antliophora (Mecoptera + Diptera + Siphonaptera + Strepsiptera), Mecopterida (Antliophora + Amphiesmenoptera), and Hymenoptera + Mecopterida are supported in the majority of total evidence analyses. Mecoptera may be paraphyletic because Boreus is often placed as sister group to the fleas; hence, Siphonaptera may be subordinate within Mecoptera. The 18S sequences for Priacma (Coleoptera: Archostemata), Colpocaccus (Coleoptera: Adephaga), Agulla (Raphidioptera), and Corydalus (Megaloptera) are nearly identical, and Neuropterida are monophyletic only when those two beetle sequences are removed from the analysis. Goleoptera are therefore paraphyletic under almost all combinations of parameter values. Halteria and Amphiesmenoptera have high Bremer support values and long branch lengths. The data do not support placement of Strepsiptera outside of Holometabola nor as sister group to Coleoptera. We reject the notion that the monophyly of Halteria is due to long branch attraction because Strepsiptera and Diptera do not have the longest branches and there is phylogenetic congruence between molecules, across the entire parameter space, and between morphological and molecular data.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Phylogenetic relationships among the holometabolous insect orders were inferred from cladistic analysis of nucleotide sequences of 18S ribosomal DNA (rDNA) (85 exemplars) and 28S rDNA (52 exemplars) and morphological characters. Exemplar outgroup taxa were Collembola (1 sequence), Archaeognatha (1), Ephemerida (1), Odonata (2), Plecoptera (2), Blattodea (1), Mantodea (1), Dermaptera (1), Orthoptera (1), Phasmatodea (1), Embioptera (1), Psocoptera (1), Phthiraptera (1), Hemiptera (4), and Thysanoptera (1). Exemplar ingroup taxa were Coleoptera: Archostemata (1), Adephaga (2), and Polyphaga (7); Megaloptera (1); Raphidioptera (1); Neuroptera (sensu stricto = Planipennia): Mantispoidea (2), Hemerobioidea (2), and Myrmeleontoidea (2); Hymenoptera: Symphyta (4) and Apocrita (19); Trichoptera: Hydropsychoidea (1) and Limnephiloidea (2); Lepidoptera: Ditrysia (3); Siphonaptera: Pulicoidea (1) and Ceratophylloidea (2); Mecoptera: Meropeidae (1), Boreidae (1), Panorpidae (1), and Bittacidae (2); Diptera: Nematocera (1), Brachycera (2), and Cyclorrhapha (1); and Strepsiptera: Corioxenidae (1), Myrmecolacidae (1), Elenchidae (1), and Stylopidae (3). We analyzed 1 kilobase of 18S rDNA, starting 398 nucleotides downstream of the 5 end, and 400 bp of 28S rDNA in expansion segment D3. Multiple alignment of the 18S and 28S sequences resulted in 1,116 nucleotide positions with 24 insert regions and 398 positions with 14 insert regions, respectively. All Strepsiptera and Neuroptera have large insert regions in 18S and 28S. The secondary structure of 18S insert 23 is composed of long stems that are GC rich in the basal Strepsiptera and AT rich in the more derived Strepsiptera. A matrix of 176 morphological characters was analyzed for holometabolous orders. Incongruence length difference tests indicate that the 28S + morphological data sets are incongruent but that 28S + 18S, 18S + morphology, and 28S + 18S + morphology fail to reject the hypothesis of congruence. Phylogenetic trees were generated by parsimony analysis, and clade robustness was evaluated by branch length, Bremer support, percentage of extra steps required to force paraphyly, and sensitivity analysis using the following parameters: gap weights, morphological character weights, methods of data set combination, removal of key taxa, and alignment region. The following are monophyletic under most or all combinations of parameter values: Holometabola, Polyphaga, Megaloptera + Raphidioptera, Neuroptera, Hymenoptera, Trichoptera, Lepidoptera, Amphiesmenoptera (Trichoptera + Lepidoptera), Siphonaptera, Siphonaptera + Mecoptera, Strepsiptera, Diptera, and Strepsiptera + Diptera (Halteria). Antliophora (Mecoptera + Diptera + Siphonaptera + Strepsiptera), Mecopterida (Antliophora + Amphiesmenoptera), and Hymenoptera + Mecopterida are supported in the majority of total evidence analyses. Mecoptera may be paraphyletic because Boreus is often placed as sister group to the fleas; hence, Siphonaptera may be subordinate within Mecoptera. The 18S sequences for Priacma (Coleoptera: Archostemata), Colpocaccus (Coleoptera: Adephaga), Agulla (Raphidioptera), and Corydalus (Megaloptera) are nearly identical, and Neuropterida are monophyletic only when those two beetle sequences are removed from the analysis. Goleoptera are therefore paraphyletic under almost all combinations of parameter values. Halteria and Amphiesmenoptera have high Bremer support values and long branch lengths. The data do not support placement of Strepsiptera outside of Holometabola nor as sister group to Coleoptera. We reject the notion that the monophyly of Halteria is due to long branch attraction because Strepsiptera and Diptera do not have the longest branches and there is phylogenetic congruence between molecules, across the entire parameter space, and between morphological and molecular data." publicationUrl="publication/31449100_The_Stresiptera_Problem_Phylogeny_of_the_Holometabolous_Insect_Orders_Inferred_from_18S_and_28S_Ribosomal_DNA_Sequences_and_Morphology" abstractClassName="js-target-abstract-5e40a6c4aa426"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/31449100_The_Stresiptera_Problem_Phylogeny_of_the_Holometabolous_Insect_Orders_Inferred_from_18S_and_28S_Ribosomal_DNA_Sequences_and_Morphology"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4aa426&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/21101482_Comparison_of_mitochondrial_DNA_sequences_of_seven_morphospecies_of_black_flies_Diptera_Simuliidae">Comparison of mitochondrial DNA sequences of seven morphospecies of black flies (Diptera: Simuliidae)</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">May 1991</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0831-2796_Genome">GENOME</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/5871538_Bai_Xiong"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Bai Xiong</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Thomas_Kocher5"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/279469894979593-1443642223647_Q64/Thomas_Kocher5.jpg" class="lite-image" alt="Thomas Kocher"/></picture></div><span class="nova-v-person-inline-item__fullname">Thomas Kocher</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4aa6e8 lite-page-hidden">Universal primers constructed from the 16S ribosomal RNA gene in the Drosophila yakuba mitochondrial genome were successfully used to amplify, via the polymerase chain reaction, the homologous region of mitochondrial DNA from seven black fly morphospecies. Amplification was achieved from single larval salivary glands and from single adults preserved in Carnoy&#x27;s fixative (ethanol - acetic acid, 3:1), allowing DNA sequences and polytene chromosome banding pattern data to be gathered from the same individuals. Nucleotide sequences of the amplified DNA segment (347 base pairs) were obtained from all the species examined. As in Drosophila, the nucleotide base composition of the sequenced segment from black flies had a high adenine (A) and thymine (T) content (A + T on average comprised 77% of all nucleotides.). Nucleotide differences among the seven species were observed at 59 positions (55 nucleotide substitutions and 4 deletions). There were more transversion differences than transition differences both among and within genera; the proportion of transversions was higher between genera than within genera. Most transversion differences were A----T type, comprising 79% of all transversion differences and 50% of all sequence differences. Phylogenetic inference based strictly on transversion differences confirmed traditional generic and tribal groupings, i.e., Prosimulium fuscum (Syme &amp; Davies) is close to Prosimulium magnum (Dyar &amp; Shannon); Simulium decorum (Walker), Simulium venustum s.l. (Say), and Simulium vittatum s.l. (Zetterstedt) are close to each other; Stegopterna mutata (Malloch) and Cnephia dacotensis (Dyar &amp; Shannon), which belong to the tribe Cnephiini, are grouped together.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Universal primers constructed from the 16S ribosomal RNA gene in the Drosophila yakuba mitochondrial genome were successfully used to amplify, via the polymerase chain reaction, the homologous region of mitochondrial DNA from seven black fly morphospecies. Amplification was achieved from single larval salivary glands and from single adults preserved in Carnoy&#x27;s fixative (ethanol - acetic acid, 3:1), allowing DNA sequences and polytene chromosome banding pattern data to be gathered from the same individuals. Nucleotide sequences of the amplified DNA segment (347 base pairs) were obtained from all the species examined. As in Drosophila, the nucleotide base composition of the sequenced segment from black flies had a high adenine (A) and thymine (T) content (A + T on average comprised 77% of all nucleotides.). Nucleotide differences among the seven species were observed at 59 positions (55 nucleotide substitutions and 4 deletions). There were more transversion differences than transition differences both among and within genera; the proportion of transversions was higher between genera than within genera. Most transversion differences were A----T type, comprising 79% of all transversion differences and 50% of all sequence differences. Phylogenetic inference based strictly on transversion differences confirmed traditional generic and tribal groupings, i.e., Prosimulium fuscum (Syme &amp; Davies) is close to Prosimulium magnum (Dyar &amp; Shannon); Simulium decorum (Walker), Simulium venustum s.l. (Say), and Simulium vittatum s.l. (Zetterstedt) are close to each other; Stegopterna mutata (Malloch) and Cnephia dacotensis (Dyar &amp; Shannon), which belong to the tribe Cnephiini, are grouped together." publicationUrl="publication/21101482_Comparison_of_mitochondrial_DNA_sequences_of_seven_morphospecies_of_black_flies_Diptera_Simuliidae" abstractClassName="js-target-abstract-5e40a6c4aa6e8"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/21101482_Comparison_of_mitochondrial_DNA_sequences_of_seven_morphospecies_of_black_flies_Diptera_Simuliidae"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4aa6e8&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/15316743_DNA_primers_for_amplification_of_mitochondrial_Cytochrome_C_oxidase_subunit_I_from_diverse_metazoan_invertebrates">DNA primers for amplification of mitochondrial Cytochrome C oxidase subunit I from diverse metazoan invertebrates</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Nov 1994</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1053-6426_Molecular_marine_biology_and_biotechnology">Mol Mar Biol Biotechnol</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Ole_Folmer"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/300293955833858-1448607066755_Q64/Ole_Folmer.jpg" class="lite-image" alt="Ole Folmer"/></picture></div><span class="nova-v-person-inline-item__fullname">Ole Folmer</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Michael_Black7"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/790072949497858-1565379478379_Q64/Michael_Black7.jpg" class="lite-image" alt="Michael B Black"/></picture></div><span class="nova-v-person-inline-item__fullname">Michael B Black</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2107801686_Hoeh_Wr"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Hoeh Wr</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Robert_Vrijenhoek"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/327719549194241-1455145837670_Q64/Robert_Vrijenhoek.jpg" class="lite-image" alt="Robert C Vrijenhoek"/></picture></div><span class="nova-v-person-inline-item__fullname">Robert C Vrijenhoek</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4aaaaa lite-page-hidden">We describe &quot;universal&quot; DNA primers for polymerase chain reaction (PCR) amplification of a 710-bp fragment of the mitochondrial cytochrome c oxidase subunit I gene (COI) from 11 invertebrate phyla: Echinodermata, Mollusca, Annelida, Pogonophora, Arthropoda, Nemertinea, Echiura, Sipuncula, Platyhelminthes, Tardigrada, and Coelenterata, as well as the putative phylum Vestimentifera. Preliminary comparisons revealed that these COI primers generate informative sequences for phylogenetic analyses at the species and higher taxonomic levels.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We describe &quot;universal&quot; DNA primers for polymerase chain reaction (PCR) amplification of a 710-bp fragment of the mitochondrial cytochrome c oxidase subunit I gene (COI) from 11 invertebrate phyla: Echinodermata, Mollusca, Annelida, Pogonophora, Arthropoda, Nemertinea, Echiura, Sipuncula, Platyhelminthes, Tardigrada, and Coelenterata, as well as the putative phylum Vestimentifera. Preliminary comparisons revealed that these COI primers generate informative sequences for phylogenetic analyses at the species and higher taxonomic levels." publicationUrl="publication/15316743_DNA_primers_for_amplification_of_mitochondrial_Cytochrome_C_oxidase_subunit_I_from_diverse_metazoan_invertebrates" abstractClassName="js-target-abstract-5e40a6c4aaaaa"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/15316743_DNA_primers_for_amplification_of_mitochondrial_Cytochrome_C_oxidase_subunit_I_from_diverse_metazoan_invertebrates"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4aaaaa&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/14616450_First_Molecular_Evidence_for_the_Existence_of_a_Tardigrada_Arthropoda_Clade">First Molecular Evidence for the Existence of a Tardigrada + Arthropoda Clade</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Feb 1996</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0737-4038_Molecular_Biology_and_Evolution">MOL BIOL EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Gonzalo_Giribet"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272480971522060-1441975934974_Q64/Gonzalo_Giribet.jpg" class="lite-image" alt="Gonzalo Giribet"/></picture></div><span class="nova-v-person-inline-item__fullname">Gonzalo Giribet</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Salvador_Carranza"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272680574255117-1442023524387_Q64/Salvador_Carranza.jpg" class="lite-image" alt="Salvador Carranza"/></picture></div><span class="nova-v-person-inline-item__fullname">Salvador Carranza</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Jaume_Baguna"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272493646708751-1441978956713_Q64/Jaume_Baguna.jpg" class="lite-image" alt="Jaume Baguna"/></picture></div><span class="nova-v-person-inline-item__fullname">Jaume Baguna</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Carles_Ribera"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273589064695817-1442240124672_Q64/Carles_Ribera.jpg" class="lite-image" alt="Carles Ribera"/></picture></div><span class="nova-v-person-inline-item__fullname">Carles Ribera</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4aae73 lite-page-hidden">The complete 18S rDNA gene sequence of Macrobiotus group hufelandi (Tardigrada) was obtained and aligned with 18S rDNA and rRNA gene sequences of 24 metazoans (mainly protostomes). Discrete character (maximum-parsimony) and distance (neighbor-joining) methods were used to infer their phylogeny. The evolution of bootstrap proportions with sequence length (pattern of resolved nodes, PRN) was studied to test the resolution of the nodes in neighbor-joining trees. The results show that arthropods are monophyletic. Tardigrades represent the sister group of arthropods (in parsimony analyses) or they are related with crustaceans (distance analysis and PRN). Arthropoda are divided into two main evolutionary lines, the Hexapoda + Crustacea line (weakly supported), and the Myriapoda + Chelicerata line. The Hexapoda + Crustacea line includes Pentastomida, but the internal resolution is far from clear. The Insecta (Ectognatha) are monophyletic, but no evidence for the monophyly of Hexapoda is found. The Chelicerata are a monophyletic group and the Myriapoda cluster close to Arachnida. Overall, the results obtained represent the first molecular evidence for a Tardigrada + Arthropoda clade. In addition, the congruence between molecular phylogenies of the Arthropoda from other authors and this obtained here indicates the need to review those obtained solely on morphological characters.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="The complete 18S rDNA gene sequence of Macrobiotus group hufelandi (Tardigrada) was obtained and aligned with 18S rDNA and rRNA gene sequences of 24 metazoans (mainly protostomes). Discrete character (maximum-parsimony) and distance (neighbor-joining) methods were used to infer their phylogeny. The evolution of bootstrap proportions with sequence length (pattern of resolved nodes, PRN) was studied to test the resolution of the nodes in neighbor-joining trees. The results show that arthropods are monophyletic. Tardigrades represent the sister group of arthropods (in parsimony analyses) or they are related with crustaceans (distance analysis and PRN). Arthropoda are divided into two main evolutionary lines, the Hexapoda + Crustacea line (weakly supported), and the Myriapoda + Chelicerata line. The Hexapoda + Crustacea line includes Pentastomida, but the internal resolution is far from clear. The Insecta (Ectognatha) are monophyletic, but no evidence for the monophyly of Hexapoda is found. The Chelicerata are a monophyletic group and the Myriapoda cluster close to Arachnida. Overall, the results obtained represent the first molecular evidence for a Tardigrada + Arthropoda clade. In addition, the congruence between molecular phylogenies of the Arthropoda from other authors and this obtained here indicates the need to review those obtained solely on morphological characters." publicationUrl="publication/14616450_First_Molecular_Evidence_for_the_Existence_of_a_Tardigrada_Arthropoda_Clade" abstractClassName="js-target-abstract-5e40a6c4aae73"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/14616450_First_Molecular_Evidence_for_the_Existence_of_a_Tardigrada_Arthropoda_Clade"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4aae73&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/12574362_Selection_of_Conserved_Blocks_from_Multiple_Alignments_for_Their_Use_in_Phylogenetic_Analysis">Selection of Conserved Blocks from Multiple Alignments for Their Use in Phylogenetic Analysis</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">May 2000</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0737-4038_Molecular_Biology_and_Evolution">MOL BIOL EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/46487715_J_Castresana"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">J. Castresana</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4aafe1 lite-page-hidden">The use of some multiple-sequence alignments in phylogenetic analysis, particularly those that are not very well conserved, requires the elimination of poorly aligned positions and divergent regions, since they may not be homologous or may have been saturated by multiple substitutions. A computerized method that eliminates such positions and at the same time tries to minimize the loss of informative sites is presented here. The method is based on the selection of blocks of positions that fulfill a simple set of requirements with respect to the number of contiguous conserved positions, lack of gaps, and high conservation of flanking positions, making the final alignment more suitable for phylogenetic analysis. To illustrate the efficiency of this method, alignments of 10 mitochondrial proteins from several completely sequenced mitochondrial genomes belonging to diverse eukaryotes were used as examples. The percentages of removed positions were higher in the most divergent alignments. After removing divergent segments, the amino acid composition of the different sequences was more uniform, and pairwise distances became much smaller. Phylogenetic trees show that topologies can be different after removing conserved blocks, particularly when there are several poorly resolved nodes. Strong support was found for the grouping of animals and fungi but not for the position of more basal eukaryotes. The use of a computerized method such as the one presented here reduces to a certain extent the necessity of manually editing multiple alignments, makes the automation of phylogenetic analysis of large data sets feasible, and facilitates the reproduction of the final alignment by other researchers.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="The use of some multiple-sequence alignments in phylogenetic analysis, particularly those that are not very well conserved, requires the elimination of poorly aligned positions and divergent regions, since they may not be homologous or may have been saturated by multiple substitutions. A computerized method that eliminates such positions and at the same time tries to minimize the loss of informative sites is presented here. The method is based on the selection of blocks of positions that fulfill a simple set of requirements with respect to the number of contiguous conserved positions, lack of gaps, and high conservation of flanking positions, making the final alignment more suitable for phylogenetic analysis. To illustrate the efficiency of this method, alignments of 10 mitochondrial proteins from several completely sequenced mitochondrial genomes belonging to diverse eukaryotes were used as examples. The percentages of removed positions were higher in the most divergent alignments. After removing divergent segments, the amino acid composition of the different sequences was more uniform, and pairwise distances became much smaller. Phylogenetic trees show that topologies can be different after removing conserved blocks, particularly when there are several poorly resolved nodes. Strong support was found for the grouping of animals and fungi but not for the position of more basal eukaryotes. The use of a computerized method such as the one presented here reduces to a certain extent the necessity of manually editing multiple alignments, makes the automation of phylogenetic analysis of large data sets feasible, and facilitates the reproduction of the final alignment by other researchers." publicationUrl="publication/12574362_Selection_of_Conserved_Blocks_from_Multiple_Alignments_for_Their_Use_in_Phylogenetic_Analysis" abstractClassName="js-target-abstract-5e40a6c4aafe1"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/12574362_Selection_of_Conserved_Blocks_from_Multiple_Alignments_for_Their_Use_in_Phylogenetic_Analysis"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4aafe1&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/11789875_Arthropod_phylogeny_based_on_eight_molecular_loci_and_morphology">Arthropod phylogeny based on eight molecular loci and morphology</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Oct 2001</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/0028-0836_Nature">NATURE</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Gonzalo_Giribet"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272480971522060-1441975934974_Q64/Gonzalo_Giribet.jpg" class="lite-image" alt="Gonzalo Giribet"/></picture></div><span class="nova-v-person-inline-item__fullname">Gonzalo Giribet</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Gregory_Edgecombe"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272474650705921-1441974427104_Q64/Gregory_Edgecombe.jpg" class="lite-image" alt="Gregory D Edgecombe"/></picture></div><span class="nova-v-person-inline-item__fullname">Gregory D Edgecombe</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Wc_Wheeler"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="W.C. Wheeler"/></picture></div><span class="nova-v-person-inline-item__fullname">W.C. Wheeler</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4ab1cc lite-page-hidden">The interrelationships of major clades within the Arthropoda remain one of the most contentious issues in systematics, which has traditionally been the domain of morphologists. A growing body of DNA sequences and other types of molecular data has revitalized study of arthropod phylogeny and has inspired new considerations of character evolution. Novel hypotheses such as a crustacean-hexapod affinity were based on analyses of single or few genes and limited taxon sampling, but have received recent support from mitochondrial gene order, and eye and brain ultrastructure and neurogenesis. Here we assess relationships within Arthropoda based on a synthesis of all well sampled molecular loci together with a comprehensive data set of morphological, developmental, ultrastructural and gene-order characters. The molecular data include sequences of three nuclear ribosomal genes, three nuclear protein-coding genes, and two mitochondrial genes (one protein coding, one ribosomal). We devised new optimization procedures and constructed a parallel computer cluster with 256 central processing units to analyse molecular data on a scale not previously possible. The optimal &#x27;total evidence&#x27; cladogram supports the crustacean-hexapod clade, recognizes pycnogonids as sister to other euarthropods, and indicates monophyly of Myriapoda and Mandibulata.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="The interrelationships of major clades within the Arthropoda remain one of the most contentious issues in systematics, which has traditionally been the domain of morphologists. A growing body of DNA sequences and other types of molecular data has revitalized study of arthropod phylogeny and has inspired new considerations of character evolution. Novel hypotheses such as a crustacean-hexapod affinity were based on analyses of single or few genes and limited taxon sampling, but have received recent support from mitochondrial gene order, and eye and brain ultrastructure and neurogenesis. Here we assess relationships within Arthropoda based on a synthesis of all well sampled molecular loci together with a comprehensive data set of morphological, developmental, ultrastructural and gene-order characters. The molecular data include sequences of three nuclear ribosomal genes, three nuclear protein-coding genes, and two mitochondrial genes (one protein coding, one ribosomal). We devised new optimization procedures and constructed a parallel computer cluster with 256 central processing units to analyse molecular data on a scale not previously possible. The optimal &#x27;total evidence&#x27; cladogram supports the crustacean-hexapod clade, recognizes pycnogonids as sister to other euarthropods, and indicates monophyly of Myriapoda and Mandibulata." publicationUrl="publication/11789875_Arthropod_phylogeny_based_on_eight_molecular_loci_and_morphology" abstractClassName="js-target-abstract-5e40a6c4ab1cc"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/11789875_Arthropod_phylogeny_based_on_eight_molecular_loci_and_morphology"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4ab1cc&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/11261438_Exploring_Data_Interaction_and_Nucleotide_Alignment_in_a_Multiple_Gene_Analysis_of_Ips_Coleoptera_Scolytinae">Exploring Data Interaction and Nucleotide Alignment in a Multiple Gene Analysis of Ips (Coleoptera: Scolytinae)</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Nov 2001</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1063-5157_Systematic_Biology">SYST BIOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Anthony_Cognato"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272360896987157-1441947306450_Q64/Anthony_Cognato.jpg" class="lite-image" alt="Anthony I Cognato"/></picture></div><span class="nova-v-person-inline-item__fullname">Anthony I Cognato</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Alfried_Vogler"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/791979591991296-1565834057121_Q64/Alfried_Vogler.jpg" class="lite-image" alt="Alfried Vogler"/></picture></div><span class="nova-v-person-inline-item__fullname">Alfried Vogler</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4ab423 lite-page-hidden">The possibility of gene tree incongruence in a species-level phylogenetic analysis of the genus Ips (Coleoptera: Scolytidae) was investigated based on mitochondrial 16S rRNA (16S) and nuclear elongation factor-1 alpha (EF-1 alpha) sequences, and existing cytochrome oxidase I (COI) and nonmolecular data sets. Separate cladistic analyses of the data partitions resulted in partially discordant most-parsimonious trees but revealed only low conflict of the phylogenetic signal. Interactions among data partitions, which differed in the extent of sequence divergence (COI &gt; 16S &gt; EF-1 alpha), base composition, and homoplasy, revealed that much of the branch support emerges only in the simultaneous analysis, particularly for deeper nodes in the tree, which are almost entirely supported through &quot;hidden support&quot; (sensu Gatesy et al., Cladistics 15:271-313, 1999). Apparent incongruence between data partitions is in part due to suboptimal alignments and bias of character transformations, but little evidence supports invoking incongruent phylogenetic histories of genetic loci. There is also no justification for eliminating or downweighting gene partitions on the basis of their apparent homoplasy or incongruence with other partitions, because the signal emerges only in the interaction of all data. In comparison with traditional taxonomy, the pini, plastographus, and perturbatus groups are polyphyletic, whereas the grandicollis group is monophyletic except for inclusion of the (monophyletic) calligraphus group. The latidens group and some European species are distantly related and closer to other genera within Ipini. Our robust cladogram was used to revise the classification of Ips. We provide new diagnoses for Ips and four subgeneric taxa.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="The possibility of gene tree incongruence in a species-level phylogenetic analysis of the genus Ips (Coleoptera: Scolytidae) was investigated based on mitochondrial 16S rRNA (16S) and nuclear elongation factor-1 alpha (EF-1 alpha) sequences, and existing cytochrome oxidase I (COI) and nonmolecular data sets. Separate cladistic analyses of the data partitions resulted in partially discordant most-parsimonious trees but revealed only low conflict of the phylogenetic signal. Interactions among data partitions, which differed in the extent of sequence divergence (COI &gt; 16S &gt; EF-1 alpha), base composition, and homoplasy, revealed that much of the branch support emerges only in the simultaneous analysis, particularly for deeper nodes in the tree, which are almost entirely supported through &quot;hidden support&quot; (sensu Gatesy et al., Cladistics 15:271-313, 1999). Apparent incongruence between data partitions is in part due to suboptimal alignments and bias of character transformations, but little evidence supports invoking incongruent phylogenetic histories of genetic loci. There is also no justification for eliminating or downweighting gene partitions on the basis of their apparent homoplasy or incongruence with other partitions, because the signal emerges only in the interaction of all data. In comparison with traditional taxonomy, the pini, plastographus, and perturbatus groups are polyphyletic, whereas the grandicollis group is monophyletic except for inclusion of the (monophyletic) calligraphus group. The latidens group and some European species are distantly related and closer to other genera within Ipini. Our robust cladogram was used to revise the classification of Ips. We provide new diagnoses for Ips and four subgeneric taxa." publicationUrl="publication/11261438_Exploring_Data_Interaction_and_Nucleotide_Alignment_in_a_Multiple_Gene_Analysis_of_Ips_Coleoptera_Scolytinae" abstractClassName="js-target-abstract-5e40a6c4ab423"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/11261438_Exploring_Data_Interaction_and_Nucleotide_Alignment_in_a_Multiple_Gene_Analysis_of_Ips_Coleoptera_Scolytinae"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4ab423&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/8667716_MUSCLE_Multiple_Sequence_Alignment_with_High_Accuracy_and_High_Throughput">MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Feb 2004</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1362-4962_Nucleic_Acids_Research">Nucleic Acids Res</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2140519230_Robert_C_Edgar"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Robert C Edgar</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4ab58e lite-page-hidden">We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences. Elements of the algorithm
include fast distance estimation using kmer counting, progressive alignment using a new profile function we call the logexpectation score, and refinement using treedependent
restricted partitioning. The speed and accuracy of MUSCLE are compared with TCoffee, MAFFT and CLUSTALW on four test sets
of reference alignments: BAliBASE, SABmark, SMART and a new benchmark, PREFAB. MUSCLE achieves the highest, or joint highest,
rank in accuracy on each of these sets. Without refinement, MUSCLE achieves average accuracy statistically indistinguishable
from TCoffee and MAFFT, and is the fastest of the tested methods for large numbers of sequences, aligning 5000 sequences
of average length 350 in 7 min on a current desktop computer. The MUSCLE program, source code and PREFAB test data are freely
available at http://www.drive5. com/muscle.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences. Elements of the algorithm
include fast distance estimation using kmer counting, progressive alignment using a new profile function we call the logexpectation score, and refinement using treedependent
restricted partitioning. The speed and accuracy of MUSCLE are compared with TCoffee, MAFFT and CLUSTALW on four test sets
of reference alignments: BAliBASE, SABmark, SMART and a new benchmark, PREFAB. MUSCLE achieves the highest, or joint highest,
rank in accuracy on each of these sets. Without refinement, MUSCLE achieves average accuracy statistically indistinguishable
from TCoffee and MAFFT, and is the fastest of the tested methods for large numbers of sequences, aligning 5000 sequences
of average length 350 in 7 min on a current desktop computer. The MUSCLE program, source code and PREFAB test data are freely
available at http://www.drive5. com/muscle." publicationUrl="publication/8667716_MUSCLE_Multiple_Sequence_Alignment_with_High_Accuracy_and_High_Throughput" abstractClassName="js-target-abstract-5e40a6c4ab58e"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/8667716_MUSCLE_Multiple_Sequence_Alignment_with_High_Accuracy_and_High_Throughput"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4ab58e&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/8149216_Phylogenetic_analysis_of_Myriapoda_using_three_nuclear_protein-coding_genes">Phylogenetic analysis of Myriapoda using three nuclear protein-coding genes</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Feb 2005</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1055-7903_Molecular_Phylogenetics_and_Evolution">MOL PHYLOGENET EVOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/38487195_Jerome_C_Regier"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">Jerome C. Regier</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2031674950_HEATHER_M_WILSON"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">HEATHER M. WILSON</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Jeffrey_Shultz2"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/647578119065602-1531406063736_Q64/Jeffrey_Shultz2.jpg" class="lite-image" alt="Jeffrey W Shultz"/></picture></div><span class="nova-v-person-inline-item__fullname">Jeffrey W Shultz</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4ab88c lite-page-hidden">We assessed the ability of three nuclear protein-encoding genes-elongation factor-1alpha (EF-1alpha), RNA polymerase II (Pol II), and elongation factor-2 (EF-2)-from 59 myriapod and 12 non-myriapod species to resolve phylogenetic relationships among myriapod classes and orders. In a previous study using EF-1alpha and Pol II (2134 nt combined) from 34 myriapod taxa, Regier and Shultz recovered widely accepted classes, orders, and families but failed to resolve interclass and interordinal relationships. The result was attributed to heterogenous rates of cladogenesis (specifically, the inability of the slowly evolving sequences to capture phylogenetic signal during rapid phylogenetic diversification) but the possibility of inadequate taxon sampling or limited sequence information could not be excluded. In the present study, the myriapod taxon sample was increased by 25 taxa (73%) and sequence length per taxon was effectively doubled through addition of EF-2 (4318 nt combined). Parsimony and Bayesian analyses of the expanded data set recovered a monophyletic Myriapoda, all four myriapod classes and all multiply sampled orders, often with high node support. However, except for three diplopod clades (Colobognatha, Helminothomorpha, and a subgroup of Pentazonia), few interordinal relationships and no interclass relationships were well supported. These results are similar to those of the earlier study by Regier and Shultz, which indicates that taxon sample and sequence length alone do not readily explain the weakly supported resolution in the earlier study. We review recent paleontological evidence to further develop our proposal that heterogeneity in phylogenetic signal provided by our slowly evolving sequences is due to heterogeneity in the temporal structure of myriapod diversification.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="We assessed the ability of three nuclear protein-encoding genes-elongation factor-1alpha (EF-1alpha), RNA polymerase II (Pol II), and elongation factor-2 (EF-2)-from 59 myriapod and 12 non-myriapod species to resolve phylogenetic relationships among myriapod classes and orders. In a previous study using EF-1alpha and Pol II (2134 nt combined) from 34 myriapod taxa, Regier and Shultz recovered widely accepted classes, orders, and families but failed to resolve interclass and interordinal relationships. The result was attributed to heterogenous rates of cladogenesis (specifically, the inability of the slowly evolving sequences to capture phylogenetic signal during rapid phylogenetic diversification) but the possibility of inadequate taxon sampling or limited sequence information could not be excluded. In the present study, the myriapod taxon sample was increased by 25 taxa (73%) and sequence length per taxon was effectively doubled through addition of EF-2 (4318 nt combined). Parsimony and Bayesian analyses of the expanded data set recovered a monophyletic Myriapoda, all four myriapod classes and all multiply sampled orders, often with high node support. However, except for three diplopod clades (Colobognatha, Helminothomorpha, and a subgroup of Pentazonia), few interordinal relationships and no interclass relationships were well supported. These results are similar to those of the earlier study by Regier and Shultz, which indicates that taxon sample and sequence length alone do not readily explain the weakly supported resolution in the earlier study. We review recent paleontological evidence to further develop our proposal that heterogeneity in phylogenetic signal provided by our slowly evolving sequences is due to heterogeneity in the temporal structure of myriapod diversification." publicationUrl="publication/8149216_Phylogenetic_analysis_of_Myriapoda_using_three_nuclear_protein-coding_genes" abstractClassName="js-target-abstract-5e40a6c4ab88c"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/8149216_Phylogenetic_analysis_of_Myriapoda_using_three_nuclear_protein-coding_genes"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4ab88c&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li><li class="nova-e-list__item publication-citations__item"><div class="nova-v-citation-item"><div class="nova-o-stack nova-o-stack--gutter-m nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="nova-v-citation-item__context-source"><div class="nova-v-publication-item nova-v-publication-item--size-m gtm-reference-item"><div class="nova-v-publication-item__body"><div class="nova-v-publication-item__stack nova-v-publication-item__stack--gutter-m"><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-2 nova-v-publication-item__title"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="publication/6185160_Improvement_of_Phylogenies_after_Removing_Divergent_and_Ambiguously_Aligned_Blocks_from_Protein_Sequence_Alignments">Improvement of Phylogenies after Removing Divergent and Ambiguously Aligned Blocks from Protein Sequence Alignments</a></div></div><div class="nova-v-publication-item__stack-item"><div class="nova-v-publication-item__meta"><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m nova-v-publication-item__type">Article</span></div><div class="nova-v-publication-item__meta-left"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-block nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m nova-v-publication-item__fulltext">Full-text available</span></div><div class="nova-v-publication-item__meta-right"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__meta-data"><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class="">Sep 2007</span></li><li class="nova-e-list__item nova-v-publication-item__meta-data-item"><span class=""><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="journal/1063-5157_Systematic_Biology">SYST BIOL</a></span></li></ul></div></div></div><div class="nova-v-publication-item__stack-item"><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-none nova-v-publication-item__person-list"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Gerard_Talavera"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/772507523096577-1561191554645_Q64/Gerard_Talavera.jpg" class="lite-image" alt="Gerard Talavera"/></picture></div><span class="nova-v-person-inline-item__fullname">Gerard Talavera</span></span></a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/46487715_J_Castresana"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname">J. Castresana</span></span></a></li></ul></div><div class="nova-v-publication-item__stack-item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit nova-e-text--clamp-3 nova-v-publication-item__description"><span class="js-target-abstract-5e40a6c4abb8b lite-page-hidden">Alignment quality may have as much impact on phylogenetic reconstruction as the phylogenetic methods used. Not only the alignment algorithm, but also the method used to deal with the most problematic alignment regions, may have a critical effect on the final tree. Although some authors remove such problematic regions, either manually or using automatic methods, in order to improve phylogenetic performance, others prefer to keep such regions to avoid losing any information. Our aim in the present work was to examine whether phylogenetic reconstruction improves after alignment cleaning or not. Using simulated protein alignments with gaps, we tested the relative performance in diverse phylogenetic analyses of the whole alignments versus the alignments with problematic regions removed with our previously developed Gblocks program. We also tested the performance of more or less stringent conditions in the selection of blocks. Alignments constructed with different alignment methods (ClustalW, Mafft, and Probcons) were used to estimate phylogenetic trees by maximum likelihood, neighbor joining, and parsimony. We show that, in most alignment conditions, and for alignments that are not too short, removal of blocks leads to better trees. That is, despite losing some information, there is an increase in the actual phylogenetic signal. Overall, the best trees are obtained by maximum-likelihood reconstruction of alignments cleaned by Gblocks. In general, a relaxed selection of blocks is better for short alignment, whereas a stringent selection is more adequate for longer ones. Finally, we show that cleaned alignments produce better topologies although, paradoxically, with lower bootstrap. This indicates that divergent and problematic alignment regions may lead, when present, to apparently better supported although, in fact, more biased topologies.</span></div></div></div></div><footer class="nova-v-publication-item__footer"><div class="nova-v-publication-item__footer-actions"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle"><div class="nova-o-pack__item nova-v-publication-item__footer-button-group"><div class="nova-c-button-group nova-c-button-group--gutter-m nova-c-button-group--orientation-horizontal nova-c-button-group--width-auto nova-v-publication-item__actions" abstract="Alignment quality may have as much impact on phylogenetic reconstruction as the phylogenetic methods used. Not only the alignment algorithm, but also the method used to deal with the most problematic alignment regions, may have a critical effect on the final tree. Although some authors remove such problematic regions, either manually or using automatic methods, in order to improve phylogenetic performance, others prefer to keep such regions to avoid losing any information. Our aim in the present work was to examine whether phylogenetic reconstruction improves after alignment cleaning or not. Using simulated protein alignments with gaps, we tested the relative performance in diverse phylogenetic analyses of the whole alignments versus the alignments with problematic regions removed with our previously developed Gblocks program. We also tested the performance of more or less stringent conditions in the selection of blocks. Alignments constructed with different alignment methods (ClustalW, Mafft, and Probcons) were used to estimate phylogenetic trees by maximum likelihood, neighbor joining, and parsimony. We show that, in most alignment conditions, and for alignments that are not too short, removal of blocks leads to better trees. That is, despite losing some information, there is an increase in the actual phylogenetic signal. Overall, the best trees are obtained by maximum-likelihood reconstruction of alignments cleaned by Gblocks. In general, a relaxed selection of blocks is better for short alignment, whereas a stringent selection is more adequate for longer ones. Finally, we show that cleaned alignments produce better topologies although, paradoxically, with lower bootstrap. This indicates that divergent and problematic alignment regions may lead, when present, to apparently better supported although, in fact, more biased topologies." publicationUrl="publication/6185160_Improvement_of_Phylogenies_after_Removing_Divergent_and_Ambiguously_Aligned_Blocks_from_Protein_Sequence_Alignments" abstractClassName="js-target-abstract-5e40a6c4abb8b"><div class="nova-c-button-group__item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" href="publication/6185160_Improvement_of_Phylogenies_after_Removing_Divergent_and_Ambiguously_Aligned_Blocks_from_Protein_Sequence_Alignments"><span class="nova-c-button__label">View</span></a></div><div class="nova-c-button-group__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-bare nova-c-button--width-auto nova-v-publication-item__action" type="button"><span class="nova-c-button__label"><span class=" js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-abstract-5e40a6c4abb8b&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;}]}">Show abstract</span></span></button></div></div></div></div></div></footer></div></div></div></div></div></li></div><li class="nova-e-list__item publication-citations__item publication-citations__item--load-more js-target-references-5e40a6c4a7614"><div class="nova-o-stack nova-o-stack--gutter-s nova-o-stack--spacing-none nova-o-stack--no-gutter-outside"><div class="nova-o-stack__item"><div class="publication-citations__more"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-xs nova-c-button--color-grey nova-c-button--theme-ghost nova-c-button--width-auto js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-references-5e40a6c4a7614&quot;,&quot;a&quot;:&quot;law(https://www.researchgate.net/lite.PublicationDetailsLoadMore.getReferencesByOffset.html?publicationUid=320321240&amp;offset=50)&quot;},{&quot;t&quot;:&quot;.js-target-references-5e40a6c4a7614&quot;,&quot;a&quot;:&quot;rc(publication-citations__item--load-more)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}"><span class="nova-c-button__label">Show more</span></button></div></div></div></li></ul></div></div></div></div></div></section></section><div id="lite-ad-yn5fqq3x1lg" class="lite-page-ad lite-page-ad--align-center lite-page-ad--has-label"><div class="lite-page-ad__inner"><div id="yn5fqq3x1lg"><rg-ad id="7ncbmlrwqoy" ad-unit="LoggedOut_PublicationWithoutFulltext_Middle" ad-unit-path="/83500759/LoggedOut_PublicationWithoutFulltext_Middle" ad-location="Middle" refresh-timeout="-1" load-on-init="false" collapse="never" class="rg-ad" ad-unit-sizes="[[728,90],[300,250],&quot;fluid&quot;]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="7ncbmlrwqoy" targeting-sequence="2"><rg-ad-size size="[728,90]"></rg-ad-size><rg-ad-size size="[300,250]"></rg-ad-size><rg-ad-size size="&quot;fluid&quot;"></rg-ad-size><rg-ad-sizeset viewport-size="[0,0]" slot-sizes="[]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[250,0]" slot-sizes="[[250,250]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[300,0]" slot-sizes="[[250,250],[300,250],[300,100],[300,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[320,0]" slot-sizes="[[250,250],[300,250],[300,100],[300,50],[320,100],[320,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[336,0]" slot-sizes="[[336,280],[300,250],[250,250],[300,100],[300,50],[320,100],[320,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[992,0]" slot-sizes="[[728,90],&quot;fluid&quot;]"></rg-ad-sizeset></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("7ncbmlrwqoy").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-yn5fqq3x1lg').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-yn5fqq3x1lg");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-yn5fqq3x1lg').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-yn5fqq3x1lg");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div></section><aside class="lite-page__side hide-l" role="complementary"><div id="lite-ad-t1po41m4ac" class="lite-page-ad lite-page-ad--align-left lite-page-ad--vertical-spacing-xl"><div class="lite-page-ad__inner"><div id="t1po41m4ac"><rg-ad id="ixfjuxewhub" ad-unit="Instream_Video" ad-unit-path="/83500759/Instream_Video" refresh-timeout="0" load-on-init="true" collapse="never" class="rg-ad" ad-unit-sizes="[[1,1]]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="ixfjuxewhub" targeting-sequence="1"><rg-ad-size size="[1,1]"></rg-ad-size></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("ixfjuxewhub").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-t1po41m4ac').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-t1po41m4ac");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-t1po41m4ac').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-t1po41m4ac");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div><div id="lite-ad-iskzqtlgkps" class="lite-page-ad lite-page-ad--align-left lite-page-ad--vertical-spacing-xl lite-page-ad--sticky lite-page-ad--has-label"><div class="lite-page-ad__inner"><div id="iskzqtlgkps"><rg-ad id="m3cg3nkpiqp" ad-unit="LoggedOut_PublicationWithoutFulltext_Right" ad-unit-path="/83500759/LoggedOut_PublicationWithoutFulltext_Right" ad-location="Right" refresh-timeout="-1" load-on-init="false" collapse="never" class="rg-ad" ad-unit-sizes="[[336,280],[300,250],[300,600],[160,600],[120,600],[320,100],[300,50],[300,100],[320,50],&quot;fluid&quot;]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="m3cg3nkpiqp" targeting-sequence="1"><rg-ad-size size="[336,280]"></rg-ad-size><rg-ad-size size="[300,250]"></rg-ad-size><rg-ad-size size="[300,600]"></rg-ad-size><rg-ad-size size="[160,600]"></rg-ad-size><rg-ad-size size="[120,600]"></rg-ad-size><rg-ad-size size="[320,100]"></rg-ad-size><rg-ad-size size="[300,50]"></rg-ad-size><rg-ad-size size="[300,100]"></rg-ad-size><rg-ad-size size="[320,50]"></rg-ad-size><rg-ad-size size="&quot;fluid&quot;"></rg-ad-size><rg-ad-sizeset viewport-size="[0,0]" slot-sizes="[]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[992,0]" slot-sizes="[[336,280],[300,250],[300,600],[160,600],[120,600],&quot;fluid&quot;]"></rg-ad-sizeset></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("m3cg3nkpiqp").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-iskzqtlgkps').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-iskzqtlgkps");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-iskzqtlgkps').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-iskzqtlgkps");elem && elem.classList.remove('lite-page-ad--empty');}}})</script><div class="lite-page-ad__fallback"><div class="lite-page-ad-fallback"><div class="publication-ad-fallback-promo"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-l nova-e-text--color-inherit">Join ResearchGate to find the people and research you need to help your work.</div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-unordered nova-e-list--spacing-auto"><li class="nova-e-list__item"><strong>15+ million</strong> members</li><li class="nova-e-list__item"><strong>118+ million</strong> publications</li><li class="nova-e-list__item"><strong>700k+</strong> research projects</li></ul><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-full" href="signup.SignUp.html"><span class="nova-c-button__label">Join for free</span></a></div></div></div></div></div></div></aside></main><aside class="lite-page__below" role="complementary"><div class="publication-recommendations"><div class="publication-recommendations__outer"><div class="publication-recommendations__inner"><div class="nova-o-grid nova-o-grid--gutter-m nova-o-grid--order-normal nova-o-grid--horizontal-align-left nova-o-grid--vertical-align-top"><div class="nova-o-grid__column nova-o-grid__column--width-12/12@s-up"><div class="nova-o-grid__column nova-o-grid__column--width-12/12@s-up nova-o-grid__column--width-6/12@m-up"><h2 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xxs nova-e-text--color-inherit">Recommended publications</h2></div><hr/></div></div><div class="nova-o-grid nova-o-grid--gutter-m nova-o-grid--order-normal nova-o-grid--horizontal-align-left nova-o-grid--vertical-align-top publication-recommendations__items"><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m">Full-text available</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/303252178_Maximum_Likelihood_Phylogenetic_Inference">Maximum Likelihood Phylogenetic Inference</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>December 2016</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/473322405273602-1489860263302_Q64/Amrit_Dhar.jpg" class="lite-image" alt="Amrit Dhar"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Amrit_Dhar">Amrit Dhar</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/39195174_Vladimir_N_Minin">Vladimir N. Minin</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">In this article, we provide an overview of maximum likelihood methods for phylogenetic inference. A brief introduction to general maximum likelihood estimation is provided. We define a phylogenetic likelihood, summarize how to compute this likelihood, and then discuss approaches used to maximize the phylogenetic likelihood function. We discuss a property of the maximum likelihood estimation, <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ac0de&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ac0de"> called consistency, that states that the maximum likelihood phylogeny will converge to the true phylogenetic tree with as more and more data are added to the analysis. We describe the bootstrap, a popular technique used to characterize the uncertainty in parameter estimates, and then outline its use in phylogenetic maximum likelihood estimation. A short example is given to illustrate the use of phylogenetic maximum likelihood techniques on a real dataset of primate mitochondrial DNA sequences.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/303252178_Maximum_Likelihood_Phylogenetic_Inference">View full-text</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/335660678_Mitochondrial_DNA_Phylogeny_of_Babblers_Timaliidae">Mitochondrial DNA Phylogeny of Babblers (Timaliidae)</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>January 2003</span>  <span>The Auk</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/391019301490688-1470237674138_Q64/Alice_Cibois.jpg" class="lite-image" alt="Alice Cibois"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Alice_Cibois">Alice Cibois</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">The systematics of the babblers (Timaliidae) and related members of the Old World insectivorous passerines have been particularly difficult. To clarify our understanding of this group, phylogenetic relationships were constructed using sequences of three mitochondrial genes (cytochrome b, rRNA 12S and 16S). The results indicated that several species traditionally placed among babblers, the shrike <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ac34a&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ac34a"> babblers (Pteruthius) and the Gray-chested Thrush Babbler (Kakamega poliothorax), are not related to the Timaliidae, but belong to other passerine groups. Furthermore, the phylogenetic hypotheses inferred from molecular data suggest that the babblers assemblage includes two other oscine taxa traditionally considered to be distantly related, Sylvia (Sylviidae) and Zosterops (Zosteropidae). The polyphyly of several babbler genera is discussed, with particular attention to the laughingthrushes (genera Garrulax and Babax) for which the phylogeny is compared to previous hypotheses of relationships. Results from different tests under the maximum-parsimony and maximum-likelihood criteria indicate the rejection of the hypothesis of monophyly for the laughingthrushes group. Thus, the molecular phylogeny challenges the traditional classification of the Timaliidae.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/335660678_Mitochondrial_DNA_Phylogeny_of_Babblers_Timaliidae">Read more</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m">Full-text available</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/336095200_The_complete_chloroplast_genomes_of_Trollius_farreri_and_Anemone_taipaiensis_Ranunculaceae">The complete chloroplast genomes of Trollius farreri and Anemone taipaiensis (Ranunculaceae)</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>July 2019</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/811973851758592-1570601060009_Q64/Ben_Yu14.jpg" class="lite-image" alt="Ben Yu"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Ben_Yu14">Ben Yu</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2125854423_Ruiting_Zhang">Ruiting Zhang</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2152089297_Qian_Yang">Qian Yang</a></span></span></li><li class="nova-e-list__item">[...]</li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/836300043214848-1576400876694_Q64/Zhan_Lin_Liu.jpg" class="lite-image" alt="Zhan-Lin Liu"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Zhan_Lin_Liu">Zhan-Lin Liu</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Ranunculaceae, with high morphological biodiversity and widely ecological amplitude tolerance, has been considered as a new good model for adaptive evolution study. In this study, two high-quality chloroplast genomes of Ranunculaceae were determined by next-generation sequencing technologies. The plastomes of Trollius farreri and Anemone taipaiensis exhibit a conserved quadripartite structure, <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ac570&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ac570"> with 160,612 bp and 160,214 bp in length, presenting similar GC contents (38% and 37.6%). The chloroplast genome of Trollius farreri contains 131 genes, including 87 protein-coding genes, 8 rRNA, and 36 tRNA genes, while the plastome of Anemone taipaiensis harbours 137 genes, including 93 protein-coding genes, 8 rRNA, and 36 tRNA genes. Phylogenetic tree constructed with 21 plastomes of Ranunculaceae species indicates that Trollius farreri and Anemone taipaiensis are closely related to their congeneric species, respectively. The phylogenetic relationships are inconsistent with the current classification of Ranunculaceae.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/336095200_The_complete_chloroplast_genomes_of_Trollius_farreri_and_Anemone_taipaiensis_Ranunculaceae">View full-text</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m">Full-text available</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/235721049_Revision_and_phylogenetic_analysis_of_the_Papuan_weevil_genus_Thyestetha_Pascoe_Coleoptera_Curculionidae_Cryptorhynchinae">Revision and phylogenetic analysis of the Papuan weevil genus Thyestetha Pascoe (Coleoptera, Curculi...</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>June 2012</span>  <span>Zootaxa</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/687036080066566-1540813574480_Q64/Jonas_Eberle2.jpg" class="lite-image" alt="Jonas Eberle"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Jonas_Eberle2">Jonas Eberle</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272712652292122-1442031171740_Q64/Rene_Taenzler.jpg" class="lite-image" alt="Rene Tnzler"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Rene_Taenzler">Rene Tnzler</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/279113874067460-1443557341624_Q64/Alexander_Riedel2.jpg" class="lite-image" alt="Alexander Riedel"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Alexander_Riedel2">Alexander Riedel</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Thyestetha Pascoe, an apterous genus of Papuan Cryptorhynchinae is revised. It contains two previously described species, T. carbonaria Faust and T. nitida Pascoe, plus seven new species: T. discolor sp. n., T. emarginata sp. n., T. glabra sp. n., T. lubrica sp. n., T. oblita sp. n., T. puncticollis sp. n., and T. splendens sp. n. Lectotypes are designated for T. carbonaria, T. nitida, and <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ac952&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ac952"> Pantoxistus rubripennis Chevrolat which is a junior synonym of T. nitida (syn. n.). The genus and its species are described and the characters relevant for their identification are illustrated. A key to the species is provided. Phylogenetic analyses are performed, utilizing molecular and morphological approaches. A maximum likelihood tree including 5 Thyestetha species and 2 outgroups is presented, based on 3 mitochondrial and 4 nuclear markers. A maximum parsimony analysis of 19 morphological characters includes all described species. Thyestetha is presumably closely related to Telaugia Pascoe and Elichora Pascoe. The group is restricted to New Guinea and some smaller neighboring islands.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/235721049_Revision_and_phylogenetic_analysis_of_the_Papuan_weevil_genus_Thyestetha_Pascoe_Coleoptera_Curculionidae_Cryptorhynchinae">View full-text</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Data</span></div><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m">Full-text available</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/293872512_Additional_file_2">Additional file 2</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>September 2010</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/438540891889666-1481567704929_Q64/Tommi_Nyman.jpg" class="lite-image" alt="Tommi Nyman"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Tommi_Nyman">Tommi Nyman</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Veli Vikberg"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Veli_Vikberg">Veli Vikberg</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/277741204197383-1443230071618_Q64/David_Smith114.jpg" class="lite-image" alt="David R. Smith"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/David_Smith114">David R. Smith</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272768876937254-1442044576936_Q64/Jean-Luc_Boeve.jpg" class="lite-image" alt="Jean-Luc Boev"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Jean-Luc_Boeve">Jean-Luc Boev</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Sequence data used in phylogeny reconstruction and resultant phylogenetic trees. NEXUS file containing the data matrix and trees obtained from the Bayesian phylogenetic analyses in MrBayes and BEAST, and the maximum-likelihood tree from the analysis using RAxML. </div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/293872512_Additional_file_2">View full-text</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/267238016_Sequence_Comparison_of_Mitochondrial_Small_Subunit_Ribosomal_DNA_in_Penicillium">Sequence Comparison of Mitochondrial Small Subunit Ribosomal DNA in Penicillium</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>March 2000</span>  <span>The Journal of Microbiology</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/278492626341888-1443409224738_Q64/Soon_Gyu_Hong.jpg" class="lite-image" alt="Soon Gyu Hong"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Soon_Gyu_Hong">Soon Gyu Hong</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/540432145043457-1505860472846_Q64/Yoon-Dong_Park.jpg" class="lite-image" alt="Yoon-Dong Park"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Yoon-Dong_Park">Yoon-Dong Park</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2081947991_Wonjin_Jeong">Wonjin Jeong</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://c5.rgstatic.net/m/4671872220764/images/template/default/profile/profile_default_m.jpg" class="lite-image" alt="Kyung Sook Bae"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Kyung_Sook_Bae">Kyung Sook Bae</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Partial sequence comparisons of mitochondrial small subunit rDNA (mt SSU rDNA) were used to examine taxonomic and evolutionary relationships among seven Penicillium species: two monoverticillate species, two biverticillate species, and three terverticillate species. Amplified fragments of mt SSU rDNA highly varied among seven species in size, suggesting the existence of multiple insertions or <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ad094&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ad094"> deletions in the region. A phylogenetic tree was constructed by exhaustive search of parsimony analysis. The phylogenetic tree distinguished two statistically supported monophyletic groups, one for two monoverticillate species and the other for three terverticillate species and one biverticillate species, P. vulpinum. The phylogenetic relationship of P. waksmanii, the biverticillate species, was not clear.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/267238016_Sequence_Comparison_of_Mitochondrial_Small_Subunit_Ribosomal_DNA_in_Penicillium">Read more</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Chapter</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/278701086_Springer_Theses">Springer Theses</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>January 2013</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/16224484_Rituparna_Bose">Rituparna Bose</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">The phylogeny and taxonomy of atrypides as proposed in the past has not been tested in terms of morphometric shape. Here, we investigated external shell shape variation in brachiopod morphology at the subfamily and generic level using geometric morphometrics. We measured valve shape in 1593 atrypid individuals from Silurian-Devonian time intervals from 8 EE subunits from 18 geographic localities <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ad529&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ad529"> in eastern North America. The following representatives of the Atrypida were included in the morphometric analyses: Atrypa, Gotatrypa, Kyrtatrypa, Oglupes?, Joviatrypa, Endrea, Dihelictera (Atrypinae); Pseudoatrypa (Variatrypinae) and Spinatrypa (Spinatrypinae).We used 8 external landmarks to determine shape differences among genera and subfamilies in time and space and to calculate pairwise distances between them. Phylogenetic divergence time was determined between atrypid generic pairs based on the phylogenetic tree published in prior literature. Maximum-likelihood was used to assess evolutionary rate and mode of morphological divergence. Results indicate that morphological divergences among these genera are very small compared to their within-genus variation. Thus, while morphometric differentiation is concordant with phylogeny proposed in the past, the small shell shape distances between genera, considerable morphological overlap between subfamilies, considerable variation within one subfamily, and greater morphological variation within genus, suggest that other characteristics such as ribbing, growth lamellae, pedicle opening, etc. prove to be more useful for distinguishing genera in atrypid brachiopods. Thus, a combination of quantified shape, external and internal morphological characters is essential for future phylogenetic classification in order to understand the evolutionary ecology of these complex organisms in its entirety.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/278701086_Springer_Theses">Read more</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/281533273_Methods_and_algorithms_for_a_statistical_approach_in_phylogenetics">Methods and algorithms for a statistical approach in phylogenetics</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>July 2003</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/2080578795_Stephane_Guindon">Stephane Guindon</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Variation of substitution rates across sites is widespread among biological sequences. A gamma distribution, defined by a shape parameter, is generally used to model this phenomenon. We propose here a new method to estimate an efficient value of the gamma shape parameter, i.e., the value that is best suited for tree topology estimation. We show that (1) efficient values lead to underestimate the <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ad672&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ad672"> rate variability and (2) the tree topologies that are obtained are more accurate than those deduced from the true (unknown) values of the parameter. Exploring the tree space is another important issue in phylogenetic inference. We propose a new approach for building maximum likelihood phylogenies. The core of this method is a simple hill climbing algorithm that adjusts tree topology and branch lengths simultaneously. We show that this approach reconstructs very accurate tree topologies from data sets containing hundreds of taxa. The speed of this method also greatly facilitates bootstrap analysis.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/281533273_Methods_and_algorithms_for_a_statistical_approach_in_phylogenetics">Read more</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Conference Paper</span></div><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m">Full-text available</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/307639479_Phylogenetic_position_of_Helicotylenchus_dihystera_from_South_Africa_using_28S_rDNA">Phylogenetic position of Helicotylenchus dihystera from South Africa using 28S rDNA</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>August 2016</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/301189448126464-1448820568142_Q64/Ebrahim_Shokoohi.jpg" class="lite-image" alt="Ebrahim Shokoohi"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Ebrahim_Shokoohi">Ebrahim Shokoohi</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/775416419721217-1561885089034_Q64/Milad_Rashidifard4.jpg" class="lite-image" alt="Milad Rashidifard"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Milad_Rashidifard4">Milad Rashidifard</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/273632597377024-1442250503023_Q64/Hendrika_Fourie.jpg" class="lite-image" alt="Hendrika Fourie"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Hendrika_Fourie">Hendrika Fourie</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">During a project on Helicotylenchus phylogeny in South Africa, H. dihystera identified morphologically and molecularly in Potchefstroom. Concern to the morphological characters, this population fit well with the other population studied previously. D2-D3 region of 28S rDNA amplified for study of its phylogenetic position. Nblast result showed close relationship with H. dihystera from China <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ad7ab&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ad7ab"> (KF443217, KF486503; 98% identity) respectively. Phylogenetic analysis revealed that the population studied places together with H. dihystera from USA (HM014244; HM014255). In addition, phylogenetic tree based on Maximum Likelihood indicated that H. dihystera form a well-supported clade (99% bootstrap values).</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/307639479_Phylogenetic_position_of_Helicotylenchus_dihystera_from_South_Africa_using_28S_rDNA">View full-text</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/2883592_Phylogenetic_Inference_from">Phylogenetic Inference from</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>April 2002</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/7126430_William_Noble_Grundy">William Noble Grundy</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/272205296697352-1441910208402_Q64/Gavin_Naylor.jpg" class="lite-image" alt="Gavin Naylor"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Gavin_Naylor">Gavin Naylor</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Molecular sequences provide a rich source of data for inferring the phylogenetic relationships among species. However, recent work indicates that even an accurate multiple alignment of a large sequence set may yield an incorrect phylogeny, and that the quality of the phylogenetic tree improves when the input consists only of the highly-conserved, motif regions of the alignment. This work <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4ada6c&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4ada6c"> introduces two methods of producing multiple alignments that include only the conserved regions of the initial alignment. The first method retains conserved motifs, whereas the second retains individual conserved sites in the initial alignment. Using parsimony analysis on a mitochondrial data set containing nineteen species among which the phylogenetic relationships are widely accepted, both conserved alignment methods produce better phylogenetic trees than the complete alignment. Unlike any of the nineteen inference methods used previously to analyze this data, both methods produce trees that are completely consistent with the known phylogeny. The motif-based method, on the other hand, employs far fewer alignment sites for comparable error rates. For a larger data set containing mitochondrial sequences from 39 species, the site-based method produces a phylogenetic tree that is largely consistent with known phylogenetic relationships and which suggests several novel placements. 1</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/2883592_Phylogenetic_Inference_from">Read more</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/254007339_TreeLign_Simultaneous_stepwise_alignment_and_phylogenetic_positioning_with_its_application_to_automatic_phylogenetic_assignment_of_16S_rRNAs">TreeLign: Simultaneous stepwise alignment and phylogenetic positioning, with its application to auto...</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>August 2011</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/58918066_Yuan_Li">Yuan Li</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/38223444_Aaron_L_Halpern">Aaron L. Halpern</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/794953731022859-1566543147958_Q64/Shaojie_Zhang9.jpg" class="lite-image" alt="Shaojie Zhang"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Shaojie_Zhang9">Shaojie Zhang</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">Phylogenetic assignment of 16s rRNA has been frequently used for taxonomic classification. Recently, high-throughput sequencing, especially in the context of environmental or metagenomic sequencing projects, has made fast and accurate taxonomic classification an important goal. Existing classification methods are either fast, but too coarse-grained and inaccurate or fine-grained and accurate but <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4adcbf&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4adcbf"> too slow for use in practice. In this paper, we propose a new computational method, TreeLign, to rapidly and accurately conduct alignment and phylogenetic assignments for novel sequences, given a reference phylogenetic tree and an alignment. TreeLign first constructs profiles of every branch on the reference tree, then, for each query sequence, tries assigning it to every possible branch, and finally obtains a new tree and a new alignment which are jointly optimal in terms of Maximum Parsimony (MP). We tested the accuracy and robustness of TreeLign on both a large and a small 16S rRNA dataset extracted from the core set of GreenGenes. The results on the large dataset show that the assignments of TreeLign are in general consistent with the phylogenetic tree of the core set of GreenGenes. And, the results on the small dataset show that TreeLign achieves comparable accuracy compared with existing maximum likelihood based methods, but requires much less computational time.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/254007339_TreeLign_Simultaneous_stepwise_alignment_and_phylogenetic_positioning_with_its_application_to_automatic_phylogenetic_assignment_of_16S_rRNAs">Read more</a></div></div><div class="nova-o-grid__column nova-o-grid__column--width-6/12@m-up publication-recommendation-item__item"><div class="publication-recommendation-item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxs nova-l-flex--direction-row@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up publication-recommendation-item__item-badges"><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-high nova-e-badge--size-l nova-e-badge--theme-solid nova-e-badge--radius-m">Article</span></div><div class="nova-l-flex__item"><span class="nova-e-badge nova-e-badge--color-green nova-e-badge--display-inline nova-e-badge--luminosity-medium nova-e-badge--size-l nova-e-badge--theme-ghost nova-e-badge--radius-m">Full-text available</span></div></div><h3 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-inherit"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare gtm-recommendation-item-link" href="publication/7656562_Predicting_Functional_Gene_Links_from_Phylogenetic-Statistical_Analyses_of_Whole_Genomes">Predicting Functional Gene Links from Phylogenetic-Statistical Analyses of Whole Genomes</a></h3><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-xs nova-e-text--color-grey-500"><span>June 2005</span>  <span>PLoS Computational Biology</span></div><ul class="nova-e-list nova-e-list--size-m nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/scientific-contributions/16633630_D_Barker">D. Barker</a></span></span></li><li class="nova-e-list__item"><span class="nova-v-person-inline-item"><div class="lite-person-inline-item__image"><picture class="lite-page-avatar lite-page-avatar--size-xs lite-page-avatar--radius-full"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN89h8AAtEB5wrzxXEAAAAASUVORK5CYII" data-src="https://i1.rgstatic.net/ii/profile.image/314169251827713-1451915194379_Q64/Mark_Pagel2.jpg" class="lite-image" alt="Mark Pagel"/></picture></div><span class="nova-v-person-inline-item__fullname"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="https://www.researchgate.net/profile/Mark_Pagel2">Mark Pagel</a></span></span></li></ul><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-grey-500">An important element of the developing field of proteomics is to understand protein-protein interactions and other functional links amongst genes. Across-species correlation methods for detecting functional links work on the premise that functionally linked proteins will tend to show a common pattern of presence and absence across a range of genomes. We describe a maximum likelihood statistical <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-5e40a6c4adfb2&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;},{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">... [Show full abstract]</a><span class="lite-page-hidden js-target-5e40a6c4adfb2"> model for predicting functional gene linkages. The method detects independent instances of the correlated gain or loss of pairs of proteins on phylogenetic trees, reducing the high rates of false positives observed in conventional across-species methods that do not explicitly incorporate a phylogeny. We show, in a dataset of 10,551 protein pairs, that the phylogenetic method improves by up to 35% on across-species analyses at identifying known functionally linked proteins. The method shows that protein pairs with at least two to three correlated events of gain or loss are almost certainly functionally linked. Contingent evolution, in which one gene&#x27;s presence or absence depends upon the presence of another, can also be detected phylogenetically, and may identify genes whose functional significance depends upon its interaction with other genes. Incorporating phylogenetic information improves the prediction of functional linkages. The improvement derives from having a lower rate of false positives and from detecting trends that across-species analyses miss. Phylogenetic methods can easily be incorporated into the screening of large-scale bioinformatics datasets to identify sets of protein links and to characterise gene networks.</span></div><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-silent publication-recommendation-item__more" href="publication/7656562_Predicting_Functional_Gene_Links_from_Phylogenetic-Statistical_Analyses_of_Whole_Genomes">View full-text</a></div></div></div><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-auto" href="search"><span class="nova-c-button__label">Discover more</span></a></div></div></div><div id="lite-ad-r8lkc0045nj" class="lite-page-ad lite-page-ad--align-center lite-page-ad--has-label"><div class="lite-page-ad__inner"><div id="r8lkc0045nj"><rg-ad id="h1xiqcreval" ad-unit="LoggedOut_PublicationWithFulltext_Middle" ad-unit-path="/83500759/LoggedOut_PublicationWithFulltext_Middle" ad-location="Middle" refresh-timeout="-1" load-on-init="false" collapse="never" class="rg-ad" ad-unit-sizes="[[728,90],[300,250],&quot;fluid&quot;]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="h1xiqcreval" targeting-sequence="2"><rg-ad-size size="[728,90]"></rg-ad-size><rg-ad-size size="[300,250]"></rg-ad-size><rg-ad-size size="&quot;fluid&quot;"></rg-ad-size><rg-ad-sizeset viewport-size="[0,0]" slot-sizes="[]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[250,0]" slot-sizes="[[250,250]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[300,0]" slot-sizes="[[250,250],[300,250],[300,100],[300,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[320,0]" slot-sizes="[[250,250],[300,250],[300,100],[300,50],[320,100],[320,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[336,0]" slot-sizes="[[336,280],[300,250],[250,250],[300,100],[300,50],[320,100],[320,50]]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[992,0]" slot-sizes="[]"></rg-ad-sizeset></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("h1xiqcreval").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-r8lkc0045nj').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-r8lkc0045nj");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-r8lkc0045nj').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-r8lkc0045nj");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div><form method="GET" action="https://www.researchgate.net/lite.publication.PublicationDownloadCitationModal.downloadCitation.html"><div id="download-citation" class="lite-page-modal lite-page-modal--hidden lite-page-modal lite-page-modal--enable-click-outside"><span><div class="nova-c-modal nova-c-modal--color-green nova-c-modal--position-center nova-c-modal--spacing-xxl nova-c-modal--width-s" style="z-index:9998"><div class="nova-c-modal__overlay"></div><div class="nova-c-modal__container"><div class="nova-c-modal__window" role="dialog" aria-modal="true" tabindex="-1"><header class="nova-c-modal__header nova-c-modal__header--spacing-none lite-page-modal__header"><div class="nova-c-modal__header-content"><button class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-grey nova-c-button--theme-bare nova-c-button--width-square lite-page-modal__close lite-page-modal__close--top-right" type="button"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#close-circle-s"></use></svg></button></div><div class="nova-c-modal__header-image"></div></header><div class="nova-c-modal__body nova-c-modal__body--spacing-inherit"><div class="nova-c-modal__body-content"><div class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-l nova-e-text--color-inherit">Download citation</div><div class="nova-l-form-group nova-l-form-group--layout-stack nova-l-form-group--gutter-s"><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><label class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-xxs nova-e-text--color-inherit nova-e-label" for=""><span class="nova-e-label__text">What type of file do you want?</span></label><div class="nova-l-form-group nova-l-form-group--layout-stack nova-l-form-group--gutter-s"><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div><label class="nova-e-radio"><input type="radio" class="nova-e-radio__input" name="fileType" value="RIS" checked=""/><span class="nova-e-radio__label"> RIS</span></label></div></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div><label class="nova-e-radio"><input type="radio" class="nova-e-radio__input" name="fileType" value="BibTeX"/><span class="nova-e-radio__label"> BibTeX</span></label></div></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div><label class="nova-e-radio"><input type="radio" class="nova-e-radio__input" name="fileType" value="Plain Text"/><span class="nova-e-radio__label"> Plain Text</span></label></div></div></div></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><label class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-xxs nova-e-text--color-inherit nova-e-label" for=""><span class="nova-e-label__text">What do you want to download?</span></label><div class="nova-l-form-group nova-l-form-group--layout-stack nova-l-form-group--gutter-s"><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div><label class="nova-e-radio"><input type="radio" class="nova-e-radio__input" name="citation" value="citation" checked=""/><span class="nova-e-radio__label"> Citation only</span></label></div></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div><label class="nova-e-radio"><input type="radio" class="nova-e-radio__input" name="citation" value="citationAndAbstract"/><span class="nova-e-radio__label"> Citation and abstract</span></label></div></div></div></div></div><input type="hidden" name="publicationUid" value="320321240"/></div></div><footer class="nova-c-modal__footer nova-c-modal__footer--align-right nova-c-modal__footer--spacing-inherit"><div class="nova-c-modal__footer-content"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-auto"><span class="nova-c-button__label">Download</span></button></div></footer><div class="nova-e-spinner nova-e-spinner--size-m nova-e-spinner--color-inherit nova-c-modal__spinner"><div class="nova-e-spinner__container"><svg width="16" height="16" viewBox="0 0 16 16" class="nova-e-spinner__asset"><g stroke-width="2"><path class="nova-e-spinner__circle" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path><path class="nova-e-spinner__stroke" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path></g></svg></div></div></div></div></div></span></div></form><div id="exit-modal" class="lite-page-modal lite-page-modal--hidden lite-page-modal lite-page-modal--enable-click-outside"><span><div class="nova-c-modal nova-c-modal--color-green nova-c-modal--position-center nova-c-modal--spacing-xl nova-c-modal--width-s" style="z-index:9998"><div class="nova-c-modal__overlay"></div><div class="nova-c-modal__container"><div class="nova-c-modal__window" role="dialog" aria-modal="true" tabindex="-1"><header class="nova-c-modal__header nova-c-modal__header--spacing-none lite-page-modal__header"><div class="nova-c-modal__header-content"><button class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-grey nova-c-button--theme-bare nova-c-button--width-square lite-page-modal__close lite-page-modal__close--top-right" type="button"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#close-circle-s"></use></svg></button></div><div class="nova-c-modal__header-image"></div></header><div class="nova-c-modal__body nova-c-modal__body--spacing-inherit"><div class="nova-c-modal__body-content"><h2 class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-auto nova-e-text--color-inherit">Looking for the full-text?</h2><p class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-l nova-e-text--color-inherit">You can request the full-text of this article directly from the authors on ResearchGate.</p><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-full js-lite-click" type="button" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.lite-page&quot;,&quot;a&quot;:&quot;ft(https://www.researchgate.net/lite.publication.PublicationRequestFulltextPromo.requestFulltext.html?publicationUid=320321240&amp;ev=su_requestFulltext)&quot;}]}"><span class="nova-c-button__label">Request full-text</span></button></div></div><footer class="nova-c-modal__footer nova-c-modal__footer--align-right nova-c-modal__footer--spacing-inherit"><div class="nova-c-modal__footer-content"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit">Already a member? <a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="login">Log in</a></div></div></footer><div class="nova-e-spinner nova-e-spinner--size-m nova-e-spinner--color-inherit nova-c-modal__spinner"><div class="nova-e-spinner__container"><svg width="16" height="16" viewBox="0 0 16 16" class="nova-e-spinner__asset"><g stroke-width="2"><path class="nova-e-spinner__circle" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path><path class="nova-e-spinner__stroke" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path></g></svg></div></div></div></div></div></span></div><div class="mobile-catfish-adslot"></div></aside><header class="lite-page__header" role="banner"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-none nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up lite-page__header-wrapper"><div class="nova-l-flex__item"><a href="https://www.researchgate.net/" rel="home" class="lite-page__header-logo-link"><svg width="124px" height="17px" viewBox="0 0 124 17" class="brand"><g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="Group" fill="#FFFFFF"><path d="M34.6035,8.8901 C34.8295,7.6121 35.6215,6.2431 37.2125,6.2431 C38.6015,6.2431 39.2335,7.4921 39.2335,8.6521 C39.2335,8.8391 39.2035,8.8901 38.9865,8.8901 L34.6035,8.8901 L34.6035,8.8901 Z M37.2995,4.7681 C34.3845,4.7681 32.2695,7.3331 32.2695,10.8691 C32.2695,14.4661 34.1895,16.6151 37.4075,16.6151 C39.1285,16.6151 40.5185,15.8441 40.9525,15.1191 L40.9965,15.0461 L40.8795,14.0491 L40.5595,14.2131 C39.7905,14.6071 38.8675,14.8071 37.8185,14.8071 C35.3555,14.8071 34.4785,12.6691 34.4785,10.6691 C34.4785,10.5451 34.4785,10.4121 34.4835,10.2771 L41.4505,10.2771 L41.4825,10.0641 C41.5285,9.7571 41.5285,9.5241 41.5285,9.2281 C41.5285,7.0801 40.2055,4.7681 37.2995,4.7681 L37.2995,4.7681 L37.2995,4.7681 Z" id="Fill-1"></path><path d="M110.4365,2.4414 L109.3025,2.4414 L109.2525,2.6264 C108.9195,3.8514 108.4855,4.6564 107.1455,5.3974 L107.0165,5.4684 L107.0165,6.5754 L108.3355,6.5754 L108.3355,13.7494 C108.3355,15.7584 109.1165,16.6144 110.9435,16.6144 C112.0345,16.6144 113.1645,16.1514 113.5155,15.5624 L113.5645,15.4804 L113.4045,14.6554 L113.1285,14.7554 C112.8635,14.8534 112.2225,15.0064 111.6355,15.0064 C110.8585,15.0064 110.4365,14.7454 110.4365,13.3724 L110.4365,6.5754 L113.5725,6.5754 L113.5725,5.0344 L110.4365,5.0344 L110.4365,2.4414 L110.4365,2.4414 Z" id="Fill-3"></path><path d="M48.0273,11.4136 C48.0063,12.0256 47.9743,13.2816 47.9713,14.0086 C47.5913,14.4166 46.8923,14.8286 45.9933,14.8286 C45.0533,14.8286 44.4703,14.2536 44.4703,13.3286 C44.4703,12.0536 45.4703,11.4066 47.4433,11.4066 L47.4723,11.4066 C47.6313,11.4066 47.8403,11.4066 48.0273,11.4136 M50.0933,13.9486 C50.0933,12.9186 50.1163,11.9706 50.1393,11.0576 C50.1603,10.2016 50.1803,9.3766 50.1803,8.5416 C50.1803,6.1436 48.7563,4.7676 46.2753,4.7676 C45.0083,4.7676 43.6763,5.4556 43.1203,6.1216 L43.0463,6.2086 L43.2323,7.3156 L43.5423,7.1416 C44.2203,6.7606 44.9873,6.5756 45.8853,6.5756 C47.3883,6.5756 48.0583,7.3056 48.0583,8.9406 L48.0583,9.9646 C47.7473,9.9536 47.3743,9.9536 47.1403,9.9536 C44.2153,9.9536 42.3263,11.3916 42.3263,13.6166 C42.3263,15.3826 43.5943,16.6146 45.4093,16.6146 C46.5793,16.6146 47.5903,16.0146 48.1563,15.4276 C48.3693,16.1646 49.0103,16.6146 49.9083,16.6146 C50.5873,16.6146 51.2873,16.4416 51.6903,16.1736 L51.8023,16.0996 L51.8023,15.0956 L51.5523,15.0956 C50.2953,15.0956 50.0933,14.8226 50.0933,13.9486" id="Fill-5"></path><path d="M59.0356,4.7681 C58.2956,4.7681 57.2576,5.2701 56.5606,5.9141 L56.5606,4.5201 L56.2076,4.6791 C55.1506,5.1561 54.3886,5.3901 53.2336,5.7311 L53.0546,5.7851 L53.0546,6.7071 L53.2536,6.7481 C54.4596,6.9951 54.4596,7.0851 54.4596,8.1871 L54.4596,13.5941 C54.4596,15.0311 54.2906,15.0611 53.9576,15.1201 L53.0326,15.2641 L53.0326,16.3581 L53.2926,16.3481 C53.4566,16.3411 53.6466,16.3311 53.8546,16.3191 C54.3236,16.2921 54.9066,16.2601 55.4886,16.2601 C56.1096,16.2601 56.9296,16.2981 57.5286,16.3271 L58.2686,16.3581 L58.2686,15.2541 L57.0586,15.1191 C56.7586,15.0781 56.5606,15.0491 56.5606,13.5941 L56.5606,7.4431 C57.1436,6.9961 57.8986,6.7311 58.6026,6.7311 C59.0176,6.7311 59.4906,6.8061 59.6786,6.9031 L59.9996,7.0671 L60.2386,4.9621 L60.0276,4.9091 C59.7376,4.8341 59.4436,4.7681 59.0356,4.7681" id="Fill-7"></path><path d="M68.7998,14.1636 C68.1938,14.5786 67.3528,14.8076 66.4328,14.8076 C64.2258,14.8076 62.8548,13.2136 62.8548,10.6466 C62.8548,8.0406 64.1388,6.2216 65.9788,6.2216 C67.2068,6.2216 67.7598,6.7796 67.8258,8.0886 L67.8388,8.3266 L69.1488,8.3266 L69.1488,8.0766 C69.1488,7.3316 69.1708,6.2826 69.2338,5.5716 L69.2498,5.3816 L69.0708,5.3156 C68.4008,5.0656 67.3418,4.7676 66.1088,4.7676 C63.1088,4.7676 60.6678,7.5046 60.6678,10.8686 C60.6678,14.4126 62.6528,16.6146 65.8488,16.6146 C67.4978,16.6146 68.8168,15.8386 69.2658,15.0716 L69.3128,14.9916 L69.1158,13.9466 L68.7998,14.1636 L68.7998,14.1636 Z" id="Fill-9"></path><path d="M102.4863,11.4136 C102.4653,12.0286 102.4343,13.2886 102.4313,14.0086 C102.0513,14.4166 101.3523,14.8286 100.4543,14.8286 C99.5143,14.8286 98.9303,14.2536 98.9303,13.3286 C98.9303,12.0536 99.9303,11.4066 101.9033,11.4066 L101.9323,11.4066 C102.0903,11.4066 102.2993,11.4066 102.4863,11.4136 M104.5533,13.9486 C104.5533,12.9186 104.5773,11.9706 104.5993,11.0576 C104.6203,10.2016 104.6403,9.3766 104.6403,8.5416 C104.6403,6.1436 103.2163,4.7676 100.7353,4.7676 C99.4683,4.7676 98.1363,5.4556 97.5803,6.1206 L97.5063,6.2086 L97.6923,7.3156 L98.0023,7.1416 C98.6813,6.7606 99.4473,6.5756 100.3453,6.5756 C101.8473,6.5756 102.5173,7.3056 102.5173,8.9406 L102.5173,9.9646 C102.2073,9.9536 101.8343,9.9536 101.5993,9.9536 C98.6753,9.9536 96.7863,11.3916 96.7863,13.6166 C96.7863,15.3826 98.0543,16.6146 99.8703,16.6146 C101.0403,16.6146 102.0513,16.0146 102.6173,15.4276 C102.8293,16.1636 103.4703,16.6146 104.3683,16.6146 C105.0573,16.6146 105.7403,16.4456 106.1503,16.1736 L106.2623,16.0996 L106.2623,15.0956 L106.0123,15.0956 C104.7553,15.0956 104.5533,14.8226 104.5533,13.9486" id="Fill-11"></path><path d="M80.5029,13.5942 L80.5029,8.8072 C80.5029,7.0732 80.1369,4.7682 76.9659,4.7682 C75.9289,4.7682 74.8679,5.3242 74.1229,5.9102 L74.1229,0.0002 L73.7589,0.1862 C72.7249,0.7162 71.9489,1.0252 70.7949,1.3652 L70.6159,1.4182 L70.6159,2.3412 L70.8159,2.3822 C72.0229,2.6292 72.0229,2.7192 72.0229,3.8212 L72.0229,13.5942 C72.0229,15.0312 71.8529,15.0612 71.5209,15.1192 L70.5949,15.2642 L70.5949,16.3582 L71.2039,16.3332 C71.7159,16.3112 72.3759,16.2822 73.0509,16.2822 C73.7289,16.2822 74.4499,16.3132 74.9829,16.3352 L75.5499,16.3582 L75.5499,15.2642 L74.6319,15.1202 C74.3149,15.0622 74.1229,15.0272 74.1229,13.5942 L74.1229,7.3252 C74.5089,7.0822 75.4279,6.5752 76.3819,6.5752 C77.8739,6.5752 78.4019,7.2232 78.4019,9.0512 L78.4019,13.5942 C78.4019,15.0312 78.2319,15.0612 77.8999,15.1192 L76.9749,15.2642 L76.9749,16.3582 L77.5829,16.3332 C78.0959,16.3112 78.7569,16.2822 79.4309,16.2822 C80.1069,16.2822 80.8289,16.3122 81.3619,16.3352 L81.9309,16.3582 L81.9309,15.2642 L81.0129,15.1202 C80.6939,15.0622 80.5029,15.0272 80.5029,13.5942" id="Fill-13"></path><path d="M116.5752,8.8901 C116.8012,7.6121 117.5932,6.2431 119.1832,6.2431 C120.5722,6.2431 121.2052,7.4921 121.2052,8.6521 C121.2052,8.8391 121.1742,8.8901 120.9572,8.8901 L116.5752,8.8901 L116.5752,8.8901 Z M119.2702,4.7681 C116.3552,4.7681 114.2402,7.3331 114.2402,10.8691 C114.2402,14.4661 116.1612,16.6151 119.3792,16.6151 C121.1002,16.6151 122.4902,15.8441 122.9242,15.1191 L122.9672,15.0461 L122.8512,14.0491 L122.5302,14.2131 C121.7622,14.6071 120.8392,14.8071 119.7892,14.8071 C117.3262,14.8071 116.4492,12.6691 116.4492,10.6691 C116.4492,10.5451 116.4492,10.4121 116.4542,10.2771 L123.4232,10.2771 L123.4542,10.0641 C123.5002,9.7571 123.5002,9.5241 123.5002,9.2281 C123.5002,7.0801 122.1772,4.7681 119.2702,4.7681 L119.2702,4.7681 L119.2702,4.7681 Z" id="Fill-15"></path><path d="M93.3818,9.7095 C92.3118,9.7095 91.5038,9.6745 90.9698,9.6515 L90.5148,9.6325 L90.5148,10.7425 L91.5368,10.9365 C91.8898,11.0065 92.2228,11.0715 92.2228,12.4415 L92.2228,14.8005 C91.5438,15.1505 90.4888,15.2055 89.8998,15.2055 C87.1598,15.2055 85.3908,12.8515 85.3908,9.2065 C85.3908,5.6185 87.2108,3.2075 89.9208,3.2075 C91.6668,3.2075 92.6998,4.1475 92.8288,5.8565 L92.8468,6.0875 L94.1908,6.0875 L94.1708,5.8195 C94.0828,4.6155 94.0628,3.6235 94.0418,2.6635 L94.0378,2.4925 L93.8778,2.4345 C93.0768,2.1425 91.8008,1.7545 89.9638,1.7545 C85.9348,1.7545 83.0098,4.9535 83.0098,9.3615 C83.0098,13.7945 85.6198,16.6585 89.6608,16.6585 C91.5688,16.6585 93.4848,16.2065 94.2368,15.8845 L94.3888,15.8195 L94.3888,12.4195 C94.3888,11.1095 94.6068,10.9985 94.9868,10.9145 L95.8598,10.7405 L95.8598,9.6325 L95.2628,9.6585 C94.7718,9.6815 94.1598,9.7095 93.3818,9.7095" id="Fill-18"></path><path d="M14.9868,8.8901 C15.2128,7.6121 16.0048,6.2431 17.5958,6.2431 C18.9848,6.2431 19.6168,7.4921 19.6168,8.6521 C19.6168,8.8391 19.5858,8.8901 19.3698,8.8901 L14.9868,8.8901 L14.9868,8.8901 Z M17.6828,4.7681 C14.7678,4.7681 12.6528,7.3331 12.6528,10.8691 C12.6528,14.4661 14.5728,16.6151 17.7908,16.6151 C19.5118,16.6151 20.9018,15.8441 21.3358,15.1191 L21.3798,15.0461 L21.2628,14.0491 L20.9428,14.2131 C20.1738,14.6071 19.2508,14.8071 18.2018,14.8071 C15.7388,14.8071 14.8618,12.6691 14.8618,10.6691 C14.8618,10.5451 14.8618,10.4121 14.8668,10.2771 L21.8338,10.2771 L21.8658,10.0641 C21.9118,9.7571 21.9118,9.5241 21.9118,9.2281 C21.9118,7.0801 20.5888,4.7681 17.6828,4.7681 L17.6828,4.7681 L17.6828,4.7681 Z" id="Fill-20"></path><path d="M27.4121,9.44 C26.0941,8.924 25.2861,8.562 25.2861,7.633 C25.2861,6.366 26.4711,6.177 27.1771,6.177 C28.4461,6.177 29.0321,6.751 29.1341,8.095 L29.1521,8.326 L30.4511,8.326 L30.4331,8.059 C30.3691,7.121 30.3481,6.544 30.3901,5.606 L30.3981,5.43 L30.2351,5.363 C29.5341,5.075 28.3851,4.768 27.2641,4.768 C24.9641,4.768 23.3581,6.074 23.3581,7.943 C23.3581,10.309 25.1621,10.967 26.6121,11.497 C27.9601,11.989 28.7871,12.343 28.7871,13.373 C28.7871,14.446 27.9771,15.139 26.7231,15.139 C25.0521,15.139 24.6691,14.424 24.4841,12.921 L24.4571,12.702 L23.1691,12.702 L23.1861,12.967 C23.2501,14.003 23.2931,14.817 23.2501,15.844 L23.2431,16.033 L23.4211,16.092 C24.4921,16.449 25.5481,16.615 26.7451,16.615 C28.9831,16.615 30.7361,15.074 30.7361,13.107 C30.7361,10.739 28.8931,10.018 27.4121,9.44" id="Fill-22"></path><path d="M8.0239,5.9712 C8.0239,7.6212 6.9349,8.6462 5.1809,8.6462 C4.5739,8.6462 4.2549,8.6312 3.8739,8.6002 L3.8739,3.5002 C4.3279,3.4732 4.8799,3.4732 5.3759,3.4732 C7.0839,3.4732 8.0239,4.3602 8.0239,5.9712 M10.5369,14.1812 C9.4649,12.9832 8.3199,11.3652 7.4349,9.8032 C9.0999,9.2492 10.3619,7.5782 10.3619,5.8602 C10.3619,3.4282 8.6439,1.9762 5.7649,1.9762 C5.0779,1.9762 4.4649,1.9952 3.8979,2.0132 C3.4119,2.0282 2.9599,2.0422 2.5209,2.0422 C1.6729,2.0422 0.7959,2.0422 0.2599,2.0202 L-0.0001,2.0092 L-0.0001,3.1202 L1.0239,3.3132 C1.3759,3.3822 1.7089,3.4482 1.7089,4.8192 L1.7089,13.5502 C1.7089,14.9202 1.3759,14.9862 1.0249,15.0542 L-0.0001,15.2482 L-0.0001,16.3582 L0.5139,16.3382 C1.1019,16.3162 1.9899,16.2822 2.8669,16.2822 C3.8529,16.2822 5.1249,16.3272 5.6259,16.3482 L5.8849,16.3582 L5.8849,15.2382 L4.5479,15.0532 C4.2009,15.0002 3.8739,14.9512 3.8739,13.5502 L3.8739,10.1172 C4.2619,10.1412 4.6409,10.1432 5.1419,10.1432 C6.4799,12.5752 7.7209,14.3452 8.3829,15.1262 C9.1169,16.0292 10.2149,16.5262 11.4749,16.5262 C11.8889,16.5262 12.3109,16.4492 12.5759,16.3252 L12.7199,16.2572 L12.7199,15.2722 L12.4699,15.2722 C11.8699,15.2722 11.1649,14.8742 10.5369,14.1812" id="Fill-24"></path></g></g></svg></a></div><div class="nova-l-flex__item"><div class="nova-o-pack nova-o-pack--gutter-m nova-o-pack--width-auto nova-o-pack--vertical-align-middle lite-page__header-search"><div class="nova-o-pack__item item"><form method="GET" action="search" class="lite-page__header-search-input-wrapper"><input type="hidden" name="context" readonly="" value="publicSearchHeader"/><input placeholder="Search for publications, researchers, or questions" name="q" autoComplete="off" class="lite-page__header-search-input"/><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-green nova-c-button--theme-bare nova-c-button--width-square lite-page__header-search-button" type="submit"><span class="nova-c-button__label"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#magnifier-s"></use></svg></span></button></form></div><div class="nova-o-pack__item item"><div class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-white lite-page__header-search-or">or</div></div><div class="nova-o-pack__item item"><a class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-s nova-c-button--color-white nova-c-button--theme-ghost nova-c-button--width-auto discover-button" href="topics"><span class="nova-c-button__label">Discover by subject area</span></a></div></div></div><div class="nova-l-flex__item nova-l-flex__item--grow"><ul class="lite-page__header-links"><li class="hide-m"><a href="scientific-recruitment/?utm_source=researchgate&amp;utm_medium=community-loggedout&amp;utm_campaign=indextop" class="lite-page__header-link">Recruit researchers</a></li><li class="lite-page__signup-element"><a href="signup.SignUp.html?hdrsu=1&amp;_sg%5B0%5D=SkLrsxIrAcSAQs-EP6C4B2cSIQ3gjFLLt1vvvWFnOzgJPm0j_jVJVsm08zO4DbIlaZsz49PVPJ4bA0d82Lgvk4YKYoU" class="lite-page__header-link">Join for free</a></li><li><div class="lite-page__header-login"><a class="lite-page__header-link js-lite-click" data-lite="{&quot;click&quot;:[{&quot;t&quot;:&quot;.js-target-login&quot;,&quot;a&quot;:&quot;tc(lite-page-hidden)&quot;,&quot;r&quot;:&quot;rco&quot;},{&quot;t&quot;:&quot;.js-target-trusted-device&quot;,&quot;a&quot;:&quot;ac(lite-page-hidden)&quot;}]}">Login</a><div class="lite-page__header-login-container lite-page-hidden js-target-login"><form method="post" action="https://www.researchgate.net/login?_sg=qjsCDtEKI5DIxVEzRcCXHBWE6yP8zXMyxZAccgvoChNMXMHoBDZYyVZYIql6DqLlYwt1kn6mlfrYwQ" name="loginForm" id="headerLoginForm"><input type="hidden" name="request_token" value="aad-MnAsgky7BBrl2CWulprPM5UIQeUzEXaElK4aaa2+VTdjxzeGrAHWs0cNP6eD9N4eZeprz/oP+IPW/pBmmk3qJjenClhdAzn/CVSc9hcpIics0M3u+NSj2wuGb4mbbRmcJBTxY+MvdCpqrOVw1nt+CDbId0Rf2dSUfclyeIkyNumPVZ+NhV+Q9ytyGXeI6qT4t5Tgoslkscqo/ug3PRO9HicP3SI3ovK7J3yH/C/ezYQ4SPCo09/aMpMni4w/Xi65C+QzrFBOS90If0QlJRE="/><input type="hidden" name="urlAfterLogin" value="publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera"/><input type="hidden" name="invalidPasswordCount" value="0"/><input type="hidden" name="headerLogin" value="yes"/><div class="lite-page__header-login-item"><label class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit lite-page__header-login-label" for="input-header-login">Email <div class="lite-page-tooltip"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#info-circle-s"></use></svg><div class="lite-page-tooltip__content lite-page-tooltip__content--middle"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><b>Tip:</b> Most researchers use their institutional email address as their ResearchGate login</div><div class="lite-page-tooltip__arrow lite-page-tooltip__arrow--middle"><div class="lite-page-tooltip__arrow-tip"></div></div></div></div></label></div><input type="email" required="" id="input-header-login" name="login" autoComplete="email" tabindex="1" placeholder="" class="nova-e-input__field nova-e-input__field--size-m lite-page__header-login-item nova-e-input__ambient nova-e-input__ambient--theme-default"/><div class="lite-page__header-login-item"><label class="lite-page__header-login-label" for="input-header-password">Password</label><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-bare lite-page__header-login-forgot" href="application.LostPassword.html">Forgot password?</a></div><input type="password" required="" id="input-header-password" name="password" autoComplete="current-password" tabindex="2" placeholder="" class="nova-e-input__field nova-e-input__field--size-m lite-page__header-login-item nova-e-input__ambient nova-e-input__ambient--theme-default"/><div><label class="nova-e-checkbox lite-page__header-login-checkbox"><input type="checkbox" class="nova-e-checkbox__input" aria-invalid="false" name="setLoginCookie" tabindex="3" value="yes" checked=""/><span class="nova-e-checkbox__label"> Keep me logged in</span></label></div><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-m nova-l-flex--direction-column@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-center@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-full" type="submit" tabindex="4"><span class="nova-c-button__label">Log in</span></button></div><div class="nova-l-flex__item nova-l-flex__item--align-self-center@s-up"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit">or</div></div><div class="nova-l-flex__item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-m nova-l-flex--direction-column@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item"><a href="connector/facebook"><img width="247" height="40" src="https://c5.rgstatic.net/m/429978991806330/images/socialNetworks/loginVia/continue-with-facebook.png"/></a></div><div class="nova-l-flex__item"><a href="connector/linkedin"><div style="display:inline-block;width:247px;height:40px;text-align:left;border-radius:2px;white-space:nowrap;background:#0077B5"><span style="margin:1px 0 0 1px;display:inline-block;vertical-align:middle;width:38px;height:38px;background-color:#0077B5"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-m nova-e-icon--theme-bare nova-e-icon--color-white nova-e-icon--luminosity-medium" style="padding:8px 0 0 8px"><use xlink:href="/m/null/images/icons/nova/icon-stack-m.svg#social-linkedin-m"></use></svg></span><span style="color:#FFF;display:inline-block;vertical-align:middle;padding-left:15px;padding-right:42px;font-size:16px;font-family:Roboto, sans-serif">Continue with LinkedIn</span></div></a></div><div class="nova-l-flex__item"><a href="connector/google"><div style="display:inline-block;width:247px;height:40px;text-align:left;border-radius:2px;white-space:nowrap;color:#444;background:#4285F4"><span style="margin:1px 0 0 1px;display:inline-block;vertical-align:middle;width:38px;height:38px;background:url(&#x27;images/socialNetworks/logos-official-2019-05/google-logo.svg&#x27;) transparent 50% no-repeat"></span><span style="color:#FFF;display:inline-block;vertical-align:middle;padding-left:15px;padding-right:42px;font-size:16px;font-family:Roboto, sans-serif">Continue with Google</span></div></a></div></div></div></div></form></div><div class="lite-page-hidden js-lite-domready" data-lite="{&quot;domready&quot;:[{&quot;t&quot;:&quot;self&quot;,&quot;a&quot;:&quot;lawg(lite.TrustedDeviceLoginPromo.html)&quot;,&quot;c&quot;:1}]}"></div><div id="login-modal" class="lite-page-modal lite-page-modal--hidden lite-page-modal lite-page-modal--enable-click-outside"><span><div class="nova-c-modal nova-c-modal--color-green nova-c-modal--position-center nova-c-modal--spacing-xxxl nova-c-modal--width-s lite-page-modal__login" style="z-index:9998"><div class="nova-c-modal__overlay"></div><div class="nova-c-modal__container"><div class="nova-c-modal__window" role="dialog" aria-modal="true" tabindex="-1"><header class="nova-c-modal__header nova-c-modal__header--spacing-none lite-page-modal__header"><div class="nova-c-modal__header-content"><button class="nova-c-button nova-c-button--align-left nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-grey nova-c-button--theme-bare nova-c-button--width-square lite-page-modal__close lite-page-modal__close--top-right" type="button"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-s nova-e-icon--theme-bare nova-e-icon--color-inherit nova-e-icon--luminosity-medium nova-c-button__icon"><use xlink:href="/m/null/images/icons/nova/icon-stack-s.svg#close-circle-s"></use></svg></button></div><div class="nova-c-modal__header-image"></div></header><div class="nova-c-modal__body nova-c-modal__body--spacing-inherit"><div class="nova-c-modal__body-content"><div class="nova-e-text nova-e-text--size-xl nova-e-text--family-sans-serif nova-e-text--spacing-s nova-e-text--color-inherit lite-page-center">Welcome back! Please log in.</div><form method="post" action="https://www.researchgate.net/login?_sg=qjsCDtEKI5DIxVEzRcCXHBWE6yP8zXMyxZAccgvoChNMXMHoBDZYyVZYIql6DqLlYwt1kn6mlfrYwQ" name="loginForm" id="modalLoginForm"><input type="hidden" name="request_token" value="aad-MnAsgky7BBrl2CWulprPM5UIQeUzEXaElK4aaa2+VTdjxzeGrAHWs0cNP6eD9N4eZeprz/oP+IPW/pBmmk3qJjenClhdAzn/CVSc9hcpIics0M3u+NSj2wuGb4mbbRmcJBTxY+MvdCpqrOVw1nt+CDbId0Rf2dSUfclyeIkyNumPVZ+NhV+Q9ytyGXeI6qT4t5Tgoslkscqo/ug3PRO9HicP3SI3ovK7J3yH/C/ezYQ4SPCo09/aMpMni4w/Xi65C+QzrFBOS90If0QlJRE="/><input type="hidden" name="urlAfterLogin" value="publication/320321240_Phylogenetic_relationships_of_millipedes_in_the_subclass_Penicillata_Diplopoda_with_a_key_to_the_genera"/><input type="hidden" name="invalidPasswordCount" value="0"/><input type="hidden" name="modalLogin" value="yes"/><div class="nova-l-form-group nova-l-form-group--layout-stack nova-l-form-group--gutter-s"><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><label class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-xxs nova-e-text--color-inherit nova-e-label" for="input-modal-login-label"><span class="nova-e-label__text">Email <div class="lite-page-tooltip"><span class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-500"> Hint</span><div class="lite-page-tooltip__content lite-page-tooltip__content--middle"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"><b>Tip:</b> Most researchers use their institutional email address as their ResearchGate login</div><div class="lite-page-tooltip__arrow lite-page-tooltip__arrow--middle"><div class="lite-page-tooltip__arrow-tip"></div></div></div></div></span></label><input type="email" required="" id="input-modal-login" name="login" autoComplete="email" tabindex="1" placeholder="Enter your email" class="nova-e-input__field nova-e-input__field--size-m nova-e-input__ambient nova-e-input__ambient--theme-default"/></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div class="lite-page-modal__forgot"><label class="nova-e-text nova-e-text--size-m nova-e-text--family-sans-serif nova-e-text--spacing-xxs nova-e-text--color-inherit nova-e-label" for="input-modal-password-label"><span class="nova-e-label__text">Password</span></label><a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-bare lite-page-modal__forgot-link" href="application.LostPassword.html">Forgot password?</a></div><input type="password" required="" id="input-modal-password" name="password" autoComplete="current-password" tabindex="2" placeholder="" class="nova-e-input__field nova-e-input__field--size-m nova-e-input__ambient nova-e-input__ambient--theme-default"/></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div><label class="nova-e-checkbox"><input type="checkbox" class="nova-e-checkbox__input" aria-invalid="false" checked="" value="yes" name="setLoginCookie" tabindex="3"/><span class="nova-e-checkbox__label"> Keep me logged in</span></label></div></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><button class="nova-c-button nova-c-button--align-center nova-c-button--radius-m nova-c-button--size-m nova-c-button--color-blue nova-c-button--theme-solid nova-c-button--width-full" type="submit" tabindex="4"><span class="nova-c-button__label">Log in</span></button></div><div class="nova-l-form-group__item nova-l-form-group__item--width-auto@m-up"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-m nova-l-flex--direction-column@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit">or</div></div><div class="nova-l-flex__item"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-m nova-l-flex--direction-column@s-up nova-l-flex--align-items-center@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item"><a href="connector/facebook"><img width="247" height="40" src="https://c5.rgstatic.net/m/429978991806330/images/socialNetworks/loginVia/continue-with-facebook.png"/></a></div><div class="nova-l-flex__item"><a href="connector/linkedin"><div style="display:inline-block;width:247px;height:40px;text-align:left;border-radius:2px;white-space:nowrap;background:#0077B5"><span style="margin:1px 0 0 1px;display:inline-block;vertical-align:middle;width:38px;height:38px;background-color:#0077B5"><svg aria-hidden="true" class="nova-e-icon nova-e-icon--size-m nova-e-icon--theme-bare nova-e-icon--color-white nova-e-icon--luminosity-medium" style="padding:8px 0 0 8px"><use xlink:href="/m/null/images/icons/nova/icon-stack-m.svg#social-linkedin-m"></use></svg></span><span style="color:#FFF;display:inline-block;vertical-align:middle;padding-left:15px;padding-right:42px;font-size:16px;font-family:Roboto, sans-serif">Continue with LinkedIn</span></div></a></div><div class="nova-l-flex__item"><a href="connector/google"><div style="display:inline-block;width:247px;height:40px;text-align:left;border-radius:2px;white-space:nowrap;color:#444;background:#4285F4"><span style="margin:1px 0 0 1px;display:inline-block;vertical-align:middle;width:38px;height:38px;background:url(&#x27;images/socialNetworks/logos-official-2019-05/google-logo.svg&#x27;) transparent 50% no-repeat"></span><span style="color:#FFF;display:inline-block;vertical-align:middle;padding-left:15px;padding-right:42px;font-size:16px;font-family:Roboto, sans-serif">Continue with Google</span></div></a></div></div></div><div class="nova-l-flex__item"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-grey-500 lite-page-center">No account? <a class="nova-e-link nova-e-link--color-blue nova-e-link--theme-decorated" href="signup.SignUp.html?hdrsu=1&amp;_sg%5B0%5D=SkLrsxIrAcSAQs-EP6C4B2cSIQ3gjFLLt1vvvWFnOzgJPm0j_jVJVsm08zO4DbIlaZsz49PVPJ4bA0d82Lgvk4YKYoU">Sign up</a></div></div></div></div></div></form></div></div><div class="nova-e-spinner nova-e-spinner--size-m nova-e-spinner--color-inherit nova-c-modal__spinner"><div class="nova-e-spinner__container"><svg width="16" height="16" viewBox="0 0 16 16" class="nova-e-spinner__asset"><g stroke-width="2"><path class="nova-e-spinner__circle" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path><path class="nova-e-spinner__stroke" d="M8,1C4.1,1,1,4.1,1,8s3.1,7,7,7s7-3.1,7-7S11.9,1,8,1"></path></g></svg></div></div></div></div></div></span></div></div></li></ul></div></div></header><footer class="lite-page__footer" role="contentinfo"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xs nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-center@s-up nova-l-flex--wrap-nowrap@s-up lite-page__footer-native-app"><div class="nova-l-flex__item"><span style="background:none"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="go.GetApp.html?interested=true&amp;originCh=footerLoggedOut"><img src="https://i1.rgstatic.net/images/nativeApp/app_store_light_logo.svg" alt="App Store" style="height:48px;width:147px"/></a></span></div></div><div class="lite-page__footer-index"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-m nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-nowrap@s-up"><div class="nova-l-flex__item nova-l-flex__item--grow"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-xxxxl nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up lite-page constrained-box-sizing"><div class="nova-l-flex__item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-m nova-e-text--color-inherit lite-page__footer-index-title">Company</div><div class="nova-o-stack nova-o-stack--gutter-xs nova-o-stack--spacing-none nova-o-stack--no-gutter-outside lite-page__footer-index-column"><div class="nova-o-stack__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="about">About us</a></div><div class="nova-o-stack__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="blog">News</a></div><div class="nova-o-stack__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="careers">Careers</a></div></div></div><div class="nova-l-flex__item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-m nova-e-text--color-inherit lite-page__footer-index-title">Support</div><div class="nova-o-stack nova-o-stack--gutter-xs nova-o-stack--spacing-none nova-o-stack--no-gutter-outside lite-page__footer-index-column"><div class="nova-o-stack__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="deref/https%3A%2F%2Fexplore.researchgate.net%2F%3Futm_source%3Dresearchgate%26utm_medium%3Dcommunity-loggedout%26utm_campaign%3Dnew-footer%26utm_content%3Dhelpcenter?forcePage=false">Help Center</a></div></div></div><div class="nova-l-flex__item"><div class="nova-e-text nova-e-text--size-l nova-e-text--family-sans-serif nova-e-text--spacing-m nova-e-text--color-inherit lite-page__footer-index-title">Business solutions</div><div class="nova-o-stack nova-o-stack--gutter-xs nova-o-stack--spacing-none nova-o-stack--no-gutter-outside lite-page__footer-index-column"><div class="nova-o-stack__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="https://solutions.researchgate.net/advertising/?utm_source=researchgate&amp;utm_medium=community-loggedout&amp;utm_campaign=new-footer&amp;utm_content=advertising">Advertising</a></div><div class="nova-o-stack__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-decorated" href="scientific-recruitment?utm_source=researchgate&amp;utm_medium=community-loggedout&amp;utm_campaign=new-footer&amp;utm_content=recruiting">Recruiting</a></div></div></div></div></div><div class="nova-l-flex__item nova-l-flex__item--shrink lite-page__footer-ad-slot"><div id="lite-ad-eidynnq1q1" class="lite-page-ad lite-page-ad--align-center lite-page-ad--has-label"><div class="lite-page-ad__inner"><div id="eidynnq1q1"><rg-ad id="eb47gfob0hk" ad-unit="LoggedOut_PublicationWithoutFulltext_Footer" ad-unit-path="/83500759/LoggedOut_PublicationWithoutFulltext_Footer" ad-location="Footer" refresh-timeout="-1" load-on-init="false" collapse="never" class="rg-ad" ad-unit-sizes="[[336,280],[300,250],[250,250],&quot;fluid&quot;]" targeting-variants="prd,lite-page,lite-ad-manager-on,exp-irl6190Kc7JvArbNObgNIY56" targeting-snippet_id="eb47gfob0hk" targeting-sequence="1"><rg-ad-size size="[336,280]"></rg-ad-size><rg-ad-size size="[300,250]"></rg-ad-size><rg-ad-size size="[250,250]"></rg-ad-size><rg-ad-size size="&quot;fluid&quot;"></rg-ad-size><rg-ad-sizeset viewport-size="[0,0]" slot-sizes="[]"></rg-ad-sizeset><rg-ad-sizeset viewport-size="[768,0]" slot-sizes="[[336,280],[300,250],[250,250],&quot;fluid&quot;]"></rg-ad-sizeset></rg-ad><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">document.getElementById("eb47gfob0hk").addEventListener('adUnitEvent', function(event){if(event.detail.type==="slotRenderEnded"){ if(event.detail.gptEvent.isEmpty){document.querySelector('#lite-ad-eidynnq1q1').classList.remove('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-eidynnq1q1");elem && elem.classList.add('lite-page-ad--empty');}else{document.querySelector('#lite-ad-eidynnq1q1').classList.add('lite-page-ad--show-label');var elem=document.querySelector("#lite-ad-eidynnq1q1");elem && elem.classList.remove('lite-page-ad--empty');}}})</script></div></div></div></div></div></div><div class="lite-page__footer-index"><div class="nova-l-flex__item nova-l-flex nova-l-flex--gutter-m nova-l-flex--direction-row@s-up nova-l-flex--align-items-stretch@s-up nova-l-flex--justify-content-flex-start@s-up nova-l-flex--wrap-wrap@s-up lite-page constrained-box-sizing"><div class="nova-l-flex__item"><div class="nova-e-text nova-e-text--size-s nova-e-text--family-sans-serif nova-e-text--spacing-none nova-e-text--color-inherit"> 2008-2020 ResearchGate GmbH. All rights reserved.</div></div><div class="nova-l-flex__item nova-l-flex__item--grow lite-page__footer-base-right"><ul class="nova-e-list nova-e-list--size-s nova-e-list--type-inline nova-e-list--spacing-auto"><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="terms-of-service">Terms</a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="privacy-policy">Privacy</a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="ip-policy">Copyright</a></li><li class="nova-e-list__item"><a class="nova-e-link nova-e-link--color-inherit nova-e-link--theme-bare" href="imprint">Imprint</a></li></ul></div></div></div></footer><div class="js-target-questionnaire"></div></div><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">var r=new XMLHttpRequest;r.open("GET","/refreshToken?c="+(requestCachedConfig.correlationId||requestConfig.correlationId));r.setRequestHeader("x-rg-referer",window.location.href);r.onreadystatechange=function(){if(4===this.readyState&&200<=this.status&&400>this.status){var a=JSON.parse(this.responseText),b=document.getElementById("Rg-Request-Token");b&&b.setAttribute("content",a.requestToken);a.result&&a.result.trackingConfig&&(window.trackingConfig=a.result.trackingConfig,window.useQuantcast||window.loadDeferredObjects&&window.loadDeferredObjects())}};r.send();r=null;</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">requestConfig.backendTime = 567</script>
<script src="https://c5.rgstatic.net/m/4183121042180509/javascript/lite/lite.js" onerror="throw new Error('Failed to load file javascript/lite/lite.js');" crossorigin="anonymous" defer></script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,"script","https://www.google-analytics.com/analytics.js","ga");
ga("create","UA-58591210-1"); ga("set","anonymizeIp",true); ga("set", "dimension5", "traffic: experiment:No Fulltext Mobile Catfish AdSlot;variant:__control");ga("set", "dimension1", "publication no texts - redesigned");ga("set", "dimension2", "Article");ga("set", "dimension3", "Logged out");ga("set", "dimension4", "lite-page");ga("set", "dimension6", "n\/a");ga("set", "dimension7", "n\/a");ga("set", "dimension8", "n\/a");ga("set", "dimension9", "n\/a");ga("set", "dimension10", "Engineering");ga("set", "dimension11", "Evolutionary_Biology");ga("set", "dimension12", "Engineering Biology Mathematics");ga("set", "dimension13", "Evolutionary_Biology Systematics_Taxonomy Genetics");(typeof requestCachedConfig!==typeof undefined&&requestCachedConfig.isCachedPage===true)?ga("set", "dimension19", "cache"):ga("set", "dimension19", "no cache");  ga("send","pageview");</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">
window.addEventListener('load', function() {
if(typeof trackingConfig!==typeof undefined&&trackingConfig.isCrawler===false){(typeof requestCachedConfig!==typeof undefined&&requestCachedConfig.isCachedPage===true)?RGCommons.monitoring.monitorPage("https:\/\/glassmoni.researchgate.net", ["lite.PublicationDetails.run.html.loggedOut.get","lite.PublicationDetails.run.html.loggedOut.get.no-text","lite.PublicationDetails.run.html.loggedOut.get.subcategory-cached","lite.PublicationDetails.run.html.loggedOut.get.subcategory-no-text-cached"], "9723c55cb9c841461916947f0fb722912dc2fa1e"):RGCommons.monitoring.monitorPage("https:\/\/glassmoni.researchgate.net", ["lite.PublicationDetails.run.html.loggedOut.get","lite.PublicationDetails.run.html.loggedOut.get.no-text","lite.PublicationDetails.run.html.loggedOut.get.subcategory-non-cached","lite.PublicationDetails.run.html.loggedOut.get.subcategory-no-text-non-cached"], "ebe70b41ef77e145eeb36ff2c9d9af10c2203172");}if(typeof trackingConfig!==typeof undefined&&trackingConfig.isCrawler===true){RGCommons.monitoring.monitorPage("https:\/\/glassmoni.researchgate.net", ["lite.PublicationDetails.run.html.loggedOut.get","lite.PublicationDetails.run.html.loggedOut.get.no-text","lite.PublicationDetails.run.html.loggedOut.get.subcategory-crawler"], "683f719c433019b604fe7e70c5ad7b15a62b3a69");}
});
</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">
document.addEventListener('DOMContentLoaded', function() {
RGCommons.parameterFilter.filter(["ev","cp","ch","ref","dbw","pli","loginT","uid","claimChannel","enrichId","enrichSource","utm_source","utm_medium","utm_campaign","utm_term","utm_content","el","ci","mfl"]);
});
</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU"> dataLayer = [{"pageCategory":"publication no texts - redesigned","publicationType":"Article","eventCategory":"Publication page"}]; </script><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MKVKH7" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MKVKH7');</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">
var googletag=googletag||{};googletag.cmd=googletag.cmd||[];
(function(){var gads=document.createElement("script");gads.async=true;gads.type="text/javascript";gads.src="https://www.googletagservices.com/tag/js/gpt.js";var node=document.getElementsByTagName("script")[0];node.parentNode.insertBefore(gads,node);})();
</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">!function(a9,a,p,s,t,A,g){if(a[a9])return;function q(c,r){a[a9]._Q.push([c,r])}a[a9]={init:function(){q("i",arguments)},fetchBids:function(){q("f",arguments)},setDisplayBids:function(){},targetingKeys:function(){return[]},_Q:[]};A=p.createElement(s);A.async=!0;A.src=t;g=p.getElementsByTagName(s)[0];g.parentNode.insertBefore(A,g)}("apstag",window,document,"script","//c.amazon-adsystem.com/aax2/apstag.js");</script><script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">var rgDelivery=rgDelivery||{};rgDelivery.cmd=rgDelivery.cmd||[];var deliveryManager=deliveryManager||{_q:[]};deliveryManager.start=deliveryManager.start||function(){deliveryManager._q.push(['s', arguments]);};(function(){var tag=document.createElement("script");tag.async=true;tag.type="text/javascript";tag.src="https://www.researchgate.net/delivery/manager.js?rgKey=PB%3A320321240&qc=1&seed=6635180501083857856";var node=document.getElementsByTagName("script")[0];node.parentNode.insertBefore(tag,node);})();</script>
<script nonce="fIemR4fSYfXX8s1w1N1WiZl72YG9wNVU">
document.addEventListener('DOMContentLoaded', function() {
deliveryManager.start();
});
</script></body></html>
